{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Scrape from biorxiv and from arxiv filtering by category, word in abstract/title, dates, etc."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Set dates to search for\n",
    "\n",
    "import datetime\n",
    "today = datetime.datetime.now().date().strftime(\"%Y-%m-%d\")\n",
    "TimeBefore = (datetime.datetime.now().date()-datetime.timedelta(days=10)).strftime(\"%Y-%m-%d\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Had to use a separate scraper for biorxiv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "import pandas as pd\n",
    "#import seaborn as sns\n",
    "import numpy as np\n",
    "from bs4 import BeautifulSoup as bs\n",
    "import matplotlib.pyplot as plt\n",
    "from tqdm import tqdm\n",
    "import time\n",
    "%matplotlib inline\n",
    "\n",
    "n_results = 1000\n",
    "url = \"http://biorxiv.org/search/neuroscience%20numresults%3A{}\".format(\n",
    "    n_results)\n",
    "resp = requests.post(url)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<!DOCTYPE html>\n",
       "\n",
       "<html dir=\"ltr\" lang=\"en\" xmlns=\"http://www.w3.org/1999/xhtml\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\">\n",
       "<head prefix=\"og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#\">\n",
       "<!--[if IE]><![endif]-->\n",
       "<meta content=\"text/html; charset=utf-8\" http-equiv=\"Content-Type\"/>\n",
       "<link href=\"//stats.g.doubleclick.net\" rel=\"dns-prefetch\"/>\n",
       "<link href=\"//d33xdlntwy0kbs.cloudfront.net\" rel=\"dns-prefetch\"/>\n",
       "<link href=\"//www.google-analytics.com\" rel=\"dns-prefetch\"/>\n",
       "<meta content=\"width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes\" name=\"viewport\"/>\n",
       "<link href=\"https://www.biorxiv.org/sites/default/files/images/favicon.ico\" rel=\"shortcut icon\" type=\"image/vnd.microsoft.icon\"/>\n",
       "<meta content=\"bioRxiv - the preprint server for biology, operated by Cold Spring Harbor Laboratory, a research and educational institution\" name=\"description\"/>\n",
       "<meta content=\"Drupal 7 (http://drupal.org)\" name=\"generator\"/>\n",
       "<link href=\"https://www.biorxiv.org/search/neuroscience%20numresults%3A1000\" rel=\"canonical\"/>\n",
       "<link href=\"https://www.biorxiv.org/search/neuroscience%20numresults%3A1000\" rel=\"shortlink\"/>\n",
       "<title>Search Results | bioRxiv</title>\n",
       "<link href=\"https://www.biorxiv.org/sites/default/files/advagg_css/css__q1jfIfL8-NmYQwIYZiQJkXRvxUhw39T6nMKFZH7cllk__6eR-HG068hreGXGvZigV_cQG10gMcVih32c6FYJBqbE__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.css\" media=\"all\" rel=\"stylesheet\" type=\"text/css\"/>\n",
       "<link href=\"https://www.biorxiv.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?pxfq9d\" media=\"all\" rel=\"stylesheet\" type=\"text/css\"/>\n",
       "<link href=\"https://www.biorxiv.org/sites/default/files/advagg_css/css__T2P134Rgsot7gxBGGcj4_r5CWWPW74URR8Mm862EyyA__GhPo8fFKFQQ1vybQiCzPdAUqYKXjiIHxOSpwbw2LyJk__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.css\" media=\"all\" rel=\"stylesheet\" type=\"text/css\"/>\n",
       "<style media=\"all\" type=\"text/css\">\n",
       "/* <![CDATA[ */\n",
       "#sliding-popup.sliding-popup-bottom,#sliding-popup.sliding-popup-bottom .eu-cookie-withdraw-banner,.eu-cookie-withdraw-tab{background:gray}#sliding-popup.sliding-popup-bottom.eu-cookie-withdraw-wrapper{background:transparent}#sliding-popup .popup-content #popup-text h1,#sliding-popup .popup-content #popup-text h2,#sliding-popup .popup-content #popup-text h3,#sliding-popup .popup-content #popup-text p,.eu-cookie-compliance-secondary-button,.eu-cookie-withdraw-tab{color:#fff !important}.eu-cookie-withdraw-tab{border-color:#fff}.eu-cookie-compliance-more-button{color:#fff !important}\n",
       "/* ]]> */\n",
       "</style>\n",
       "<!--[if lte IE 7]>\n",
       "<link type=\"text/css\" rel=\"stylesheet\" href=\"https://www.biorxiv.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.css\" media=\"all\" />\n",
       "<![endif]-->\n",
       "<link href=\"https://www.biorxiv.org/sites/default/files/advagg_css/css__B6tGdsQSt1vT8d8j-19-oS4FKJe83RrSNBkjPs9jfos__6VkE5WcQmmQpSVTQx-xS2-zn_nlwFlDm-EN1HLJNjrA__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.css\" media=\"all\" rel=\"stylesheet\" type=\"text/css\"/>\n",
       "<link href=\"https://www.biorxiv.org/sites/default/files/advagg_css/css__DB9hNnhwIj3Yq8csxMZked15MhFt3vm-txAnatDbqlY__F3sVKlGyHKCaAPI_WdiHD7l_-05B6b9bLdUwzi1me90__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.css\" media=\"all\" rel=\"stylesheet\" type=\"text/css\"/>\n",
       "<!--[if (lt IE 9)&(!IEMobile)]>\n",
       "<link type=\"text/css\" rel=\"stylesheet\" href=\"https://www.biorxiv.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.css\" media=\"all\" />\n",
       "<![endif]-->\n",
       "<!--[if gte IE 9]><!-->\n",
       "<link href=\"https://www.biorxiv.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.css\" media=\"all\" rel=\"stylesheet\" type=\"text/css\"/>\n",
       "<!--<![endif]-->\n",
       "<link href=\"https://www.biorxiv.org/sites/default/files/advagg_css/css__YsqcsJ9a6cOSCZcaq1450ci_eHVySr_QlqGwCt4noCc__cHebPmPNdg2jyPmpyQpZWssJQ4cPrqfaabp5Or9KlkU__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.css\" media=\"all\" rel=\"stylesheet\" type=\"text/css\"/>\n",
       "<link href=\"https://d33xdlntwy0kbs.cloudfront.net/cshl_custom.css\" media=\"all\" rel=\"stylesheet\" type=\"text/css\"/>\n",
       "<script type=\"text/javascript\">\n",
       "<!--//--><![CDATA[//><!--\n",
       "/*!\n",
       " * yepnope1.5.4\n",
       " * (c) WTFPL, GPLv2\n",
       " */\n",
       "(function(a,b,c){function d(a){return\"[object Function]\"==o.call(a)}function e(a){return\"string\"==typeof a}function f(){}function g(a){return!a||\"loaded\"==a||\"complete\"==a||\"uninitialized\"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){(\"c\"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){\"img\"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),\"object\"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height=\"0\",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),\"img\"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||\"j\",e(a)?i(\"c\"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName(\"script\")[0],o={}.toString,p=[],q=0,r=\"MozAppearance\"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&\"[object Opera]\"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?\"object\":l?\"script\":\"img\",v=l?\"script\":u,w=Array.isArray||function(a){return\"[object Array]\"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split(\"!\"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split(\"=\"),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(\".\").pop().split(\"?\").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split(\"/\").pop().split(\"?\")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&\"css\"==i.url.split(\".\").pop().split(\"?\").shift()?\"c\":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState=\"loading\",b.addEventListener(\"DOMContentLoaded\",A=function(){b.removeEventListener(\"DOMContentLoaded\",A,0),b.readyState=\"complete\"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement(\"script\"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement(\"link\"),j,c=i?h:c||f;e.href=a,e.rel=\"stylesheet\",e.type=\"text/css\";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);\n",
       "\n",
       "//--><!]]>\n",
       "</script>\n",
       "<script src=\"https://www.biorxiv.org/sites/all/libraries/modernizr/modernizr.min.js?pxfq9d\" type=\"text/javascript\"></script>\n",
       "<script type=\"text/javascript\">\n",
       "<!--//--><![CDATA[//><!--\n",
       "yepnope({\n",
       "  test: Modernizr.matchmedia,\n",
       "  nope: '/sites/all/libraries/media-match/media.match.min.js'\n",
       "});\n",
       "//--><!]]>\n",
       "</script>\n",
       "<script src=\"https://www.biorxiv.org/sites/default/files/advagg_js/js__yOhrJUBmMpN-jsuQlJYmS8ovkQU6I6KInr295HrSTb0__c--zHhbuOWmkdWkJYXiNWDmkpDRPesmbBjGlQcyqBOY__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.js\" type=\"text/javascript\"></script>\n",
       "<script type=\"text/javascript\">\n",
       "<!--//--><![CDATA[//><!--\n",
       "var _prum=[['id', '52e95df3abe53d2e33000000'], ['mark', 'firstbyte',\n",
       "      (new Date()).getTime()]]; (function() {\n",
       "      var s=document.getElementsByTagName('script')[0],\n",
       "      p=document.createElement('script');\n",
       "      p.async='async'; p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();\n",
       "//--><!]]>\n",
       "</script>\n",
       "<script src=\"https://www.biorxiv.org/sites/default/files/advagg_js/js__dioWZdkZ5DJ0dfTqTsZRAHiGLlLuJoabg9VyZCEqGn4__JbOvqvpFnOTkVG2lj8k48UUlmucZsXyD0I8FARNagqU__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.js\" type=\"text/javascript\"></script>\n",
       "<script type=\"text/javascript\">\n",
       "<!--//--><![CDATA[//><!--\n",
       "(function(i,s,o,g,r,a,m){i[\"GoogleAnalyticsObject\"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,\"script\",\"//www.google-analytics.com/analytics.js\",\"ga\");ga(\"create\", \"UA-45638026-1\", {\"cookieDomain\":\"auto\"});ga(\"set\", \"page\", location.pathname + location.search + location.hash);ga(\"send\", \"pageview\");ga('create', 'UA-189672-38', 'auto', {'name': 'hwTracker'});\n",
       "ga('set', 'anonymizeIp', true);\n",
       "ga('hwTracker.send', 'pageview');\n",
       "//--><!]]>\n",
       "</script>\n",
       "<script src=\"https://www.biorxiv.org/sites/default/files/advagg_js/js__3IkZVk0yWjBeun_Z3_usxoNjwSWRrZ3Tg8HH_CVSmlk__t5cc6bWAOf7iTgS-Ohair1gd-_P86-jGYmTCh-6Nxqw__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.js\" type=\"text/javascript\"></script>\n",
       "<script type=\"text/javascript\">\n",
       "<!--//--><![CDATA[//><!--\n",
       "jQuery.extend(Drupal.settings,{\"basePath\":\"\\/\",\"pathPrefix\":\"\",\"ajaxPageState\":{\"theme\":\"jcore_1\",\"theme_token\":\"Ebsk7LRgwqnhQVy327LloNqeIdRviN1ZtO2mcDd1El4\",\"jquery_version\":\"1.8\",\"css\":{\"modules\\/system\\/system.base.css\":1,\"modules\\/system\\/system.menus.css\":1,\"modules\\/system\\/system.messages.css\":1,\"modules\\/system\\/system.theme.css\":1,\"sites\\/all\\/libraries\\/chosen\\/chosen.css\":1,\"sites\\/all\\/modules\\/contrib\\/chosen\\/css\\/chosen-drupal.css\":1,\"misc\\/ui\\/jquery.ui.core.css\":1,\"misc\\/ui\\/jquery.ui.theme.css\":1,\"misc\\/ui\\/jquery.ui.datepicker.css\":1,\"sites\\/all\\/modules\\/contrib\\/date\\/date_popup\\/themes\\/jquery.timeentry.css\":1,\"modules\\/comment\\/comment.css\":1,\"sites\\/all\\/modules\\/contrib\\/date\\/date_api\\/date.css\":1,\"sites\\/all\\/modules\\/contrib\\/date\\/date_popup\\/themes\\/datepicker.1.7.css\":1,\"modules\\/field\\/theme\\/field.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_draw\\/css\\/highwire-draw.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_folders\\/highwire_folders.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_responsive\\/css\\/highwire-responsive.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_saved_searches\\/highwire_saved_searches.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_user\\/highwire_user.css\":1,\"modules\\/node\\/node.css\":1,\"modules\\/search\\/search.css\":1,\"modules\\/user\\/user.css\":1,\"modules\\/forum\\/forum.css\":1,\"sites\\/all\\/modules\\/contrib\\/views\\/css\\/views.css\":1,\"sites\\/all\\/modules\\/contrib\\/ckeditor\\/css\\/ckeditor.css\":1,\"sites\\/all\\/modules\\/contrib\\/colorbox\\/styles\\/default\\/colorbox_style.css\":1,\"sites\\/all\\/modules\\/contrib\\/ctools\\/css\\/ctools.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire.style.highwire.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/css\\/nlm-elements.css\":1,\"sites\\/all\\/modules\\/contrib\\/panels\\/css\\/panels.css\":1,\"sites\\/default\\/modules\\/biorxiv\\/css\\/biorxiv.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_search\\/highwire_search.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_search\\/css\\/highwire_search_results_tool.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_save_citation\\/css\\/highwire_save_citation.css\":1,\"sites\\/all\\/modules\\/contrib\\/panels\\/plugins\\/layouts\\/flexible\\/flexible.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_collection\\/css\\/highwire_subject_collection.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/css\\/highwire.highwire-list-expand.css\":1,\"public:\\/\\/ctools\\/css\\/bab59090150caf0c64e27afa1b36eef1.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_search\\/plugins\\/content_types\\/css\\/highwire_search_facets.css\":1,\"sites\\/all\\/modules\\/contrib\\/panels\\/plugins\\/layouts\\/onecol\\/onecol.css\":1,\"sites\\/all\\/modules\\/contrib\\/eu_cookie_compliance\\/css\\/eu_cookie_compliance.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_theme_tools\\/css\\/font-hwicons.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_theme_tools\\/css\\/font-hwicons-glyphs.css\":1,\"0\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/eu_cookie_compliance.css\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/font-awesome-ie7.css\":1,\"sites\\/all\\/themes\\/contrib\\/omega\\/alpha\\/css\\/alpha-reset.css\":1,\"sites\\/all\\/themes\\/contrib\\/omega\\/alpha\\/css\\/alpha-mobile.css\":1,\"sites\\/all\\/themes\\/contrib\\/omega\\/alpha\\/css\\/alpha-alpha.css\":1,\"sites\\/all\\/themes\\/contrib\\/omega\\/omega\\/css\\/formalize.css\":1,\"sites\\/all\\/themes\\/contrib\\/omega\\/omega\\/css\\/omega-text.css\":1,\"sites\\/all\\/themes\\/contrib\\/omega\\/omega\\/css\\/omega-branding.css\":1,\"sites\\/all\\/themes\\/contrib\\/omega\\/omega\\/css\\/omega-menu.css\":1,\"sites\\/all\\/themes\\/contrib\\/omega\\/omega\\/css\\/omega-forms.css\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/css\\/highwire-forms.css\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/mobile-reset.css\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/font-awesome.css\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/forms.css\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/global.css\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/jquery-ui-elements.css\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/text.css\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/colors.css.less\":1,\"ie::normal::sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/jcore-1-jcore-default.css\":1,\"ie::normal::sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/jcore-1-jcore-default-normal.css\":1,\"ie::normal::sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/grid\\/jcore_default\\/normal\\/jcore-default-normal-30.css\":1,\"narrow::sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/jcore-1-jcore-default.css\":1,\"narrow::sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/jcore-1-jcore-default-narrow.css\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/grid\\/jcore_default\\/narrow\\/jcore-default-narrow-30.css\":1,\"normal::sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/jcore-1-jcore-default.css\":1,\"normal::sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/jcore-1-jcore-default-normal.css\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/grid\\/jcore_default\\/normal\\/jcore-default-normal-30.css\":1,\"wide::sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/jcore-1-jcore-default.css\":1,\"wide::sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/jcore-1-jcore-default-wide.css\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/css\\/grid\\/jcore_default\\/wide\\/jcore-default-wide-30.css\":1,\"sites\\/default\\/modules\\/jnl_biorxiv_styles\\/css\\/biorxiv-styles.css.less\":1,\"https:\\/\\/d33xdlntwy0kbs.cloudfront.net\\/cshl_custom.css\":1},\"js\":{\"sites\\/all\\/libraries\\/enquire.js\\/enquire.min.js\":1,\"http:\\/\\/d33xdlntwy0kbs.cloudfront.net\\/cshl_custom.js\":1,\"sites\\/all\\/modules\\/contrib\\/eu_cookie_compliance\\/js\\/eu_cookie_compliance.js\":1,\"sites\\/default\\/modules\\/jnl_biorxiv_styles\\/js\\/adv-search-collection-toggle.js\":1,\"sites\\/all\\/libraries\\/modernizr\\/modernizr.min.js\":1,\"sites\\/all\\/modules\\/contrib\\/jquery_update\\/replace\\/jquery\\/1.8\\/jquery.min.js\":1,\"misc\\/jquery-extend-3.4.0.js\":1,\"misc\\/jquery.once.js\":1,\"misc\\/drupal.js\":1,\"sites\\/all\\/modules\\/contrib\\/jquery_update\\/replace\\/ui\\/ui\\/minified\\/jquery.ui.core.min.js\":1,\"sites\\/all\\/modules\\/contrib\\/eu_cookie_compliance\\/js\\/jquery.cookie-1.4.1.min.js\":1,\"sites\\/all\\/modules\\/contrib\\/jquery_update\\/replace\\/misc\\/jquery.form.min.js\":1,\"sites\\/all\\/modules\\/contrib\\/jquery_update\\/replace\\/ui\\/ui\\/minified\\/jquery.ui.datepicker.min.js\":1,\"modules\\/locale\\/locale.datepicker.js\":1,\"sites\\/all\\/modules\\/contrib\\/date\\/date_popup\\/jquery.timeentry.pack.js\":1,\"misc\\/form.js\":1,\"misc\\/ajax.js\":1,\"sites\\/all\\/modules\\/contrib\\/jquery_update\\/js\\/jquery_update.js\":1,\"sites\\/all\\/libraries\\/chosen\\/chosen.jquery.min.js\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/js\\/highwire.equal-heights.js\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/js\\/highwire.panels-ajax-tab.js\":1,\"sites\\/default\\/modules\\/jnl_biorxiv_styles\\/js\\/biorxiv-scripts.js\":1,\"sites\\/all\\/libraries\\/colorbox\\/jquery.colorbox-min.js\":1,\"sites\\/all\\/modules\\/contrib\\/colorbox\\/js\\/colorbox.js\":1,\"sites\\/all\\/modules\\/contrib\\/colorbox\\/styles\\/default\\/colorbox_style.js\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_log\\/highwire_log.js\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_nps_survey\\/js\\/nps_survey.js\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_user\\/js\\/highwire_user_meta.js\":1,\"sites\\/all\\/modules\\/contrib\\/panels\\/js\\/panels.js\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_search\\/js\\/highwire_search_results_tool.js\":1,\"misc\\/progress.js\":1,\"sites\\/all\\/modules\\/contrib\\/date\\/date_popup\\/date_popup.js\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/js\\/highwire.date-field.js\":1,\"misc\\/collapse.js\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/js\\/highwire.highwire-list-expand.js\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_search\\/plugins\\/content_types\\/js\\/highwire_search_facets.js\":1,\"sites\\/all\\/modules\\/contrib\\/google_analytics\\/googleanalytics.js\":1,\"sites\\/all\\/modules\\/contrib\\/chosen\\/chosen.js\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_save_citation\\/js\\/highwire_save_citation.js\":1,\"sites\\/all\\/themes\\/highwire\\/jcore_1\\/js\\/theme-scripts.js\":1,\"sites\\/all\\/themes\\/contrib\\/omega\\/omega\\/js\\/jquery.formalize.js\":1,\"sites\\/all\\/themes\\/contrib\\/omega\\/omega\\/js\\/omega-mediaqueries.js\":1,\"sites\\/all\\/themes\\/contrib\\/omega\\/omega\\/js\\/omega-equalheights.js\":1,\"sites\\/all\\/modules\\/highwire\\/highwire\\/highwire_responsive\\/js\\/highwire-mediaqueries.js\":1}},\"colorbox\":{\"opacity\":\"0.85\",\"current\":\"{current} of {total}\",\"previous\":\"\\u00ab Prev\",\"next\":\"Next \\u00bb\",\"close\":\"Close\",\"maxWidth\":\"98%\",\"maxHeight\":\"98%\",\"fixed\":true,\"mobiledetect\":true,\"mobiledevicewidth\":\"480px\"},\"jcarousel\":{\"ajaxPath\":\"\\/jcarousel\\/ajax\\/views\"},\"chosen\":{\"selector\":\"select:visible\",\"minimum_single\":20,\"minimum_multiple\":20,\"minimum_width\":250,\"options\":{\"disable_search\":false,\"disable_search_threshold\":0,\"search_contains\":false,\"placeholder_text_multiple\":\"Choose some options\",\"placeholder_text_single\":\"Choose an option\",\"no_results_text\":\"No results match\",\"inherit_select_classes\":true}},\"ajax\":{\"edit-numresults\":{\"callback\":\"highwire_search_result_change_on_numresults\",\"event\":\"change\",\"url\":\"\\/system\\/ajax\",\"submit\":{\"_triggering_element_name\":\"numresults\"}},\"edit-sort\":{\"callback\":\"highwire_search_result_change_on_numresults\",\"event\":\"change\",\"url\":\"\\/system\\/ajax\",\"submit\":{\"_triggering_element_name\":\"sort\"}}},\"urlIsAjaxTrusted\":{\"\\/system\\/ajax\":true,\"\\/search\\/neuroscience%20numresults:1000\":true},\"jquery\":{\"ui\":{\"datepicker\":{\"isRTL\":false,\"firstDay\":0}}},\"datePopup\":{\"edit-limit-from-datepicker-popup-0\":{\"func\":\"datepicker\",\"settings\":{\"changeMonth\":true,\"changeYear\":true,\"autoPopUp\":\"focus\",\"closeAtTop\":false,\"speed\":\"immediate\",\"firstDay\":0,\"dateFormat\":\"yy-mm-dd\",\"yearRange\":\"-50:+3\",\"fromTo\":false,\"defaultDate\":\"0y\"}},\"edit-limit-to-datepicker-popup-0\":{\"func\":\"datepicker\",\"settings\":{\"changeMonth\":true,\"changeYear\":true,\"autoPopUp\":\"focus\",\"closeAtTop\":false,\"speed\":\"immediate\",\"firstDay\":0,\"dateFormat\":\"yy-mm-dd\",\"yearRange\":\"-50:+3\",\"fromTo\":false,\"defaultDate\":\"0y\"}}},\"highwire_list_expand\":{\"is_collapsed\":\"1\"},\"highwireResponsive\":{\"enquire_enabled\":1,\"breakpoints_configured\":1,\"breakpoints\":{\"zero\":\"all and (min-width: 0px)\",\"xsmall\":\"all and (min-width: 380px)\",\"narrow\":\"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)\",\"normal\":\"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)\",\"wide\":\"all and (min-width: 1220px)\"}},\"eu_cookie_compliance\":{\"popup_enabled\":1,\"popup_agreed_enabled\":0,\"popup_hide_agreed\":0,\"popup_clicking_confirmation\":1,\"popup_scrolling_confirmation\":false,\"popup_html_info\":\"\\u003Cdiv\\u003E\\n  \\u003Cdiv class =\\u0022popup-content info\\u0022\\u003E\\n    \\u003Cdiv id=\\u0022popup-text\\u0022\\u003E\\n      \\u003Cp\\u003EWe use cookies on this site to enhance your user experience. By clicking any link on this page you are giving your consent for us to set cookies.\\u003C\\/p\\u003E\\n    \\u003C\\/div\\u003E\\n    \\u003Cdiv id=\\u0022popup-buttons\\u0022\\u003E\\n      \\u003Cbutton type=\\u0022button\\u0022 class=\\u0022agree-button eu-cookie-compliance-default-button\\u0022\\u003EContinue\\u003C\\/button\\u003E\\n                    \\u003Cbutton type=\\u0022button\\u0022 class=\\u0022find-more-button eu-cookie-compliance-more-button\\u0022\\u003EFind out more\\u003C\\/button\\u003E\\n          \\u003C\\/div\\u003E\\n  \\u003C\\/div\\u003E\\n\\u003C\\/div\\u003E\",\"use_mobile_message\":false,\"mobile_popup_html_info\":\"\\u003Cdiv\\u003E\\n  \\u003Cdiv class =\\u0022popup-content info\\u0022\\u003E\\n    \\u003Cdiv id=\\u0022popup-text\\u0022\\u003E\\n          \\u003C\\/div\\u003E\\n    \\u003Cdiv id=\\u0022popup-buttons\\u0022\\u003E\\n      \\u003Cbutton type=\\u0022button\\u0022 class=\\u0022agree-button eu-cookie-compliance-default-button\\u0022\\u003EContinue\\u003C\\/button\\u003E\\n                    \\u003Cbutton type=\\u0022button\\u0022 class=\\u0022find-more-button eu-cookie-compliance-more-button\\u0022\\u003EFind out more\\u003C\\/button\\u003E\\n          \\u003C\\/div\\u003E\\n  \\u003C\\/div\\u003E\\n\\u003C\\/div\\u003E\\n\",\"mobile_breakpoint\":\"768\",\"popup_html_agreed\":\"\\u003Cdiv\\u003E\\n  \\u003Cdiv class=\\u0022popup-content agreed\\u0022\\u003E\\n    \\u003Cdiv id=\\u0022popup-text\\u0022\\u003E\\n      \\u003Ch2\\u003EThank you for accepting cookies\\u003C\\/h2\\u003E\\u003Cp\\u003EYou can now hide this message or find out more about cookies.\\u003C\\/p\\u003E    \\u003C\\/div\\u003E\\n    \\u003Cdiv id=\\u0022popup-buttons\\u0022\\u003E\\n      \\u003Cbutton type=\\u0022button\\u0022 class=\\u0022hide-popup-button eu-cookie-compliance-hide-button\\u0022\\u003EHide\\u003C\\/button\\u003E\\n              \\u003Cbutton type=\\u0022button\\u0022 class=\\u0022find-more-button eu-cookie-compliance-more-button-thank-you\\u0022 \\u003EMore info\\u003C\\/button\\u003E\\n          \\u003C\\/div\\u003E\\n  \\u003C\\/div\\u003E\\n\\u003C\\/div\\u003E\",\"popup_use_bare_css\":false,\"popup_height\":\"auto\",\"popup_width\":\"100%\",\"popup_delay\":1000,\"popup_link\":\"\\/help\\/cookie-policy\",\"popup_link_new_window\":1,\"popup_position\":null,\"popup_language\":\"en\",\"store_consent\":false,\"better_support_for_screen_readers\":0,\"reload_page\":0,\"domain\":\"\",\"popup_eu_only_js\":0,\"cookie_lifetime\":\"365\",\"cookie_session\":false,\"disagree_do_not_show_popup\":0,\"method\":\"default\",\"whitelisted_cookies\":\"\",\"withdraw_markup\":\"\\u003Cbutton type=\\u0022button\\u0022 class=\\u0022eu-cookie-withdraw-tab\\u0022\\u003EPrivacy settings\\u003C\\/button\\u003E\\n\\u003Cdiv class=\\u0022eu-cookie-withdraw-banner\\u0022\\u003E\\n  \\u003Cdiv class=\\u0022popup-content info\\u0022\\u003E\\n    \\u003Cdiv id=\\u0022popup-text\\u0022\\u003E\\n      \\u003Cp\\u003E\\u0026lt;h2\\u0026gt;We use cookies on this site to enhance your user experience\\u0026lt;\\/h2\\u0026gt;\\u0026lt;p\\u0026gt;You have given your consent for us to set cookies.\\u0026lt;\\/p\\u0026gt;\\u003C\\/p\\u003E\\n    \\u003C\\/div\\u003E\\n    \\u003Cdiv id=\\u0022popup-buttons\\u0022\\u003E\\n      \\u003Cbutton type=\\u0022button\\u0022 class=\\u0022eu-cookie-withdraw-button\\u0022\\u003EWithdraw consent\\u003C\\/button\\u003E\\n    \\u003C\\/div\\u003E\\n  \\u003C\\/div\\u003E\\n\\u003C\\/div\\u003E\\n\",\"withdraw_enabled\":false},\"googleanalytics\":{\"trackOutbound\":1,\"trackMailto\":1,\"trackDownload\":1,\"trackDownloadExtensions\":\"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip\",\"trackColorbox\":1,\"trackUrlFragments\":1},\"jnl_biorxiv_styles\":{\"defaultJCode\":\"biorxiv\"},\"omega\":{\"layouts\":{\"primary\":\"normal\",\"order\":[\"narrow\",\"normal\",\"wide\"],\"queries\":{\"narrow\":\"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)\",\"normal\":\"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)\",\"wide\":\"all and (min-width: 1220px)\"}}}});\n",
       "//--><!]]>\n",
       "</script>\n",
       "<!--[if lt IE 9]><script src=\"http://html5shiv.googlecode.com/svn/trunk/html5.js\"></script><![endif]-->\n",
       "</head>\n",
       "<body class=\"html not-front not-logged-in page-search page-search-neuroscience-numresults1000 context-search hw-default-jcode-biorxiv\">\n",
       "<!-- Google Tag Manager -->\n",
       "<noscript><iframe height=\"0\" src=\"//www.googletagmanager.com/ns.html?id=GTM-M677548\" style=\"display:none;visibility:hidden\" width=\"0\"></iframe></noscript>\n",
       "<script type=\"text/javascript\">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-M677548');</script>\n",
       "<!-- End Google Tag Manager -->\n",
       "<div id=\"skip-link\">\n",
       "<a class=\"element-invisible element-focusable\" href=\"#main-content\">Skip to main content</a>\n",
       "</div>\n",
       "<div class=\"page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col\" id=\"page\">\n",
       "<header class=\"section section-header\" id=\"section-header\">\n",
       "<div class=\"zone zone-branding clearfix print-display-block container-30\" id=\"zone-branding\">\n",
       "<div class=\"grid-15 prefix-1 region region-branding print-display-block\" id=\"region-branding\">\n",
       "<div class=\"region-inner region-branding-inner\">\n",
       "<div class=\"branding-data clearfix\">\n",
       "<div class=\"logo-img\">\n",
       "<a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/\" rel=\"home\"><img alt=\"bioRxiv\" src=\"https://www.biorxiv.org/sites/default/files/bioRxiv_article.jpg\"/></a> </div>\n",
       "</div>\n",
       "</div>\n",
       "</div><div class=\"grid-11 suffix-1 region region-branding-second print-hidden\" id=\"region-branding-second\">\n",
       "<div class=\"region-inner region-branding-second-inner\">\n",
       "<div class=\"block block-system block-menu block-main-menu block-system-main-menu odd block-without-title\" id=\"block-system-main-menu\">\n",
       "<div class=\"block-inner clearfix\">\n",
       "<div class=\"content clearfix\">\n",
       "<nav class=\"menubar-nav\"><ul class=\"menu\" role=\"menu\"><li class=\"first leaf\" role=\"menuitem\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/\" title=\"\">Home</a></li>\n",
       "<li class=\"leaf\" role=\"menuitem\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/about-biorxiv\">About</a></li>\n",
       "<li class=\"leaf\" role=\"menuitem\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/submit-a-manuscript\">Submit</a></li>\n",
       "<li class=\"last leaf\" role=\"menuitem\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/content/alertsrss\" title=\"\">ALERTS / RSS</a></li>\n",
       "</ul></nav> </div>\n",
       "</div>\n",
       "</div><div class=\"block block-panels-mini block-biorxiv-search-box block-panels-mini-biorxiv-search-box even block-without-title\" id=\"block-panels-mini-biorxiv-search-box\">\n",
       "<div class=\"block-inner clearfix\">\n",
       "<div class=\"content clearfix\">\n",
       "<div class=\"panel-display panel-1col clearfix\" id=\"mini-panel-biorxiv_search_box\">\n",
       "<div class=\"panel-panel panel-col\">\n",
       "<div><div class=\"panel-pane pane-highwire-seach-quicksearch\">\n",
       "<div class=\"pane-content\">\n",
       "<form accept-charset=\"UTF-8\" action=\"/search/neuroscience%20numresults:1000\" class=\"highwire-quicksearch button-style-mini button-style-mini\" id=\"highwire-search-quicksearch-form-0\" method=\"post\"><div><div class=\"form-item form-item-label-invisible form-type-textfield form-item-keywords\">\n",
       "<label class=\"element-invisible\" for=\"edit-keywords--2\">Search for this keyword </label>\n",
       "<input class=\"form-text\" id=\"edit-keywords--2\" maxlength=\"128\" name=\"keywords\" placeholder=\"Search\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "</div>\n",
       "<div class=\"button-wrapper button-mini\"><i class=\"icon-search\"></i><input class=\"form-submit\" data-font-icon=\"icon-search\" data-icon-only=\"1\" data-icon-position=\"after\" id=\"edit-submit--3\" name=\"op\" type=\"submit\" value=\"Search\"/></div><input name=\"form_build_id\" type=\"hidden\" value=\"form-6KAwg33MhfYKcn3EZZpIHAmSPO2L9EEWdSfkPqhtHsI\"/>\n",
       "<input name=\"form_id\" type=\"hidden\" value=\"highwire_search_quicksearch_form_0\"/>\n",
       "</div></form> </div>\n",
       "</div>\n",
       "<div class=\"panel-separator\"></div><div class=\"panel-pane pane-custom pane-1 advanced-search-link\">\n",
       "<div class=\"pane-content\">\n",
       "<a href=\"/search\">Advanced Search</a> </div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div> </div>\n",
       "</div> </div>\n",
       "<div class=\"zone zone-header clearfix container-30\" id=\"zone-header\">\n",
       "</div>\n",
       "</header>\n",
       "<section class=\"section section-content\" id=\"section-content\">\n",
       "<div class=\"zone zone-content clearfix container-30\" id=\"zone-content\">\n",
       "<div class=\"grid-28 suffix-1 prefix-1 region region-content\" id=\"region-content\">\n",
       "<div class=\"region-inner region-content-inner\">\n",
       "<a id=\"main-content\"></a>\n",
       "<div class=\"block block-system block-main block-system-main odd block-without-title\" id=\"block-system-main\">\n",
       "<div class=\"block-inner clearfix\">\n",
       "<div class=\"content clearfix\">\n",
       "<div class=\"panel-display panels-960-layout jcore-2col-layout\">\n",
       "<div class=\"panel-row-wrapper panel-row-first clearfix\">\n",
       "<div class=\"top-wrapper\">\n",
       "<div class=\"panel-panel panel-region-top\">\n",
       "<div class=\"inside\"><div class=\"panel-pane pane-highwire-search-token-summary pane-page-title\">\n",
       "<div class=\"pane-content\">\n",
       "<div class=\"highwire-search-summary\" id=\"search-summary-wrapper\"><h1 id=\"page-title\">\n",
       "\tNo Results</h1>\n",
       "</div> </div>\n",
       "</div>\n",
       "<div class=\"panel-separator\"></div><div class=\"panel-pane pane-highwire-search-token-summary pane-page-title-suffix\">\n",
       "<div class=\"pane-content\">\n",
       "<div class=\"highwire-search-summary\" id=\"search-summary-wrapper--2\"><p>for term \"neuroscience\"</p>\n",
       "</div> </div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div> <!-- /.panel-row-wrapper -->\n",
       "<div class=\"panel-row-wrapper clearfix\">\n",
       "<div class=\"main-content-wrapper grid-17 suffix-1 alpha\">\n",
       "<div class=\"panel-panel panel-region-content\">\n",
       "<div class=\"inside\"><div class=\"panel-pane pane-highwire-search-results-tool\">\n",
       "<div class=\"pane-content\">\n",
       "<div class=\"highwire-search-results-tools-wrapper inline clearfix display-extra-form-options display-search-menu\"><div class=\"hw-search-extra-wrapper\"><form accept-charset=\"UTF-8\" action=\"/search/neuroscience%20numresults:1000\" class=\"hw-search-extra\" id=\"highwire-search-extra-dropdown\" method=\"post\"><div><input name=\"custom_redirect\" type=\"hidden\" value=\"search\"/>\n",
       "<input name=\"facet_queries\" type=\"hidden\" value=\"\"/>\n",
       "<div class=\"form-item form-item-label-before form-type-select form-item-numresults\">\n",
       "<label for=\"edit-numresults\">Items/Page </label>\n",
       "<select class=\"hw-search-tool-numresults form-select\" id=\"edit-numresults\" name=\"numresults\"><option value=\"10\">10</option><option value=\"25\">25</option><option value=\"50\">50</option><option value=\"75\">75</option><option value=\"100\">100</option></select>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-before form-type-select form-item-sort\">\n",
       "<label for=\"edit-sort\">Order by </label>\n",
       "<select class=\"hw-search-tool-sort form-select\" id=\"edit-sort\" name=\"sort\"><option selected=\"selected\" value=\"relevance-rank\">Best Match</option><option value=\"publication-date-ascending\">Oldest First</option><option value=\"publication-date-descending\">Newest First</option></select>\n",
       "</div>\n",
       "<input class=\"form-submit\" id=\"highwire_search_result_tools_submit\" name=\"op\" type=\"submit\" value=\"Submit\"/><input name=\"form_build_id\" type=\"hidden\" value=\"form-b71U094hhptTxCKp5zjm3PxFsCIq_EQzciFsnsZft9A\"/>\n",
       "<input name=\"form_id\" type=\"hidden\" value=\"highwire_search_extra_dropdown\"/>\n",
       "</div></form></div><ul class=\"links search-result-tool inline\"><li class=\"edit_link first\"><a class=\"hw-edit-search link-icon-only link-icon\" href=\"# \" title=\"Edit Search\"><i class=\"icon-pencil icon-large\"></i></a></li><li class=\"alert_link last\"><a class=\"hw-alert-search link-icon-only link-icon\" href=\"/alerts?destination=alerts/search/add&amp;search_query=neuroscience%20numresults%3A1000\" title=\"Alert\"><i class=\"icon-rss icon-large\"></i></a></li></ul></div> </div>\n",
       "</div>\n",
       "<div class=\"panel-separator\"></div><div class=\"panel-pane pane-highwire-search-form search-hidden\" id=\"hw-advance-search-form\">\n",
       "<div class=\"pane-content\">\n",
       "<form accept-charset=\"UTF-8\" action=\"/search/neuroscience%20numresults:1000\" class=\"highwire-search-form highwire-advanced-search-form\" id=\"highwire-search-form\" method=\"post\"><div><div class=\"form-item form-item-label-before form-type-textfield form-item-txtsimple\">\n",
       "<label for=\"edit-txtsimple\">Search Terms &amp; Keywords </label>\n",
       "<input class=\"form-text\" id=\"edit-txtsimple\" maxlength=\"128\" name=\"txtsimple\" size=\"60\" type=\"text\" value=\"neuroscience\"/><i class=\"icon-search\"></i>\n",
       "<div class=\"description\">Type a term to search within all articles in this journal: e.g., stem cell</div>\n",
       "</div>\n",
       "<fieldset class=\"collapsible form-wrapper\" id=\"edit-limits\"><legend><span class=\"fieldset-legend\">Limit Results</span></legend><div class=\"fieldset-wrapper\"><span class=\"label filter-by-date-label\">Date of Publication</span><div class=\"filter-by-date clearfix form-wrapper\" id=\"edit-limit-by-date\"><div class=\"container-inline-date\"><div class=\"form-item form-item-label-before form-type-date-popup form-item-limit-from\">\n",
       "<label for=\"edit-limit-from\"><span class=\"element-invisible\">From</span> </label>\n",
       "<div class=\"date-padding\" id=\"edit-limit-from\"><div class=\"form-item form-item-label-invisible form-type-textfield form-item-limit-from-date\">\n",
       "<label class=\"element-invisible\" for=\"edit-limit-from-datepicker-popup-0\">Date </label>\n",
       "<input class=\"form-text\" id=\"edit-limit-from-datepicker-popup-0\" maxlength=\"30\" name=\"limit_from[date]\" size=\"20\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\"> e.g., 2019-09-10</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div><span class=\"date-spacer\"> — </span><div class=\"container-inline-date\"><div class=\"form-item form-item-label-before form-type-date-popup form-item-limit-to\">\n",
       "<label for=\"edit-limit-to\"><span class=\"element-invisible\">Through</span> </label>\n",
       "<div class=\"date-padding\" id=\"edit-limit-to\"><div class=\"form-item form-item-label-invisible form-type-textfield form-item-limit-to-date\">\n",
       "<label class=\"element-invisible\" for=\"edit-limit-to-datepicker-popup-0\">Date </label>\n",
       "<input class=\"form-text\" id=\"edit-limit-to-datepicker-popup-0\" maxlength=\"30\" name=\"limit_to[date]\" size=\"20\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\"> e.g., 2019-09-10</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div></div><div class=\"form-item form-item-label-before form-type-select form-item-jcode\">\n",
       "<label for=\"edit-jcode\">Include articles in bioRxiv and/or medRxiv: </label>\n",
       "<select class=\"chosen-enable form-select\" data-placeholder=\"Select\" id=\"edit-jcode\" multiple=\"multiple\" name=\"jcode[]\"><option value=\"biorxiv\">bioRxiv</option><option value=\"\">bioRxiv and medRxiv</option><option value=\"medrxiv\">medRxiv</option></select>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-before form-type-select form-item-subject-collection-code\">\n",
       "<label for=\"edit-subject-collection-code\">Include articles in subject area(s) </label>\n",
       "<select class=\"chosen-enable form-select\" data-placeholder=\"Select\" id=\"edit-subject-collection-code\" multiple=\"multiple\" name=\"subject_collection_code[]\"><option value=\"0\">-- All Collections --</option><option value=\"Animal Behavior and Cognition\">Animal Behavior and Cognition</option><option value=\"Biochemistry\">Biochemistry</option><option value=\"Bioengineering\">Bioengineering</option><option value=\"Bioinformatics\">Bioinformatics</option><option value=\"Biophysics\">Biophysics</option><option value=\"Cancer Biology\">Cancer Biology</option><option value=\"Cell Biology\">Cell Biology</option><option value=\"Clinical Trials\">Clinical Trials</option><option value=\"Developmental Biology\">Developmental Biology</option><option value=\"Ecology\">Ecology</option><option value=\"Epidemiology\">Epidemiology</option><option value=\"Evolutionary Biology\">Evolutionary Biology</option><option value=\"Genetics\">Genetics</option><option value=\"Genomics\">Genomics</option><option value=\"Immunology\">Immunology</option><option value=\"Microbiology\">Microbiology</option><option value=\"Molecular Biology\">Molecular Biology</option><option value=\"Neuroscience\">Neuroscience</option><option value=\"Paleontology\">Paleontology</option><option value=\"Pathology\">Pathology</option><option value=\"Pharmacology\">Pharmacology and Toxicology</option><option value=\"Physiology\">Physiology</option><option value=\"Plant Biology\">Plant Biology</option><option value=\"Scientific Communication\">Scientific Communication and Education</option><option value=\"Synthetic Biology\">Synthetic Biology</option><option value=\"Systems Biology\">Systems Biology</option><option value=\"Zoology\">Zoology</option></select>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-before form-type-select form-item-toc-section\">\n",
       "<label for=\"edit-toc-section\">Include articles in Table of Contents section: </label>\n",
       "<select class=\"chosen-enable form-select\" data-placeholder='Choose \"Table of Contents section(s)\"' id=\"edit-toc-section\" multiple=\"multiple\" name=\"toc_section[]\"><option value=\"0\">-- All Sections --</option><option value=\"New Results\">New Results</option><option value=\"Contradictory Results\">Contradictory Results</option><option value=\"Confirmatory Results\">Confirmatory Results</option></select>\n",
       "</div>\n",
       "<div class=\"element-hidden\"><div class=\"form-item form-item-label-before form-type-select form-item-series\">\n",
       "<label for=\"edit-series\">Include articles in Series </label>\n",
       "<select class=\"element-hidden chosen-enable form-select\" data-placeholder=\"Choose Article Series\" id=\"edit-series\" multiple=\"multiple\" name=\"series[]\"><option value=\"0\">-- All Series --</option></select>\n",
       "</div>\n",
       "</div><div class=\"element-hidden\"><div class=\"form-item form-item-label-before form-type-checkboxes form-item-flag\">\n",
       "<label for=\"edit-flag\">Include Only: </label>\n",
       "<div class=\"form-checkboxes element-hidden\" id=\"edit-flag\"><div class=\"form-item form-item-label-after form-type-checkbox form-item-flag-OPENACCESS\">\n",
       "<input class=\"element-hidden form-checkbox\" id=\"edit-flag-openaccess\" name=\"flag[OPENACCESS]\" type=\"checkbox\" value=\"OPENACCESS\"/> <label class=\"option\" for=\"edit-flag-openaccess\"> Open Access Articles </label>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-after form-type-checkbox form-item-flag-HWREVIEW\">\n",
       "<input class=\"element-hidden form-checkbox\" id=\"edit-flag-hwreview\" name=\"flag[HWREVIEW]\" type=\"checkbox\" value=\"HWREVIEW\"/> <label class=\"option\" for=\"edit-flag-hwreview\"> Review Articles </label>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div></div></fieldset>\n",
       "<fieldset class=\"collapsible form-wrapper\" id=\"edit-keywords\"><legend><span class=\"fieldset-legend\">Authors, Keywords</span></legend><div class=\"fieldset-wrapper\"><div class=\"fieldset-description\">Search for specific authors and/or words and phrases.</div><div class=\"form-item form-item-label-before form-type-textfield form-item-author1\">\n",
       "<label for=\"edit-author1\">Author </label>\n",
       "<input class=\"form-text\" id=\"edit-author1\" maxlength=\"128\" name=\"author1\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\">e.g., Smith, JS</div>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-before form-type-textfield form-item-author2\">\n",
       "<label for=\"edit-author2\">Author </label>\n",
       "<input class=\"form-text\" id=\"edit-author2\" maxlength=\"128\" name=\"author2\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\">e.g., Smith, JS</div>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-before form-type-textfield form-item-title\">\n",
       "<label for=\"edit-title\">Title </label>\n",
       "<input class=\"form-text\" id=\"edit-title\" maxlength=\"128\" name=\"title\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\">Type any phrase that appears in the article title</div>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-before form-type-radios form-item-title-flags\">\n",
       "<label for=\"edit-title-flags\">words </label>\n",
       "<div class=\"form-radios\" id=\"edit-title-flags\"><div class=\"form-item form-item-label-after form-type-radio form-item-title-flags\">\n",
       "<input class=\"form-radio\" id=\"edit-title-flags-match-any\" name=\"title_flags\" type=\"radio\" value=\"match-any\"/> <label class=\"option\" for=\"edit-title-flags-match-any\">any </label>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-after form-type-radio form-item-title-flags\">\n",
       "<input checked=\"checked\" class=\"form-radio\" id=\"edit-title-flags-match-all\" name=\"title_flags\" type=\"radio\" value=\"match-all\"/> <label class=\"option\" for=\"edit-title-flags-match-all\">all </label>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-after form-type-radio form-item-title-flags\">\n",
       "<input class=\"form-radio\" id=\"edit-title-flags-match-phrase\" name=\"title_flags\" type=\"radio\" value=\"match-phrase\"/> <label class=\"option\" for=\"edit-title-flags-match-phrase\">phrase </label>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-before form-type-textfield form-item-abstract-title\">\n",
       "<label for=\"edit-abstract-title\">Abstract or Title </label>\n",
       "<input class=\"form-text\" id=\"edit-abstract-title\" maxlength=\"128\" name=\"abstract_title\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\">Type any phrase that appears within article title or abstract</div>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-before form-type-radios form-item-abstract-title-flags\">\n",
       "<label for=\"edit-abstract-title-flags\">words </label>\n",
       "<div class=\"form-radios\" id=\"edit-abstract-title-flags\"><div class=\"form-item form-item-label-after form-type-radio form-item-abstract-title-flags\">\n",
       "<input class=\"form-radio\" id=\"edit-abstract-title-flags-match-any\" name=\"abstract_title_flags\" type=\"radio\" value=\"match-any\"/> <label class=\"option\" for=\"edit-abstract-title-flags-match-any\">any </label>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-after form-type-radio form-item-abstract-title-flags\">\n",
       "<input checked=\"checked\" class=\"form-radio\" id=\"edit-abstract-title-flags-match-all\" name=\"abstract_title_flags\" type=\"radio\" value=\"match-all\"/> <label class=\"option\" for=\"edit-abstract-title-flags-match-all\">all </label>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-after form-type-radio form-item-abstract-title-flags\">\n",
       "<input class=\"form-radio\" id=\"edit-abstract-title-flags-match-phrase\" name=\"abstract_title_flags\" type=\"radio\" value=\"match-phrase\"/> <label class=\"option\" for=\"edit-abstract-title-flags-match-phrase\">phrase </label>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-before form-type-textfield form-item-text-abstract-title\">\n",
       "<label for=\"edit-text-abstract-title\">Full Text or Abstract or Title </label>\n",
       "<input class=\"form-text\" id=\"edit-text-abstract-title\" maxlength=\"128\" name=\"text_abstract_title\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\">Type any phrase that appears within article body, title or abstract</div>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-before form-type-radios form-item-text-abstract-title-flags\">\n",
       "<label for=\"edit-text-abstract-title-flags\">words </label>\n",
       "<div class=\"form-radios\" id=\"edit-text-abstract-title-flags\"><div class=\"form-item form-item-label-after form-type-radio form-item-text-abstract-title-flags\">\n",
       "<input class=\"form-radio\" id=\"edit-text-abstract-title-flags-match-any\" name=\"text_abstract_title_flags\" type=\"radio\" value=\"match-any\"/> <label class=\"option\" for=\"edit-text-abstract-title-flags-match-any\">any </label>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-after form-type-radio form-item-text-abstract-title-flags\">\n",
       "<input checked=\"checked\" class=\"form-radio\" id=\"edit-text-abstract-title-flags-match-all\" name=\"text_abstract_title_flags\" type=\"radio\" value=\"match-all\"/> <label class=\"option\" for=\"edit-text-abstract-title-flags-match-all\">all </label>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-after form-type-radio form-item-text-abstract-title-flags\">\n",
       "<input class=\"form-radio\" id=\"edit-text-abstract-title-flags-match-phrase\" name=\"text_abstract_title_flags\" type=\"radio\" value=\"match-phrase\"/> <label class=\"option\" for=\"edit-text-abstract-title-flags-match-phrase\">phrase </label>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "<div class=\"element-hidden\"><div class=\"form-item form-item-label-before form-type-textfield form-item-references\">\n",
       "<label for=\"edit-references\">Bibliography </label>\n",
       "<input class=\"element-hidden form-text\" id=\"edit-references\" maxlength=\"128\" name=\"references\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\">e.g., Smith, JS</div>\n",
       "</div>\n",
       "</div><div class=\"element-hidden\"><div class=\"form-item form-item-label-before form-type-radios form-item-references-flags\">\n",
       "<label for=\"edit-references-flags\">words </label>\n",
       "<div class=\"form-radios element-hidden\" id=\"edit-references-flags\"><div class=\"form-item form-item-label-after form-type-radio form-item-references-flags\">\n",
       "<input class=\"element-hidden form-radio\" id=\"edit-references-flags-match-any\" name=\"references_flags\" type=\"radio\" value=\"match-any\"/> <label class=\"option\" for=\"edit-references-flags-match-any\">any </label>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-after form-type-radio form-item-references-flags\">\n",
       "<input checked=\"checked\" class=\"element-hidden form-radio\" id=\"edit-references-flags-match-all\" name=\"references_flags\" type=\"radio\" value=\"match-all\"/> <label class=\"option\" for=\"edit-references-flags-match-all\">all </label>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-after form-type-radio form-item-references-flags\">\n",
       "<input class=\"element-hidden form-radio\" id=\"edit-references-flags-match-phrase\" name=\"references_flags\" type=\"radio\" value=\"match-phrase\"/> <label class=\"option\" for=\"edit-references-flags-match-phrase\">phrase </label>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div><div class=\"element-hidden\"><div class=\"form-item form-item-label-before form-type-select form-item-publisher\">\n",
       "<label for=\"edit-publisher\">Publisher </label>\n",
       "<select class=\"element-hidden form-select\" id=\"edit-publisher\" name=\"publisher\"><option selected=\"selected\" value=\"\"> - All Publishers - </option></select>\n",
       "<div class=\"description\">Book publisher name</div>\n",
       "</div>\n",
       "</div></div></fieldset>\n",
       "<fieldset class=\"collapsible form-wrapper\" id=\"edit-citation\"><legend><span class=\"fieldset-legend\">Citation</span></legend><div class=\"fieldset-wrapper\"><div class=\"fieldset-description\">Citation-specific search information</div><div class=\"element-hidden\"><div class=\"form-item form-item-label-before form-type-textfield form-item-pubyear\">\n",
       "<label for=\"edit-pubyear\">Year </label>\n",
       "<input class=\"element-hidden form-text\" id=\"edit-pubyear\" maxlength=\"128\" name=\"pubyear\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\">e.g., 2009</div>\n",
       "</div>\n",
       "</div><div class=\"element-hidden\"><div class=\"form-item form-item-label-before form-type-textfield form-item-volume\">\n",
       "<label for=\"edit-volume\">Volume </label>\n",
       "<input class=\"element-hidden form-text\" id=\"edit-volume\" maxlength=\"128\" name=\"volume\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\">e.g., 20</div>\n",
       "</div>\n",
       "</div><div class=\"element-hidden\"><div class=\"form-item form-item-label-before form-type-textfield form-item-issue\">\n",
       "<label for=\"edit-issue\">Issue </label>\n",
       "<input class=\"element-hidden form-text\" id=\"edit-issue\" maxlength=\"128\" name=\"issue\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\">e.g., 3</div>\n",
       "</div>\n",
       "</div><div class=\"element-hidden\"><div class=\"form-item form-item-label-before form-type-textfield form-item-firstpage\">\n",
       "<label for=\"edit-firstpage\">First page </label>\n",
       "<input class=\"element-hidden form-text\" id=\"edit-firstpage\" maxlength=\"128\" name=\"firstpage\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\">e.g., 29</div>\n",
       "</div>\n",
       "</div><div class=\"form-item form-item-label-before form-type-textfield form-item-doi\">\n",
       "<label for=\"edit-doi\">DOI </label>\n",
       "<input class=\"form-text\" id=\"edit-doi\" maxlength=\"128\" name=\"doi\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "<div class=\"description\">e.g., 10.9999/123XYZ456</div>\n",
       "</div>\n",
       "<div class=\"element-hidden\"><div class=\"form-item form-item-label-before form-type-textfield form-item-isbn\">\n",
       "<label for=\"edit-isbn\">ISBN </label>\n",
       "<input class=\"element-hidden form-text\" id=\"edit-isbn\" maxlength=\"128\" name=\"isbn\" size=\"60\" type=\"text\" value=\"\"/>\n",
       "</div>\n",
       "</div></div></fieldset>\n",
       "<fieldset class=\"collapsible form-wrapper\" id=\"edit-format\"><legend><span class=\"fieldset-legend\">Format Results</span></legend><div class=\"fieldset-wrapper\"><div class=\"form-item form-item-label-before form-type-select form-item-numresults\">\n",
       "<label for=\"edit-numresults--2\">Number of results per page </label>\n",
       "<select class=\"form-select\" id=\"edit-numresults--2\" name=\"numresults\"><option value=\"10\">10</option><option value=\"25\">25</option><option value=\"50\">50</option><option value=\"75\">75</option><option value=\"100\">100</option></select>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-before form-type-radios form-item-sort\">\n",
       "<label for=\"edit-sort--2\">Sort </label>\n",
       "<div class=\"form-radios\" id=\"edit-sort--2\"><div class=\"form-item form-item-label-after form-type-radio form-item-sort\">\n",
       "<input checked=\"checked\" class=\"form-radio\" id=\"edit-sort-relevance-rank\" name=\"sort\" type=\"radio\" value=\"relevance-rank\"/> <label class=\"option\" for=\"edit-sort-relevance-rank\">Best Match </label>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-after form-type-radio form-item-sort\">\n",
       "<input class=\"form-radio\" id=\"edit-sort-publication-date-ascending\" name=\"sort\" type=\"radio\" value=\"publication-date-ascending\"/> <label class=\"option\" for=\"edit-sort-publication-date-ascending\">Oldest First </label>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-after form-type-radio form-item-sort\">\n",
       "<input class=\"form-radio\" id=\"edit-sort-publication-date-descending\" name=\"sort\" type=\"radio\" value=\"publication-date-descending\"/> <label class=\"option\" for=\"edit-sort-publication-date-descending\">Newest First </label>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-before form-type-radios form-item-format-result\">\n",
       "<label for=\"edit-format-result\">Format Results </label>\n",
       "<div class=\"form-radios\" id=\"edit-format-result\"><div class=\"form-item form-item-label-after form-type-radio form-item-format-result\">\n",
       "<input checked=\"checked\" class=\"form-radio\" id=\"edit-format-result-standard\" name=\"format_result\" type=\"radio\" value=\"standard\"/> <label class=\"option\" for=\"edit-format-result-standard\">Standard Format </label>\n",
       "</div>\n",
       "<div class=\"form-item form-item-label-after form-type-radio form-item-format-result\">\n",
       "<input class=\"form-radio\" id=\"edit-format-result-condensed\" name=\"format_result\" type=\"radio\" value=\"condensed\"/> <label class=\"option\" for=\"edit-format-result-condensed\">Condensed Format </label>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div></fieldset>\n",
       "<div class=\"element-hidden\"><div class=\"form-item form-item-label-after form-type-checkbox form-item-exclude-meeting-abstracts\">\n",
       "<input class=\"element-hidden form-checkbox\" id=\"edit-exclude-meeting-abstracts\" name=\"exclude_meeting_abstracts\" type=\"checkbox\" value=\"1\"/> <label class=\"option\" for=\"edit-exclude-meeting-abstracts\">Exclude meeting abstracts </label>\n",
       "</div>\n",
       "</div><input name=\"jcode_option\" type=\"hidden\" value=\"medrxiv biorxiv\"/>\n",
       "<input name=\"form_build_id\" type=\"hidden\" value=\"form-gAJVV-8F9hA2oXpKcWHCOBKK05du9WQuk83FZhsSFcI\"/>\n",
       "<input name=\"form_id\" type=\"hidden\" value=\"highwire_search_form\"/>\n",
       "<div class=\"form-actions form-wrapper\" id=\"edit-actions\"><input class=\"form-submit\" id=\"edit-submit--2\" name=\"op\" type=\"submit\" value=\"Search\"/></div></div></form> </div>\n",
       "</div>\n",
       "<div class=\"panel-separator\"></div><div class=\"panel-pane pane-highwire-search-results active\" id=\"hw-advance-search-result\">\n",
       "<div class=\"pane-content\">\n",
       "<div class=\"highwire-search-results highwire-article-citation-list no-show-abstract\">No results found</div> </div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "<div class=\"sidebar-right-wrapper grid-10 omega\">\n",
       "<div class=\"panel-panel panel-region-sidebar-right\">\n",
       "<div class=\"inside\"><div class=\"panel-pane pane-panels-mini pane-biorxiv-save-citation-links\">\n",
       "<div class=\"pane-content\">\n",
       "<div class=\"panel-display panel-1col clearfix\" id=\"mini-panel-biorxiv_save_citation_links\">\n",
       "<div class=\"panel-panel panel-col\">\n",
       "<div><div class=\"panel-pane pane-report-view-link\">\n",
       "<div class=\"pane-content\">\n",
       "<a class=\"link-save-citation-report link-icon\" href=\"/highwire-save-citation/manage\" id=\"link-save-citation-report\"><i class=\"icon-bookmark\"></i> <span class=\"title\">View Selected Citations (0)</span></a> </div>\n",
       "</div>\n",
       "<div class=\"panel-separator\"></div><div class=\"panel-pane pane-save-all-action-link\">\n",
       "<div class=\"pane-content\">\n",
       "<ul class=\"inline hw-link-save-unsave-all-citation\"><li class=\"save-all-link first\"><a class=\"link-save-all-citation-save link-icon\" href=\"/highwire-save-citation/save-all//nojs\" title=\"Add All Citations\"><i class=\"icon-plus\"></i> <span class=\"title\">Add All Citations</span></a></li><li class=\"unsave-all-link last\"><a class=\"link-save-all-citation-unsave link-icon\" href=\"/highwire-save-citation/unsave-all//nojs\" title=\"Remove All Citations\"><i class=\"icon-ok\"></i> <span class=\"title\">Remove All Citations</span></a></li></ul> </div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "<div class=\"panel-separator\"></div><div class=\"panel-pane pane-panels-mini pane-biorxiv-subject-collections\">\n",
       "<div class=\"pane-content\">\n",
       "<div class=\"panel-flexible panels-flexible-8 clearfix\" id=\"mini-panel-biorxiv_subject_collections\">\n",
       "<div class=\"panel-flexible-inside panels-flexible-8-inside\">\n",
       "<div class=\"panels-flexible-region panels-flexible-region-8-center panels-flexible-region-first panels-flexible-region-last\">\n",
       "<div class=\"inside panels-flexible-region-inside panels-flexible-region-8-center-inside panels-flexible-region-inside-first panels-flexible-region-inside-last\">\n",
       "<div class=\"panel-pane pane-snippet\">\n",
       "<div class=\"pane-content\">\n",
       "<div class=\"snippet biorxiv-subject-areas-table-title\" id=\"biorxiv-subject-areas-table-title\">\n",
       "<div class=\"snippet-content\">\n",
       "<b>Subject Areas</b> </div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "<div class=\"panel-separator\"></div><div class=\"panel-pane pane-snippet\">\n",
       "<div class=\"pane-content\">\n",
       "<div class=\"snippet biorxiv-subject-areas-view-papers\" id=\"biorxiv-subject-areas-view-papers\">\n",
       "<div class=\"snippet-content\">\n",
       "<a href=\"/content/early/recent\"><strong>All Articles</strong></a> </div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "<div class=\"panel-separator\"></div><div class=\"panel-pane pane-highwire-subject-collections\">\n",
       "<div class=\"pane-content\">\n",
       "<ul class=\"collection highwire-list-expand\" id=\"collection\"><li class=\"outer collection depth-2 child first\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/animal-behavior-and-cognition\">Animal Behavior and Cognition</a> <span class=\"article-count\">(1337)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/biochemistry\">Biochemistry</a> <span class=\"article-count\">(2166)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/bioengineering\">Bioengineering</a> <span class=\"article-count\">(1480)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/bioinformatics\">Bioinformatics</a> <span class=\"article-count\">(8785)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/biophysics\">Biophysics</a> <span class=\"article-count\">(3437)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/cancer-biology\">Cancer Biology</a> <span class=\"article-count\">(2564)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/cell-biology\">Cell Biology</a> <span class=\"article-count\">(3635)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/clinical-trials\">Clinical Trials</a> <span class=\"article-count\">(134)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/developmental-biology\">Developmental Biology</a> <span class=\"article-count\">(2326)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/ecology\">Ecology</a> <span class=\"article-count\">(3591)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/epidemiology\">Epidemiology</a> <span class=\"article-count\">(2007)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/evolutionary-biology\">Evolutionary Biology</a> <span class=\"article-count\">(6254)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/genetics\">Genetics</a> <span class=\"article-count\">(4678)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/genomics\">Genomics</a> <span class=\"article-count\">(5890)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/immunology\">Immunology</a> <span class=\"article-count\">(1829)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/microbiology\">Microbiology</a> <span class=\"article-count\">(6073)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/molecular-biology\">Molecular Biology</a> <span class=\"article-count\">(2348)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/neuroscience\">Neuroscience</a> <span class=\"article-count\">(15038)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/paleontology\">Paleontology</a> <span class=\"article-count\">(106)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/pathology\">Pathology</a> <span class=\"article-count\">(363)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/pharmacology-and-toxicology\">Pharmacology and Toxicology</a> <span class=\"article-count\">(620)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/physiology\">Physiology</a> <span class=\"article-count\">(889)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/plant-biology\">Plant Biology</a> <span class=\"article-count\">(2122)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/scientific-communication-and-education\">Scientific Communication and Education</a> <span class=\"article-count\">(563)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/synthetic-biology\">Synthetic Biology</a> <span class=\"article-count\">(746)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/systems-biology\">Systems Biology</a> <span class=\"article-count\">(2438)</span></div></li>\n",
       "<li class=\"outer collection depth-2 child last\"><div class=\"data-wrapper\"><a class=\"\" data-hide-link-title=\"0\" data-icon-position=\"\" href=\"/collection/zoology\">Zoology</a> <span class=\"article-count\">(368)</span></div></li>\n",
       "</ul> </div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div> <!-- /.panel-row-wrapper -->\n",
       "</div>\n",
       "</div>\n",
       "</div>\n",
       "</div> </div>\n",
       "</div> </div>\n",
       "</section>\n",
       "</div> <div class=\"region region-page-bottom\" id=\"region-page-bottom\">\n",
       "<div class=\"region-inner region-page-bottom-inner\">\n",
       "</div>\n",
       "</div><script src=\"https://www.biorxiv.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.js\" type=\"text/javascript\"></script>\n",
       "<script type=\"text/javascript\">\n",
       "<!--//--><![CDATA[//><!--\n",
       "function euCookieComplianceLoadScripts() {}\n",
       "//--><!]]>\n",
       "</script>\n",
       "<script type=\"text/javascript\">\n",
       "<!--//--><![CDATA[//><!--\n",
       "var eu_cookie_compliance_cookie_name = \"\";\n",
       "//--><!]]>\n",
       "</script>\n",
       "<script src=\"//d33xdlntwy0kbs.cloudfront.net/cshl_custom.js\" type=\"text/javascript\"></script>\n",
       "<script src=\"https://www.biorxiv.org/sites/default/files/advagg_js/js__BmqjBnkz3MgYCAoc25s1lDRMEjLhC3mEPVonUFIHi08__Unwv5-ZIuHBfFwytsjEx1niBVJ7n1T4lPws7VrkHXM4__j1V7SlgkFe90DR_fdU0CK8Y9Tuf1qmo7Y4afiWDHAhY.js\" type=\"text/javascript\"></script>\n",
       "</body>\n",
       "</html>"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Let's see what html output looks like\n",
    "text = bs(resp.text)\n",
    "text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://biorxiv.org/search/jcode%3Abiorxiv%20limit_from%3A2019-08-31%20limit_to%3A2019-09-10%20numresults%3A1000%20sort%3Arelevance-rank%20format_result%3Astandard\n"
     ]
    }
   ],
   "source": [
    "# Define the URL with start/stop dates\n",
    "search_term = \"\"\n",
    "url_base = \"https://biorxiv.org/search/{}\".format(search_term)\n",
    "url_params = \"jcode%3Abiorxiv%20limit_from%3A\"+TimeBefore+\"%20limit_to%3A\"+today+\"%20numresults%3A\"+str(n_results)+\"%20sort%3Arelevance-rank%20format_result%3Astandard\".format(\n",
    "    n_results)\n",
    "\n",
    "url = url_base + url_params\n",
    "print(url)\n",
    "\n",
    "#https://www.biorxiv.org/search/jcode%3Abiorxiv%20limit_from%3A2019-05-07%20limit_to%3A2019-05-17%20numresults%3A10%20sort%3Arelevance-rank%20format_result%3Astandard"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Scraper\n",
    "parser = 'html.parser'\n",
    "\n",
    "all_articles = []\n",
    "all_authors = []\n",
    "this_url = url \n",
    "resp = requests.post(this_url)\n",
    "time.sleep(100) #takes ages to load 1000 articles\n",
    "html = bs(resp.text,parser)\n",
    "print(\"Souping complete.\")\n",
    "        \n",
    "# Collect the articles in the result in a list\n",
    "articleData = [] # The article listings for this page, indexed by DOI\n",
    "articles = html.find_all('li', attrs={'class': 'search-result'})\n",
    "for article in articles:\n",
    "    # Get the item header \n",
    "    citation = article.find('div', attrs={'class': 'highwire-article-citation'})\n",
    "    master_version = citation.get('data-pisa-master')\n",
    "    version = citation.get('data-pisa')\n",
    "    atom_path = citation.get('data-apath')\n",
    "    \n",
    "\n",
    "    # Get abstract\n",
    "    article_url = \"https://www.biorxiv.org/content/\"+atom_path\n",
    "    art_resp = requests.post(article_url)\n",
    "    article_html = bs(art_resp.text, parser)\n",
    "    abstract = str(article_html.find('meta', attrs={'name': 'DC.Description'}))\n",
    "    abstract = abstract.replace('<meta content=\"', \"\").replace('>\"', \"\")\n",
    "\n",
    "    # Get the DOI\n",
    "    doispan = article.find('span', attrs={'class': 'highwire-cite-metadata-doi'})\n",
    "    doi = doispan.text.strip().replace('doi: https://', '')\n",
    " \n",
    "    # Get the title info\n",
    "    title = article.find('span', attrs={'class': 'highwire-cite-title'})\n",
    "    title = title.text.strip().replace(\"\\n\", \"\")\n",
    " \n",
    "    # Now collect author information\n",
    "    authors = article.find_all('span', attrs={'class': 'highwire-citation-author'})\n",
    "    all_authors = []\n",
    "    for author in authors:\n",
    "        all_authors.append(author.text)\n",
    " \n",
    "    author_list = ', '.join(all_authors)\n",
    "    #outdata = pd.DataFrame(title, atom_path, author_list, version, columns=['title', 'atom_path', 'author_list', 'version'])\n",
    "    outdata = [title, atom_path, author_list, abstract, version]\n",
    " \n",
    "    articleData.append(outdata)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Collect these into DataFrames\n",
    "df_biorxiv = pd.DataFrame(articleData, columns=['title', 'atom_path', 'authors', 'abstract','version'])\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(1000, 5)"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_biorxiv.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>atom_path</th>\n",
       "      <th>authors</th>\n",
       "      <th>abstract</th>\n",
       "      <th>version</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Draft genome sequences of Hirudo medicinalis a...</td>\n",
       "      <td>/biorxiv/early/2019/09/10/357681.atom</td>\n",
       "      <td>Vladislav V. Babenko, Oleg V. Podgorny, Valent...</td>\n",
       "      <td>None</td>\n",
       "      <td>biorxiv;357681v2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ATG13 dynamics in non-selective autophagy and ...</td>\n",
       "      <td>/biorxiv/early/2019/09/10/370114.atom</td>\n",
       "      <td>Piero Dalle Pezze, Eleftherios Karanasios, Var...</td>\n",
       "      <td>None</td>\n",
       "      <td>biorxiv;370114v2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>BET inhibition prevents aberrant RUNX1 and ERG...</td>\n",
       "      <td>/biorxiv/early/2019/09/10/762781.atom</td>\n",
       "      <td>Jisha Antony, Gregory Gimenez, Terry Taylor, U...</td>\n",
       "      <td>None</td>\n",
       "      <td>biorxiv;762781v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Skp1 Dimerization Conceals its F-box Protein B...</td>\n",
       "      <td>/biorxiv/early/2019/09/10/764126.atom</td>\n",
       "      <td>Hyun W. Kim, Alexander Eletsky, Karen J. Gonza...</td>\n",
       "      <td>None</td>\n",
       "      <td>biorxiv;764126v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Co-ordinated Ras and Rac activity shapes macro...</td>\n",
       "      <td>/biorxiv/early/2019/09/10/763748.atom</td>\n",
       "      <td>Catherine M Buckley, Henderikus Pots, Aurelie ...</td>\n",
       "      <td>None</td>\n",
       "      <td>biorxiv;763748v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>DeepCryoPicker: Fully Automated Deep Neural Ne...</td>\n",
       "      <td>/biorxiv/early/2019/09/10/763839.atom</td>\n",
       "      <td>Adil Al-Azzawi, Anes Ouadou, Max R Highsmith, ...</td>\n",
       "      <td>None</td>\n",
       "      <td>biorxiv;763839v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Metabolite therapy guided by liquid biopsy pro...</td>\n",
       "      <td>/biorxiv/early/2019/09/10/764100.atom</td>\n",
       "      <td>Katherine Wert, Gabriel Velez, Kanchustambham ...</td>\n",
       "      <td>None</td>\n",
       "      <td>biorxiv;764100v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Expression Profile of Circular RNAs in Epicard...</td>\n",
       "      <td>/biorxiv/early/2019/09/10/764266.atom</td>\n",
       "      <td>Xinchun Yang, Meili Zheng, Lei Zhao</td>\n",
       "      <td>None</td>\n",
       "      <td>biorxiv;764266v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Animal biosynthesis of complex polyketides in ...</td>\n",
       "      <td>/biorxiv/early/2019/09/10/764225.atom</td>\n",
       "      <td>Joshua P. Torres, Zhenjian Lin, Jaclyn M. Wint...</td>\n",
       "      <td>None</td>\n",
       "      <td>biorxiv;764225v1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>FARFAR2: Improved de novo Rosetta prediction o...</td>\n",
       "      <td>/biorxiv/early/2019/09/10/764449.atom</td>\n",
       "      <td>Rhiju Das, Andrew M Watkins</td>\n",
       "      <td>None</td>\n",
       "      <td>biorxiv;764449v1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                               title  \\\n",
       "0  Draft genome sequences of Hirudo medicinalis a...   \n",
       "1  ATG13 dynamics in non-selective autophagy and ...   \n",
       "2  BET inhibition prevents aberrant RUNX1 and ERG...   \n",
       "3  Skp1 Dimerization Conceals its F-box Protein B...   \n",
       "4  Co-ordinated Ras and Rac activity shapes macro...   \n",
       "5  DeepCryoPicker: Fully Automated Deep Neural Ne...   \n",
       "6  Metabolite therapy guided by liquid biopsy pro...   \n",
       "7  Expression Profile of Circular RNAs in Epicard...   \n",
       "8  Animal biosynthesis of complex polyketides in ...   \n",
       "9  FARFAR2: Improved de novo Rosetta prediction o...   \n",
       "\n",
       "                               atom_path  \\\n",
       "0  /biorxiv/early/2019/09/10/357681.atom   \n",
       "1  /biorxiv/early/2019/09/10/370114.atom   \n",
       "2  /biorxiv/early/2019/09/10/762781.atom   \n",
       "3  /biorxiv/early/2019/09/10/764126.atom   \n",
       "4  /biorxiv/early/2019/09/10/763748.atom   \n",
       "5  /biorxiv/early/2019/09/10/763839.atom   \n",
       "6  /biorxiv/early/2019/09/10/764100.atom   \n",
       "7  /biorxiv/early/2019/09/10/764266.atom   \n",
       "8  /biorxiv/early/2019/09/10/764225.atom   \n",
       "9  /biorxiv/early/2019/09/10/764449.atom   \n",
       "\n",
       "                                             authors abstract  \\\n",
       "0  Vladislav V. Babenko, Oleg V. Podgorny, Valent...     None   \n",
       "1  Piero Dalle Pezze, Eleftherios Karanasios, Var...     None   \n",
       "2  Jisha Antony, Gregory Gimenez, Terry Taylor, U...     None   \n",
       "3  Hyun W. Kim, Alexander Eletsky, Karen J. Gonza...     None   \n",
       "4  Catherine M Buckley, Henderikus Pots, Aurelie ...     None   \n",
       "5  Adil Al-Azzawi, Anes Ouadou, Max R Highsmith, ...     None   \n",
       "6  Katherine Wert, Gabriel Velez, Kanchustambham ...     None   \n",
       "7                Xinchun Yang, Meili Zheng, Lei Zhao     None   \n",
       "8  Joshua P. Torres, Zhenjian Lin, Jaclyn M. Wint...     None   \n",
       "9                        Rhiju Das, Andrew M Watkins     None   \n",
       "\n",
       "            version  \n",
       "0  biorxiv;357681v2  \n",
       "1  biorxiv;370114v2  \n",
       "2  biorxiv;762781v1  \n",
       "3  biorxiv;764126v1  \n",
       "4  biorxiv;763748v1  \n",
       "5  biorxiv;763839v1  \n",
       "6  biorxiv;764100v1  \n",
       "7  biorxiv;764266v1  \n",
       "8  biorxiv;764225v1  \n",
       "9  biorxiv;764449v1  "
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_biorxiv.head(10)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Now scrape arxiv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "http://export.arxiv.org/oai2?verb=ListRecords&from=2019-08-31&until=2019-09-10&metadataPrefix=arXiv&set=q-bio\n",
      "fetching up to  1000 records...\n",
      "fetching is completed in 4.0 seconds.\n",
      "Total number of records 155\n"
     ]
    }
   ],
   "source": [
    "#check that you have arxivscraper.py in directory or have installed it\n",
    "import arxivscraper as ax\n",
    "\n",
    "scraper = ax.Scraper(category='q-bio',date_from=TimeBefore,date_until=today,t=20 \n",
    "                    )\n",
    "output = scraper.scrape()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "cols = ('id', 'title', 'categories', 'abstract', 'doi', 'created', 'updated', 'authors')\n",
    "df_arxiv = pd.DataFrame(output,columns=cols)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>title</th>\n",
       "      <th>categories</th>\n",
       "      <th>abstract</th>\n",
       "      <th>doi</th>\n",
       "      <th>created</th>\n",
       "      <th>updated</th>\n",
       "      <th>authors</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1709.09236</td>\n",
       "      <td>large scale evaluation of differences between ...</td>\n",
       "      <td>q-bio.mn q-bio.gn</td>\n",
       "      <td>dendrograms are a way to represent evolutionar...</td>\n",
       "      <td>10.1371/journal.pone.0221631</td>\n",
       "      <td>2017-09-26</td>\n",
       "      <td></td>\n",
       "      <td>[gamermann, montagud, conejero, de córdoba, ur...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1710.02431</td>\n",
       "      <td>emerging whole-cell modeling principles and me...</td>\n",
       "      <td>q-bio.qm</td>\n",
       "      <td>whole-cell computational models aim to predict...</td>\n",
       "      <td>10.1016/j.copbio.2017.12.013</td>\n",
       "      <td>2017-10-06</td>\n",
       "      <td>2017-12-08</td>\n",
       "      <td>[goldberg, szigeti, chew, sekar, roth, karr]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1803.02136</td>\n",
       "      <td>limit distribution of the quartet balance inde...</td>\n",
       "      <td>q-bio.pe math.pr</td>\n",
       "      <td>this paper builds up on t. martinez-coronado, ...</td>\n",
       "      <td></td>\n",
       "      <td>2018-03-06</td>\n",
       "      <td>2019-08-30</td>\n",
       "      <td>[bartoszek]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1803.05404</td>\n",
       "      <td>coupling the yoccoz-birkeland population model...</td>\n",
       "      <td>math.ds q-bio.pe</td>\n",
       "      <td>we propose a new model for the time evolution ...</td>\n",
       "      <td>10.1088/1361-6544/ab0eb7</td>\n",
       "      <td>2018-03-14</td>\n",
       "      <td>2018-05-09</td>\n",
       "      <td>[arlot, marmi, papini]</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1803.08362</td>\n",
       "      <td>consciousness: from the perspective of the dyn...</td>\n",
       "      <td>q-bio.nc</td>\n",
       "      <td>beings, animate or inanimate, are dynamical sy...</td>\n",
       "      <td></td>\n",
       "      <td>2018-03-04</td>\n",
       "      <td></td>\n",
       "      <td>[schad]</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "           id                                              title  \\\n",
       "0  1709.09236  large scale evaluation of differences between ...   \n",
       "1  1710.02431  emerging whole-cell modeling principles and me...   \n",
       "2  1803.02136  limit distribution of the quartet balance inde...   \n",
       "3  1803.05404  coupling the yoccoz-birkeland population model...   \n",
       "4  1803.08362  consciousness: from the perspective of the dyn...   \n",
       "\n",
       "          categories                                           abstract  \\\n",
       "0  q-bio.mn q-bio.gn  dendrograms are a way to represent evolutionar...   \n",
       "1           q-bio.qm  whole-cell computational models aim to predict...   \n",
       "2   q-bio.pe math.pr  this paper builds up on t. martinez-coronado, ...   \n",
       "3   math.ds q-bio.pe  we propose a new model for the time evolution ...   \n",
       "4           q-bio.nc  beings, animate or inanimate, are dynamical sy...   \n",
       "\n",
       "                            doi     created     updated  \\\n",
       "0  10.1371/journal.pone.0221631  2017-09-26               \n",
       "1  10.1016/j.copbio.2017.12.013  2017-10-06  2017-12-08   \n",
       "2                                2018-03-06  2019-08-30   \n",
       "3      10.1088/1361-6544/ab0eb7  2018-03-14  2018-05-09   \n",
       "4                                2018-03-04               \n",
       "\n",
       "                                             authors  \n",
       "0  [gamermann, montagud, conejero, de córdoba, ur...  \n",
       "1       [goldberg, szigeti, chew, sekar, roth, karr]  \n",
       "2                                        [bartoszek]  \n",
       "3                             [arlot, marmi, papini]  \n",
       "4                                            [schad]  "
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_arxiv.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(155, 8)"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_arxiv.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "#save as pickle for future use\n",
    "#make sure you have data folders for this\n",
    "import pickle\n",
    "df_arxiv.to_pickle('arxiv_nobody'+today+'.pkl')\n",
    "df_biorxiv.to_pickle('biorxiv_nobody'+today+'.pkl')\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Start here if you have previously scraped and pickled the articles that you want\n",
    "\n",
    "## From here, we have a dataframe of arxiv articles and we want to grab the pdf body from."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "#to read pickle in from previous scrape:\n",
    "import pandas as pd\n",
    "import datetime\n",
    "today = datetime.datetime.now().date().strftime(\"%Y-%m-%d\")\n",
    "#today = \"2019-05-17\"\n",
    "df_arxiv=pd.read_pickle('arxiv_nobody'+today+'.pkl')\n",
    "df_biorxiv=pd.read_pickle('biorxiv_nobody'+today+'.pkl')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "#function to download arxiv file as pdf temp file\n",
    "import requests\n",
    "import pickle\n",
    "\n",
    "def download_file(download_url):\n",
    "    print(download_url)\n",
    "    agent = {'User-Agent': 'Mozilla/5.0'}\n",
    "    response = requests.get(download_url, headers=agent)\n",
    "    temp_filename = 'document_placeholder.pdf'\n",
    "    with open(temp_filename, 'wb') as file:\n",
    "        pickle.dump(response.content, file, protocol=pickle.HIGHEST_PROTOCOL)\n",
    "    return temp_filename"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## import body of a pdf as an example"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/10/763748.full.pdf\n"
     ]
    }
   ],
   "source": [
    "from PdfConverter import PdfConverter\n",
    "# use --> !pip3 install pdfminer.six\n",
    "# read in pdf from arxiv url\n",
    "\n",
    "#example running this on the article in row 4 of the dataframe\n",
    "#pdf_filename=download_file(\"https://arxiv.org/pdf/%s.pdf\" %df_arxiv.id.loc[10])\n",
    "pdf_filename=download_file(df_biorxiv.atom_path.loc[4].replace('/biorxiv/', 'https://www.biorxiv.org/content/biorxiv/').replace( '.atom','')+'.full.pdf')\n",
    "\n",
    "#Use class PdfConverter\n",
    "pdfConverted = PdfConverter(file_path=pdf_filename)\n",
    "text_out = pdfConverted.convert_pdf_to_txt()\n",
    "\n",
    "#replace linebreaks with spaces in doc\n",
    "text_out = text_out.replace('\\n',' ')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   Co-ordinated Ras and Rac activity shapes macropinocytic cups  and enables phagocytosis of geometrically diverse bacteria   1\t 2\t 3\t \t 4\t Catherine\tM.\tBuckley1,\tHenderikus\tPots2,\tAurelie\tGueho3$,\tBen\tA.\tPhillips1,\tBernd\t 5\t Gilsbach4,\tAnton\tNikolaev1,\tThierry\tSoldati3,\tAndrew\tJ.\t\tParnell5,\tArjan\tKortholt2\t 6\t and\tJason\tS.\tKing1*\t 7\t \t 8\t 1Department\tof\tBiomedical\tSciences,\tUniversity\tof\tSheffield,\tSheffield,\tUK.\t\t 2Department\tof\tCell\tBiochemistry,\tUniversity\tof\tGroningen,\tGroningen,\t 9\t Netherlands.\t 10\t 3Department\tof\tBiochemistry,\tFaculty\tof\tSciences,\tSciences\tII,\tUniversity\tof\t 11\t 12\t Geneva,\tGeneva,\tSwitzerland\t 4German\tCentre\tfor\tNeurodegenerative\tDiseases,\tTübingen,\tGermany\t 13\t 14\t 5Department\tof\tPhysics\tand\tAstronomy,\tUniversity\tof\tSheffield,\tSheffield,\tUK\t $\tcurrent\taddress:\tFish\tPhysiology\tand\tGenomics\tInstitute,\tINRA,\tRennes,\tFrance\t 15\t 16\t \t \t *Corresponding\tauthor:\tJason.King@sheffield.ac.uk\t 17\t  \t  1\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   Abstract   1\t 2\t Engulfment\tof\textracellular\tmaterial\tby\tphagocytosis\tor\tmacropinocytosis\t 3\t depends\ton\tthe\tability\tof\tcells\tto\tgenerate\tspecialised\tcup\tshaped\tprotrusions.\t\t 4\t To\teffectively\tcapture\tand\tinternalise\ttheir\ttargets,\tthese\tcups\tare\torganised\tinto\t 5\t a\tring\tor\truffle\tof\tactin-driven\tprotrusion\tencircling\ta\tstatic\tinterior\tdomain.\t 6\t These\tfunctional\tdomains\tdepend\ton\tthe\tcombined\tactivities\tof\tmultiple\tRas\tand\t 7\t Rho\tfamily\tsmall\tGTPases,\tbut\thow\ttheir\tactivities\tare\tintegrated\tand\t 8\t differentially\tregulated\tover\tspace\tand\ttime\tis\tunknown.\tHere,\twe\tshow\tthat\tthe\t 9\t multidomain\tprotein\tRGBARG\t(RCC1,\tRhoGEF,\tBAR\tand\tRasGAP-containing\t amoeba\tDictyostelium\tdiscoideum\tcoordinates\tRas\tand\tRac\tactivity\tusing\tthe\t 10\t 11\t protein).\tWe\tfind\tRGBARG\tuses\ta\ttripartite\tmechanism\tof\tRas,\tRac\tand\t phospholipid\tinteractions\tto\tlocalise\tat\tthe\tprotruding\tedge\tand\tinterface\twith\t 12\t 13\t the\tinterior\tof\tboth\tmacropinocytic\tand\tphagocytic\tcups.\tThere,\tRGBARG\tshapes\t the\tprotrusion\tby\tdriving\tRac\tactivation\tat\tthe\trim\twhilst\tsuppressing\texpansion\t 14\t of\tthe\tactive\tRas\tinterior\tdomain.\t\tConsequently,\tcells\tlacking\tRGBARG\tform\t 15\t 16\t enlarged,\tflat\tinterior\tdomains\tunable\tto\tform\tlarge\tmacropinosomes.\tDuring\t phagocytosis,\twe\tfind\tthat\tdisruption\tof\tRGBARG\tcauses\ta\tgeometry-specific\t 17\t 18\t defect\tin\tengulfing\trod-shaped\tbacteria\tand\tellipsoidal\tbeads.\tThis\tdemonstrates\t 19\t more\tcomplex\tshapes\tand\tthus\tthe\tfull\tphysiological\trange\tof\tmicrobes,\tand\thow\t the\timportance\tof\tco-ordinating\tsmall\tGTPase\tactivities\tduring\tengulfment\tof\t 20\t this\tis\tachieved\tin\ta\tmodel\tprofessional\tphagocyte.\t 21\t \t  \t  2\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   Introduction   1\t 2\t The\tcapture\tand\tengulfment\tof\textracellular\tmaterial\tserves\ta\tnumber\tof\t 3\t microbes\tor\tapoptotic\tcells\tby\tphagocytic\timmune\tcells,\tbut\tthe\tengulfment\tof\t important\tcellular\tfunctions.\tBest\tunderstood\tis\tthe\tclearance\tof\tpathogenic\t 4\t 5\t fluid\tby\tthe\trelated\tprocess\tof\tmacropinocytosis\talso\tplays\timportant\tfunctions\t 6\t by\tallowing\tcells\tto\tcapture\tantigens\tor\tother\tfactors\tfrom\ttheir\tenvironment\t such\tas\tnutrients\tto\tsupport\tgrowth\t(Bloomfield\tand\tKay,\t2016;\tCommisso\tet\tal.,\t 7\t 8\t 2013;\tNorbury\tet\tal.,\t1995;\tSallusto\tet\tal.,\t1995;\tSwanson\tand\tKing,\t2019).\t\t \t 9\t 10\t within\ta\tvesicle.\tThis\tcan\tbe\tachieved\tby\tseveral\tmechanisms,\tbut\tthe\tbest\t To\tcapture\textracellular\tmaterial,\tcells\tmust\tencircle\tand\tisolate\ttheir\ttarget\t 11\t understood\tand\tevolutionarily\twidespread\tinvolves\tthe\textension\tof\ta\tcircular\t 12\t 13\t cup\tor\truffle-shaped\tprotrusion\tfrom\tthe\tcell\tsurface\tto\tenwrap\tthe\ttarget\tand\t close\tto\tinternalise\tit\t(Buckley\tand\tKing,\t2017;\tKaplan,\t1977;\tSwanson,\t2008;\t 14\t Veltman\tet\tal.,\t2016).\tWhilst\tmany\tcomponents\tof\tcup\tformation\thave\tbeen\t 15\t 16\t Here\twe\tdescribe\ta\tnovel\tmechanism\tused\tby\tthe\tamoebae\tDictyostelium\t identified,\thow\tthey\tare\tco-ordinated\tin\tspace\tand\ttime\tis\tpoorly\tunderstood.\t 17\t 18\t discoideum\tto\tintegrate\tdifferent\tsignaling\telements\tand\tform\tcomplex-shaped\t protrusions\tthat\tefficiently\tmediate\tengulfment.\t 19\t \t 20\t Macropinocytic\tand\tphagocytic\tprotrusions\tare\tformed\tby\tlocalised\tactin\t 21\t polymerisation\tat\tthe\tplasma\tmembrane,\tusing\tmuch\tof\tthe\tsame\tmachinery\tthat\t 22\t 23\t generates\tpseudopods\tand\tlamellipodia\tduring\tcell\tmigration\t(King\tand\tKay,\t 2019;\tSwanson,\t2008).\t\tWhilst\tmigratory\tprotrusions\tonly\tneed\tthe\tcell\tto\tdefine\t 24\t 25\t a\tsimple\tpatch\tof\tactin\tpolymerisation,\tforming\ta\tcup\trequires\ta\thigher\tlevel\tof\t organisation,\twith\tthe\tprotrusive\tactivity\trestricted\tto\ta\tring\tencircling\ta\tstatic\t 26\t 27\t interior\tdomain.\tDuring\tphagocytosis\tthis\tis\taided\tby\tthe\tpresence\tof\ta\tparticle\tto\t act\tas\ta\tphysical\tscaffold\tand\tlocally\tactivate\treceptors.\tThese\tinteractions\tare\t 28\t proposed\tto\tguide\tengulfment\tby\ta\tzippering\tmechanism\t(Griffin\tet\tal.,\t1975;\t 29\t 30\t Tollis\tet\tal.,\t2010).\tHowever,\tmacropinocytic\tcups\tself-organise\twith\tan\talmost\t identical\tstructure\tin\tthe\tabsence\tof\tany\texternal\tspatial\tcues\t(Veltman\tet\tal.,\t 31\t 32\t 2016).\tCup\tformation\tcan\ttherefore\toccur\tspontaneously\tby\tthe\tintrinsic\t dynamics\tof\tthe\tunderlying\tsignaling.\t\t 33\t \t  3\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   1\t \t 2\t Recent\tstudies\tin\tDictyostelium\tproposed\ta\tmodel\twhereby\tthe\tcup\tinterior\tis\t 3\t defined\tby\tspontaneous\tlocalised\tactivation\tof\tthe\tsmall\tGTPase\tRas\tand\t 4\t consequent\taccumulation\tof\tthe\tphospholipid\tPIP3\t(Veltman\tet\tal.,\t2016).\tThis\t 5\t patch\tappears\tto\trestrict\tactin\tpolymerisation\tto\tits\tperiphery\tto\tcreate\ta\t 6\t may\tdepend\ton\tthe\tactivity\tof\tthe\tPIP3-activated\tProtein\tkinase\tB/Akt\t(Williams\t protrusive\tring.\t\tHow\tthis\tis\tachieved\tis\tunknown,\tbut\tin\tat\tleast\tDictyostelium\tit\t 7\t et\tal.,\t2019).\tBoth\tactive\tRas\tand\tPIP3\talso\taccumulate\tat\tcups\tin\tmammalian\t 8\t 9\t cells\t(Araki\tet\tal.,\t2007;\tMarshall\tet\tal.,\t2001;\tVieira\tet\tal.,\t2001)\tand\tRas\t activation\tis\tsufficient\tto\tdrive\truffling\tand\tmacropinocytosis\tin\tcancer\tcells\t(Bar- 10\t 11\t Sagi\tand\tFeramisco,\t1986;\tCommisso\tet\tal.,\t2013).\tPI3K\tinhibition\talso\t completely\tblocks\tmacropinocytosis\t(Amyere\tet\tal.,\t2000;\tAraki\tet\tal.,\t1996;\t 12\t Hoeller\tet\tal.,\t2013;\tVeltman\tet\tal.,\t2014)\tas\twell\tas\tphagocytosis\tof\tlarge\t 13\t 14\t particles\tby\tmacrophages\t(Araki\tet\tal.,\t1996;\tCox\tet\tal.,\t1999;\tSchlam\tet\tal.,\t 2015).\tRas\tand\tPIP3\ttherefore\tplay\ta\tgeneral\trole\tin\tmacropinosome\tand\t 15\t 16\t phagosome\torganisation\tacross\tevolution.\t 17\t Other\tsmall\tGTPases\tare\talso\tinvolved\tin\tcup\tformation.\tActive\tRac1\toverlaps\t \t 18\t with\tRas\tactivity\tin\tthe\tcup\tinterior\tin\tboth\tmacrophages\tand\tDictyostelium\t 19\t (Hoppe\tand\tSwanson,\t2004;\tVeltman\tet\tal.,\t2016).\tRac1\tis\ta\tdirect\tactivator\tof\t 20\t 21\t the\tSCAR/WAVE\tcomplex,\twhich\tdrives\tactivation\tof\tactin\tpolymerisation\tvia\tthe\t ARP2/3\tcomplex\t(Eden\tet\tal.,\t2002;\tMachesky\tand\tInsall,\t1998).\tConsistent\twith\t 22\t 23\t this,\tRac1\tis\trequired\tfor\tmacropinosome\tformation\tin\tdendritic\tcells\t(West\tet\t 24\t macropinocytosis\tin\tmacrophages\t(Fujii\tet\tal.,\t2013).\tExpression\tof\t al.,\t2000)\tand\toptogenetic\tRac1\tactivation\tis\tsufficient\tto\tdrive\truffling\tand\t 25\t constitutively\tactive\tRac1\talso\tleads\tto\texcessive\tactin\tat\tmacropinocytic\tcups\tin\t 26\t Dictyostelium\t(Dumontier\tet\tal.,\t2000).\tTherefore,\twhilst\tRas\tappears\tto\tdefine\t 27\t 28\t the\tcup\tinterior,\tRac1\tis\timportant\tfor\tregulating\tactin\tprotrusions,\tas\tit\tis\tdoes\t during\tcell\tmigration.\t\t 29\t 30\t \t The\tpresence\tof\tactive\tRac1\tthroughout\tthe\tcup\tinterior\tis\tat\todds\twith\tthe\t 31\t tightly\trestricted\tSCAR/WAVE\tactivity\tand\tprotrusion\tat\tthe\textending\trim\t 32\t 33\t (Veltman\tet\tal.,\t2016).\tA\tfurther\tlayer\tof\tregulation\tmust\ttherefore\texist.\tThis\tis\t \t  4\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   likely\tprovided\tby\tthe\tsmall\tGTPase\tCDC42\twhich\tis\talso\trequired\tfor\tFc-g- receptor\tmediated\tphagocytosis\tand collaborates with Rac1 during engulfment   of large particles (Caron and Hall, 1998; Cox et al., 1997; Massol et al., 1998;  Schlam et al., 2015). In contrast to Rac1, active CDC42 is restricted to the  protrusive cup rim in macrophages indicating differential regulation and  functionality (Hoppe and Swanson, 2004). In Dictyostelium however, no clear  CDC42 orthologue has been identified.     1\t 2\t 3\t 4\t 5\t 6\t 7\t 8\t 9\t Cup\tformation\trequires\tintegrated\tspatio-temporal\tcontrol\tover\tmultiple\t 10\t GTPases.\tThis\tmust\tbe\table\tto\tself-organise\tin\tthe\tabsence\tof\texternal\tcues\t 11\t during\tmacropinocytosis,\tand\trobust\tenough\tto\tphagocytose\tphysiological\t 12\t targets\tof\tvarying\tsize\tand\tshape.\tSmall\tGTPase\tactivity\tis\tcontrolled\tby\ta\tlarge\t 13\t family\tof\tproteins\tsuch\tas\tGTPase\tExchange\tFactors\t(GEFs)\twhich\tpromote\tthe\t 14\t GTP-bound\tactive\tform,\tand\tGTPase\tActivating\tProteins\t(GAPs)\twhich\tstimulate\t 15\t hydrolysis\tand\ttransition\tto\ta\tGDP-bound\tinactive\tstate.\tIn\tthis\tstudy,\twe\t 16\t characterise\ta\tpreviously\tunstudied\tdual\tGEF\tand\tGAP\tprotein\tin\tDictyostelium\t 17\t that\tintegrates\tRas,\tRac\tand\tlipid\tsignaling.\tThis\tprovides\ta\tmechanism\tto\t 18\t macropinosome\tformation\tand\tthe\tengulfment\tof\tdiverse\tbacteria\tof\tdiffering\t coordinate\tthe\tcup\tinterior\twith\tthe\tprotrusive\trim,\tallowing\tefficient\t 19\t geometry.\t 20\t 21\t 22\t 23\t Our\tinitial\thypothesis\twas\tthat\tcells\tmay\tuse\tthe\tdifferent\tmembrane\tcurvature\t \t 24\t 25\t at\tthe\tprotrusive\trim\tcompared\tto\tthe\tcup\tbase\tto\trecognise\tand\tdifferentially\t 26\t regulate\tcup\tshape.\tMembrane\tcurvature\tcan\trecruit\tspecific\t\tproteins\tcontaining\t 27\t BAR\t(Bin-Amphiphysin-Rvs)\tdomains\t(Peter\tet\tal.,\t2004).\tThese\tare\toften\tfound\t in\tmultidomain\tproteins,\tincluding\tseveral\tinvolved\tin\tGTPase\tregulation\tand\t 28\t trafficking\t(Aspenstrom,\t2014).\tTo\tidentify\tcandidate\tproteins\tinvolved\tin\t 29\t macropinocytosis,\twe\ttherefore\tsearched\tthe\tDictyostelium\tgenome\tfor\tBAR\t 30\t 31\t domain-containing\tproteins.\tExcluding\tproteins\tof\tknown\tlocalisation\tor\t function,\twe\tsystematically\tcloned\teach\tcandidate\tand\texpressed\tthem\tas\tboth\tN-\t 32\t \t 5\t  Results  Identification\tof\ta\tnovel\tBAR-domain\tcontaining\tprotein\trecruited\tto\tcups\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   and\tC-\tterminal\tGFP-fusions\tin\taxenic\tAx2\tcells.\tUsing\tthis\tstrategy\twe\t 1\t 2\t successfully\tcloned\t9\tpreviously\tuncharacterised\tBAR-containing\tproteins\tand\t 3\t were\texpressed\tat\tdetectable\tlevels\t(Figure\t1A).\t\t observed\ttheir\tlocalisation\tin\tlive\tcells\tby\tfluorescence\tmicroscopy.\tOf\tthese,\t6\t 4\t \t 5\t DDB_G0284997,\tDDB_G0305372\tand\tDDB_G0285851\twere\tassociated\twith\tsmall\t 6\t 7\t puncta\tat\tthe\tplasma\tmembrane,\tconsistent\twith\tthe\twell-characterised\trole\tof\t 8\t DDB_G0276447\tlocalised\tto\tintracellular\tvesicles\ttoo\tsmall\tto\tbe\t BAR\tdomain\tproteins\tin\tclathrin\tmediated\tendocytosis\t(Dawson\tet\tal.,\t2006).\t 9\t macropinosomes,\tand\tGFP-DDB_G0272368\twas\texclusively\tobserved\tin\tthe\t 10\t 11\t nucleus.\tOnly\tone\tof\tthe\tproteins\ttested\t(DDB_G0269934)\tlocalised\tto\twhat\t appeared\tto\tbe\tthe\tprotrusive\tregions\tof\tmacropinocytic\tcups.\t 12\t 13\t DDB_G0269934\tis\ta\t223\tkDa\tmultidomain\tprotein\tand\talso\tcontains\tRegulator\tof\t \t 14\t 15\t DDB_G0269934\thas\tnot\tpreviously\tbeen\tdescribed\tand\tdue\tto\tits\tdomain\t Chromatin\tCondensation\t(RCC1),\tRhoGEF\tand\tRasGAP\tdomains\t(Figure\t1B).\t 16\t organisation\twe\twill\tsubsequently\trefer\tto\tit\tas\tRGBARG\t(RCC1,\tGEF,\tBAR\tand\t 17\t 18\t GAP\tdomain\tcontaining\tprotein).\tHow\tRas\tand\tRac\tactivity\tare\tcoordinated\tin\t space\tand\ttime\tto\tgenerate\ta\t3-dimensional\tcup\tshaped\tprotrusion\tis\tnot\tknown.\t 19\t Combining\tBAR,\tGEF\tand\tGAP\tactivities\tin\ta\tsingle\tprotein\tpotentially\tprovides\t 20\t 21\t an\telegant\tmechanism\tto\torganise\tengulfment.\tTherefore\tthe\tfunction\tand\t regulation\tof\tRGBARG\twas\tinvestigated\tin\tdetail.\t 22\t 23\t \t Examining\tRGBARG-GFP\tdynamics\tby\ttimelapse\tfluorescence\tmicroscopy\t 24\t 25\t confirmed\tstrong\tenrichment\tat\tthe\tprotrusive\trim\tof\tboth\tmacropinocytic\tand\t phagocytic\tcups\tthat\tdisappeared\trapidly\tafter\tengulfment\t(Figure\t1C\tand\tD,\t 26\t Videos\t1\tand\t2).\tCo-expression\twith\tthe\tPIP3\treporter\tPHCRAC\t-RFP\tthat\tdemarks\t 27\t 28\t the\tcup\tinterior\tconfirmed\tRGBARG-GFP\tlocalised\tspecifically\tto\tthe\tperiphery\tof\t this\tsignaling\tdomain\t(Figure\t1E-H,\tand\tVideo\t3).\tImportantly,\tthis\tdiffers\tfrom\t 29\t 30\t the\tRasGAP\tNF1,\twhich\tlocalises\tthroughout\tthe\tcup\tinterior\t(Bloomfield\tet\tal.,\t 2015).\tRGBARG\tmay\ttherefore\tplay\ta\tspecific\trole\tin\torganising\tcup\tdynamics\t 31\t and\tengulfment.\t 32\t 33\t \t \t  6\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   A  B  C  D  E  F  DDB_G0284997-GFP  DDB_G0276447-GFP  GFP-DDB_G0305372  DDB_G0285851-GFP  DDB_G0272368-GFP  DDB_G0269934-GFP  DDB_G0269934 (225kD)  N  0s  0s  RCC1 RhoGEF BAR  RasGAP  C  751  1072 1080 1284  1332 1528  1715  2044  5s  10s  15s  10s  30s  80s  110s  RGBARG-GFP PHCRAC-RFP  Budding yeast  G  H  )  %  (    e c n e c s e r o u F  l  120 100 80 60 40 20  )  %  (    e c n e c s e r o u F  l  120 100 80 60 40 20  RGBARG PHCRAC  0 2 4 6 8 10  Linescan  0 5 10 15 20 25  Linescan  Figure 1: Identification of BAR domain proteins associated with macropinocytosis. (A) Uncharac- terised BAR-domain containing proteins were expressed as GFP-fusions in Ax2 Dictyostelium cells.  Images are maximum intensity projections of confocal Z-stacks. (B) Illustrates the domain organisation  of DDB_G0269934/RGBARG. (C) Time series of spinning disc images of RGBARG-GFP recruitment  during macropinocytosis and (D) phagocytosis of TRITC-labeled heat-killed budding yeast. (E) and (F)  show the localization of RGBARG-GFP relative to the PIP3 reporter PHCRAC-RFP during macropinocyto- sis and phagocytosis respectively. (G) and (H) show the relative intensity profiles of each fluorescent  protein in these images. Linescans were drawn along the cup interior from protrusive tip-to-tip. All scale  bars denote 5 μm.  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   RGBAR\tregulates\tcup\tsignaling\tand\tmacropinosome\tformation\t \t  1\t 2\t 3\t To\ttest\tfor\ta\tfunctional\trole\tin\tengulfment\ta\t3.6\tKb\tcentral\tsection\tof\tRGBARG\t gene\t(containing\tthe\tRCC1\tRhoGEF\tand\tBAR\tdomains)\twas\tdeleted\tand\treplaced\t 4\t with\ta\tblasticidin\tselection\tcassette\tby\thomologous\trecombination\t(Figure\t2\t 5\t 6\t supplement\t1).\tIndependent\tclones\twere\tisolated\t(JSK02\tand\t03)\tand\tshared\t 7\t comparable\tphenotypes.\tIn\tthe\tfollowing\texperiments\tstrain\tJSK02\twas\tused\t unless\totherwise\tstated\twith\teffects\tof\tRGBARG\tmutation\tvalidated\tby\trescue\t 8\t 9\t experiments.\t \t 10\t 11\t To\tcheck\tfor\tdefects\tin\tmacropinosome\tformation,\tcells\twere\tincubated\twith\t FITC-dextran,\ta\tpH\tsensitive\tdye\tthat\tis\tquenched\tat\tlow\tpH.\tAs\tmacropinosomes\t 12\t acidify\tin\tunder\ttwo\tminutes\tin\tDictyostelium,\tand\timages\twere\tacquired\twithin\t 13\t 14\t the\t30\tminutes\trequired\tto\ttransit\tto\tneutral\tpost-lysosomes,\tintracellular\tFITC- dextran\tis\tonly\tvisible\tin\tnascent\tmacropinosomes\t(Figure\t2A).\tFrom\tconfocal\tZ- 15\t stacks\tof\tlive\tcells,\twe\tfound\tthat\tRGBARG-\tcells\tformed\tsignificantly\tsmaller\t 16\t macropinosomes\tthan\tparental\tcontrols,\tmeasuring\t0.5±0.1\tμm3\tcompared\tto\t 17\t 18\t 1.5±0.2\tμm3\tin\tAx2\t(Figure\t2B).\tThis\tphenotype\tcould\tbe\tcompletely\trescued\tby\t re-expression\tof\tRGBARG-GFP.\tRGBARG-\tcells\talso\tproduced\tmore\t 19\t macropinosomes\t(2.6±0.2\tper\tcell\tcompared\tto\t2.2\t±0.1\tfor\tAx2)\tleading\tto\tno\t 20\t 21\t net\tchange\tin\teither\ttotal\tfluid\tuptake\tor\taxenic\tgrowth\t(Figure\t2C-E).\tRGBARG\tis\t therefore\tfunctionally\timportant\tfor\tthe\tdynamics\tof\tmacropinocytosis\tbut\tnot\t 22\t 23\t essential\tfor\tengulfment.\t \t 24\t 25\t To\tunderstand\twhy\tRGBARG-\tcells\tform\tsmaller\tmacropinosomes\twe\tfollowed\t their\tformation\tby\tfluorescence\tmicroscopy.\t\tUsing\tthe\tPHCRAC-GFP\treporter\twe\t 26\t found\tin\tRGBARG-\tcells\thad\tlarger\tand\tmore\tnumerous\tpatches\tof\tPIP3\tthan\t 27\t 28\t controls,\taveraging\t3.8±1.1\tpatches\tper\tconfocal\tsection\twith\tan\taverage\tlength\t of\t\t5.5±2.4\tµm\tcompared\tto\t1.6±0.6\tpatches\taveraging\t4.4±1.2\tµm\tin\tAx2\t\t(Figure\t 29\t 30\t 3A-C).\tSimilarly\tenlarged\tpatches\twere\talso\tobserved\tusing\tthe\tactive\tRas\t reporter\tGFP-RBD\t(the\tRas\tbinding\tdomain\tof\tRaf1,\tFigure\t3B-C\tand\tFigure\t3\t 31\t 32\t mild\tcytokinesis\tdefect\twith\t10\t±\t%\tcontaining\t>2\tnuclei,\tcompared\tto\t5%\tof\t Supplement\t1A).\tDuring\tthese\texperiments\twe\talso\tnoted\tthat\tRGBARG-\tcells\ta\t 33\t \t  7\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   Ax2  RGBARG-  RGBARG- +  RGBARG-GFP  A  B  2.0  ) 3  l  m μ (   e m u o v   e m o s o n p o r c a M  i  1.5  1.0  0.5  0.0  D  150  100  )  %  (    e k a  t  p u     n=178        n=213         n=110  ns  ***  Ax2  RGBARG- RGBARG- +  RGBARG-GFP  Ax2 RGBARG-  n a r t x e d C T F  I     50  0  n=3  n=3          80  100  0  20  40 60 Time (min)  C  l l  e c / s e m o s o n p o r c a M  i  E  ) s r h (    e m  i t    n o  i t  a r e n e G  3  2  1  0  15  10  5  0  n=82  n=82   *  n=57  ns  Ax2  RGBARG- RGBARG- +  RGBARG-GFP  Ax2  JSK02  JSK03 Random  integrant  RGBARG-  Figure 2: RGBARG- cells produce more, but smaller macropinosomes. (A) Confocal images  of the indicated cell lines incubated in FITC-dextran for 10 minutes. The pH-sensitive FITC is only  visible in pre-acidification macropinosomes <2 minutes after formation. The average volume of  macropinosomes formed is shown in (B) and the number of macropinosomes per cell is shown in  (C). n indicates the total number of macropinosomes or cells measured over 3 independent  experiments. (D) Total fluid uptake measured by FITC dextran uptake over time, measured in a  fluorimeter. (E) Growth rate of RGBARG mutants in axenic culture compared to the Ax2 parental  cell line and a random integrant control from 3 independent experiments. All graphs show means  ± SEM, * P<0.05, *** P<0.001 as determined by Students T-test.  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   1\t mutants\twhich\talso\thave\texcessive\tPIP3\taccumulation\t(Janetopoulos\tet\tal.,\t2005)\t controls.\tThis\tis\tconsistent\twith\tthe\tcytokinesis\tdefects\tdescribed\tin\tPTEN\t 2\t and\tall\tmultinucleate\tcells\twere\texcluded\tfrom\tanalysis.\t 3\t 4\t \t To\tunderstand\thow\tthe\tenlarged\tRas\tand\tPIP3\tpatches\tin\tRGBARG-\tcells\tgive\trise\t 5\t to\tsmaller\tmacropinosomes\twe\tstudied\ttheir\tformation\tover\ttime\tin\t3D.\tAs\t 6\t 7\t recently\tdescribed,\tthe\tmacropinocytic\tcups\tof\tAx2\tcells\tform\tby\texpanding\t around\ta\tdefined\tspontaneous\tpatch\tof\tPIP3\t(Veltman\tet\tal.,\t2016).\tThese\tcups\t 8\t subsequently\tclose,\tusually\tforming\tone,\tor\tsometimes\ttwo,\tlarge\t 9\t macropinosomes\taccompanied\tby\ttermination\tof\tPIP3\tsignaling\ton\tboth\tthe\tnew\t 10\t 11\t vesicle\tand\tthe\tcell\tsurface\t(Figure\t3D\tand\tVideo\t4).\tThis\tprocess\tis\trelatively\t consistent,\twith\teach\tPIP3\tpatch\tlasting\tan\taverage\tof\t150\tseconds\t(Figure\t3\t 12\t Supplement\t1B).\tIn\tRGBARG-\tcells\thowever,\twhilst\tPHCRAC-GFP\tstill\tdisappeared\t 13\t 14\t from\tinternalised\tvesicles,\tthe\tplasma\tmembrane\tdomains\twere\tmuch\tmore\t stable.\t\tWhilst\tPIP3\tpatches\tfrequently\tsplit,\tthey\trarely\tdissipated\tcompletely\t 15\t 16\t and\toften\tlasted\tlonger\tthan\teach\tof\tthe\t30\tminute\tmovies\trecorded\t(Figure\t3E\t and\tVideo\t5).\tIt\twas\ttherefore\tnot\tpossible\tto\tmeaningfully\tmeasure\tthe\tlifetime\t 17\t 18\t of\tsurface\tPIP3\t(and\tby\textension\tRas)\tsignaling\tin\tRGBARG-\tcells.\tAs\tRGBARG\tis\t restricted\tto\tthe\tperiphery\tof\tRas\tsignaling\tdomains\tit\tappears\tto\trestrict\tboth\t 19\t lateral\texpansion\tof\tactivated\tRas\tand\ttermination\tof\tRas/PIP3\tsignaling\tupon\t 20\t 21\t cup\tcompletion.\t\t \t 22\t 23\t Although\textinction\tof\tPIP3\tsignaling\tdid\tnot\taccompany\tcup\tclosure\tin\tRGBARG-\t cells,\tnumerous\tsmall\tvesicles\tcould\tbe\tobserved\tcontinuously\tbudding\tfrom\tthe\t 24\t 25\t base\tof\tthe\truffles\twhen\tfolds\tof\tmembrane\tcollapsed\tin\ton\tthemselves.\tThis\t explains\twhy\tthese\tcells\tform\tmore\tfrequent\tbut\tsmaller\tmacropinosomes\t 26\t (Figure\t2).\tWe\tspeculate\tthat\tthis\tindicates\tthat\tthe\tentirety\tof\tthe\tPIP3\tpatch\tis\t 27\t 28\t potentially\tfusogenic\tand\tcan\tinternalise\tvesicles\tby\tsimply\tfolding\tonto\titself\t rather\tthan\trequiring\ta\tspecific\tmechanism\tto\torchestrate\tclosure\tand\tfission\tat\t 29\t 30\t the\trim.\t\tHow\tthis\tmight\tbe\tachieved\tmechanistically\tis\tunclear,\tbut\tis\t reminiscent\tof\tthe\tless\torganised,\tmore\truffle-like\tmacropinosome\tformation\t 31\t observed\tin\tserum\tstimulated\tmammalian\tepithelial\tcells,\tor\tRas\ttransformed\t 32\t 33\t cancer\tcell\tlines\t(Williamson\tand\tDonaldson,\t2019).\t\t\t \t  8\t  \f",
      "A  2 x A  -  G R A B G R  bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   C  PHCRAC-GFP  B  l l  e c   r e p   s e h c t a P  6  4  2  0  ****  ****  n=30 n=30 n=30 n=30  Ax2 RGBARG- Ax2 RGBARG-  )  i  m μ (   e z s   h c t a P  30  20  10  0  ****  n=46 n=116 n= 54 n=121  **  Ax2 RGBARG-  Ax2 RGBARG-  PHCRAC-GFP  GFP-RBD  PHCRAC-GFP  GFP-RBD  D  Ax2  80 s  120 s  170 s  190 s  230 s  E  RGBarG-  70 s  220 s  F  300 s  610 s  830 s  210 s  230 s  250 s  270 s  Figure 3: Dynamics of macropinosome formation in RGBARG- cells. (A) Membrane localization of the  PIP3 probe PHCRAC-GFP. Images are single confocal sections taken by spinning disc microscopy. The number  of patches of PHCRAC-GFP or the active Ras probe GFP-RBD per cell in single plane through the middle of  each cell is quantified in (B). The average size of each patch is shown in (C). n is the total number of cells or  patches measured over 3 independent experiments. Error bars show the mean ± standard deviation, **  P<0.01, *** P<0.001, Mann-Whitney T-test. (D) and (E) Time series of a maximum intensity projection  through the entire depth of PHCRAC-GFP expressing cells. (D) Shows Ax2 cells, indicating the formation,  closure and subsequent extinction of PHCRAC-GFP patch at the cell surface (arrow). (E) Shows an equivalent  movie of RGBARG- cells, where the PHCRAC-GFP patch remains after closure (arrow). (F) Is an enlargement  of the boxed area in (E), showing multiple vesicles forming from a single, large PHCRAC-GFP patch (arrow- heads). All scale bars indicate 5 μm.  ****\f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   GEF,\tGAP\tand\tBAR\tdomain\tinteractions\teach\tcontribute\tto\tRGBARG\t localisation\t \t  1\t \t 2\t 3\t 4\t 5\t Localisation\tof\tRGBARG\tat\tthe\tinterface\tbetween\tthe\tcup\tinterior\tand\tthe\t 6\t protrusive\trim\tis\tlikely\tto\tbe\tcritical\tto\teffectively\tcontrol\tthe\tshape\tand\t 7\t dynamics\tof\tthese\tdomains\tduring\tengulfment.\tThis\twill\tposition\tits\tRhoGEF\t 8\t activity\twhere\tprotrusion\tis\tpromoted\tand\tits\tRasGAP\tactivity\twhere\tit\tcan\t 9\t restrain\texpansion\tof\tthe\tinterior,\tleading\tto\tthe\torganised\tcup\tformation\t 10\t observed\tin\tAx2\tcells\tand\tabsent\tin\tRGBARG\t–\tmutants.\t 11\t \t 12\t To\tdissect\tthe\tmechanisms\tof\tRGBARG\trecruitment\twe\ttested\tthe\teffect\tof\t 13\t deleting\teach\tprotein\tdomain\tin\tturn.\tTo\tquantify\tRGBARG\tenrichment\tacross\t 14\t the\tcup,\tlinescans\tfrom\tcup\ttip\tto\ttip\twere\tnormalised\tto\tnon-protruding\tregions\t 15\t of\tthe\tsame\tcell\tand\taveraged\tacross\tmultiple\tcells\t(Figure\t4A\tand\tB).\tGFP-fused\t to\tthe\tcyclic\tAMP\treceptor\t(cAR1-GFP)\tlocalises\tuniformly\tto\tthe\tplasma\t 16\t membrane\tand\twas\tused\tas\ta\tcontrol\t(Figure\t4C\tand\tD).\tThis\tmethod\tconfirmed\t 17\t 18\t RGBARG-GFP\twas\tenriched\t3-fold\tat\tthe\tprotruding\tedges\tof\tmacropinocytic\t cups\tand\tallowed\tus\tto\tquantify\thow\teach\tprotein\tdomain\tcontributes\tto\t 19\t recruitment\tat\tthe\tcup\t(Figure\t4\tand\tsupplement).\t 20\t 21\t \t Removal\tof\tthe\tRCC1\tdomain\thad\tno\teffect\ton\tlocalisation\tand\twas\table\tto\tfully\t 22\t 23\t rescue\tthe\tability\tof\tRGBARG-\tcells\tto\tform\tlarge\tmacropinosomes\t(Figure\t4E-H).\t In\tcontrast,\tdeletion\tof\teither\tthe\tRhoGEF\tor\tBAR\tdomains\tcaused\tRGBARG\tto\t 24\t 25\t become\tuniformly\tcytosolic\tand\tdid\tnot\trescue\t(Figure\t4E-H).\tIn\tthe\tabsence\tof\t the\tRasGAP\tdomain\thowever,\tRGBARG\twas\tstill\trecruited\tto\tthe\tplasma\t 26\t membrane\tbut\twas\tmuch\tmore\tbroadly\tdistributed\tthroughout\tthe\tcup\tand\t 27\t significantly\tless\tenriched\tat\tthe\tprotruding\trim\t(Figure\t4G).\tRGBARGΔGAP-GFP\t 28\t was\talso\tunable\tto\trescue\tmacropinosome\tformation\t(Figure\t4H).\tCo-expression\t 29\t 30\t of\tPHCRAC-RFP\tconfirmed\tthat\tRGBARGΔRasGAP-GFP\twas\tno\tlonger\texcluded\t from\tPIP3,\tand\ttherefore\tactive\tRas\tdomains\t(Figure\t4\tsupplement\t2A).\tRasGAP\t 31\t interactions\ttherefore\trestrict\tRGBARG\tto\tthe\tperiphery\tof\tthe\tcup\tinterior\t 32\t 33\t domain.\t \t  9\t  \f",
      "A  E  F  t  n e m h c i r n e d o F     l  bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   B  C  RGBARG-GFP  Zoom  cAR1-GFP  D  t n e m h c i r n e   d o F  l  5 4 3 2 1 0  cAR1-GFP RGBARG-GFP  n=10  n=57 Profile across cup  RGBARGΔRCC1-GFP  RGBARGΔGEF-GFP  RGBARGΔBAR-GFP  RGBARGΔGAP-GFP  RGBarGR1792K-GFP  n=34  5  4  3  2  1  0  n=32  5  4  3  2  1  0  RGBARG RGBARG∆RCC1  RGBARG RGBARG∆GEF  G  5 4  3  2  1  0  **  s p  i t   t  a    t  n e m h c i r n e d o F     l  3  2  1  0  ns  ***  ***  ***  cAR1 Full-length  ∆RCC1  ∆GEF  ∆BAR  ∆GAP R1792K  n=37  n=30  5  4  3  2  1  0  n=30  5  4  3  2  1  0  RGBARG RGBAR∆BAR  RGBARG RGBARG∆GAP  RGBARG RGBARGR1792K  H  l  ) 3 m µ (   e m u o v   e m o s o n p o r c a M  i  2.0  1.5  1.0  0.5  0.0  ns  ns  **  *** ***  ***  ***  Full-length ∆RCC1 Ax2 RGBarG-  ∆GEF  ∆BAR  R1792K ∆GAP  RGBarG- rescue  Figure 4: Multiple interaction regulate RGBARG recruitment at the cup. Full-length RGBARG or mutants lacking each  domain in turn were expressed as GFP-fusions in RGBARG- cells. (A) Shows an example of full-length RGBARG-GFP.  Enrichement was measured relative to the average intensity of a non-protrusive membrane region (green dotted line). The  boxed region is enlarged in (B), showing an example of the line measured from the rim along the cup interior.  (C) Shows  the uniform localization of cAR1-GFP used as a control. (D) Averaged, normalized linescans along the cup from multiple  cells, demonstrating a 3-fold enrichment of RGBARG-GFP at the cup rim and uniform cAR1-GFP concentration. (E) Shows  representative images of cells expressing RGBARG-GFP with the domains indicated delected, as well as the R1792K point  mutation that inactivates RasGAP activity. The averaged intensity of each construct across the cup is shown in (F), with the  profile of the full-length protein from (D) in red for comparison. Values plotted are the mean ± standard deviation. (G) The  enrichment at the protruding rim of each construct measured by averaging the first 10% of each individual linescan and  calculating the mean and SEM across each group. (H) The ability of each construct to rescue large macropinosome  formation in RGBARG- cells was determined by measuring the size of nascent FITC dextran-containing macropinosomes  by confocal microscopy, as in Figure 2A.  >100 macropinosomes over 3 independent experiments were measured. Bars  denote mean macropinosome volume ± SEM, ** P<0.01, *** P<0.005 Mann-Whitney T-test.  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   \t 1\t 2\t To\tconfirm\tthe\trole\tof\tthe\tRasGAP\tinteractions\tin\trestricting\tRGBARG\t 3\t localisation,\twe\talso\tmade\ta\tpoint\tmutant\tin\tthe\tconserved\targinine\tresponsible\t for\tstabilising\tthe\ttransition\tfrom\tRas-GTP\tto\tRas-GDP(Bos\tet\tal.,\t2007).\tThis\t 4\t mutation\t(R1792K)\tis\tpredicted\tto\tdisrupt\tGAP\tactivity\tbut\tstill\tallow\tRas\t 5\t binding\tand\tcompletely\trescued\tRGBARG\texclusion\tfrom\tthe\tcup\tinterior,\tand\t 6\t 7\t slightly\tbut\tsignificantly\tincreased\tenrichment\tat\tthe\tcup\ttip\t(Figure\t4F-G\tand\t supplement\t2B).\tHowever,\tdespite\tlocalising\tto\tthe\tprotruding\trim,\t 8\t 9\t RGBARGR1792K-\tGFP\tdid\tnot\trescue\tthe\tcup\torganisation\tof\tRGBARG-\tcells,\twhich\t still\tproduced\tenlarged\tPIP3\tpatches\tand\tsmall\tmacropinosomes\t(Figure\t4H\tand\t 10\t 11\t Supplement\t2).\tRGBARG\tis\ttherefore\tan\tactive\tRasGAP\tand\tthis\tdomain\talso\t provides\tspatial\tinformation\tto\tposition\tRGBARG\tto\tthe\tperiphery\tof\tthe\tactive\t 12\t Ras/PIP3\tpatch\twhere\tit\tcan\tprevent\tits\texpansion\tand\tshape\tthe\tforming\tcup.\t 13\t 14\t \t The\tdata\tabove\tshow\tthat\tRGBARG\tintegrates\tspatial\tcues\tfrom\tits\tRhoGEF,\tBAR\t 15\t 16\t and\tRasGAP\tdomains\tfor\tcorrect\tpositioning\tand\tcup\torganisation.\tTo\tidentify\t the\trelevant\tbinding\tpartners\tand\tcontribution\tof\teach\tdomain,\twe\talso\t 17\t 18\t expressed\tthem\tindividually\tfused\tto\tGFP.\tWhilst\tRhoGEF-GFP\texpressed\ttoo\t poorly\tto\tobserve\tits\tlocalisation,\tboth\tthe\tRCC1\tand\tGAP\tdomains\twere\t 19\t completely\tcytosolic\t(Figure\t5A\tand\tC).\tIn\tcontrast,\tthe\tBAR\tdomain\talone\twas\t 20\t 21\t sufficient\tfor\tstrong\trecruitment\tthroughout\tthe\tplasma\tmembrane\t(Figure\t5B).\t This\twas\tblocked\tby\tincluding\teither\tof\tthe\tadjacent\tRhoGEF\tor\tRasGAP\tdomains\t 22\t 23\t (Figure\t5D\tand\tE),\tsuggesting\tthat\tBAR-domain\tbinding\tmay\talso\tbe\tregulated\tby\t intramolecular\tinteractions.\tIn\tcontradiction\tof\tour\tinitial\thypothesis\thowever,\t 24\t 25\t BAR-GFP\twas\tnot\tenriched\tat\tareas\tof\tcurvature\tor\tprotrusion.\tThe\tBAR\tdomain\t therefore\tappears\tto\tdrive\tgeneral\trecruitment\tto\tthe\tplasma\tmembrane\trather\t 26\t than\trecognising\tcurvature\tat\tcups.\t 27\t 28\t \t As\tthe\tBAR\tdomain\tof\tRGBARG\tdoes\tnot\tconcentrate\tat\tspecific\tmembrane\t 29\t 30\t shapes,\twe\tinvestigated\tits\tlipid\tbinding\tspecificity\tby\tlipid-protein\toverlay.\t Incubation\tof\tlysates\tfrom\tcells\texpressing\tBAR-GFP\twith\tPIP\tstrips\tindicated\t 31\t binding\tto\tall\tPIPs\twith\ttwo\tor\tmore\tphosphates\t(Figure\t5F).\tThis\twas\tconfirmed\t 32\t 33\t by\tPIP\tarray,\tindicating\ta\tslight\tselectivity\tfor\tPI(3,4)P2\t(Figure\t5\tSupplement\t \t 10\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   1A).\tGiven\tthis\tbroad\tability\tto\tbind\tall\thighly\tphosphorylated\tphosphoinositides\t 1\t 2\t it\tis\tlikely\tthat\tthis\tBAR\tdomain\tgenerally\trecognises\ttheir\thigh\tnegative\tcharge\t 3\t whereby\thighly-phosphorylated\tPIPs\trecruit\tRGBARG\tto\tthe\tplasma\tmembrane\t rather\tthan\tspecific\tphosphate\tconfigurations.\tThis\tsupports\ta\tmechanism\t 4\t via\tits\tBAR\tdomain\twhere\tadditional\tinteractions\twith\tthe\tRhoGEF\tand\tRasGAP\t 5\t domains\tfurther\trestrict\tits\tposition\tand\tactivity\tto\tthe\tprotruding\tedges\tof\t 6\t 7\t forming\tcups.\t\t \t 8\t 9\t To\tidentify\tthe\ttargets\tof\tthe\tRhoGEF\tdomain,\twe\tperformed\tco- immunoprecipitations\twith\ta\tlibrary\tof\trecombinant\tGST-tagged\tsmall\tGTPases.\t 10\t 11\t The\tDictyostelium\tgenome\tcontains\tan\texpanded\tset\tof\tRac\tsmall\tGTPases,\tbut\tno\t Rho\tor\tCDC42\tsubfamily\tmembers\t(Vlahou\tand\tRivero,\t2006).\tOf\tthese\tonly\tRacH\t 12\t and\tRacG\tbound\tthe\tRhoGEF\tdomain\tof\tRGBARG\twith\tno\tdetectable\tbinding\tto\t 13\t 14\t other\tRacs,\tincluding\tRac1\twhich\thas\tpreviously\tbeen\timplicated\tin\tcup\t formation\t(Dumontier\tet\tal.,\t2000).\t\t 15\t \t 16\t Whilst\tRacH\tis\tinvolved\tprimarily\tin\tendocytic\ttrafficking\tand\tlocalises\t 17\t 18\t exclusively\tto\tintracellular\tcompartments\t(Somesh\tet\tal.,\t2006a),\tRacG\tlocalises\t to\tthe\tplasma\tmembrane\tand\tis\tenriched\tat\tthe\tprotruding\trim\tof\tphagocytic\t 19\t cups\t(Somesh\tet\tal.,\t2006b).\tPrevious\tstudies\thave\talso\tshown\tthat\t 20\t 21\t overexpression\tof\twild-type\tor\tconstitutively\tactive\tRacG\talso\tpromotes\t phagocytosis,\tindicating\ta\tpotential\tinteraction\twith\tRGBARG\t(Somesh\tet\tal.,\t 22\t 23\t 2006b).\t\t \t 24\t 25\t Consistent\twith\tprevious\treports,\twe\tfound\tloss\tof\tRacG\thad\tno\tsignificant\teffect\t on\tmacropinocytosis,\twith\tmutants\tforming\tnormal\tsized\tactive\tRas\tpatches\tand\t 26\t macropinosomes\t(Somesh\tet\tal.,\t2006b)(Figure\t5\tsupplement\t1B\tand\tC).\tWhen\t 27\t we\tmeasured\tRGBARG-GFP\trecruitment,\tits\tassociation\twith\tcups\twas\tmore\t 28\t uniform\tand\twas\tonly\tenriched\t1.8±0.6\tfold\t\tat\tthe\trim\tin\tRacG-\tcells\tcompared\t 29\t 30\t to\t2.6±0.7\tfold\tisogenic\tcontrols\t(Figure\t5H-L).\tThis\tindicates\tthat\tRacG\tand\t RGBARG\tfunctionally\tinteract\tin\tvivo\tand\tpartly\tcontribute\tto\tRGBARG\t 31\t localisation.\tHowever,\tthe\tremaining\tsignals\tor\tfunctional\tredundancy\twith\tother\t 32\t \t 11\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   A  B  C  D  E  RCC1-GFP  BAR-GFP  GAP-GFP  GEF-BAR-GFP  BAR-GAP-GFP  F  G  Marker  Rac1A  RacE  RacF2  RacG  RacH  RacI  RacJ  RacL  Input GFP-GEF  LPA  LPC  PtdIns  PI(3)P  PI(4)P  PI(5)P  PE  PC  H  70 kDa  55 kDa  S1P  PI(3,4)P2  PI(3,5)P2  PI(4,5)P2  PI(3,4,5)P3  PA  PS  Blue blank  RGBarG-GFP  I  L  s p  i t   t  a    t  n e m h c i r n e d o F     l  RGBARG-GFP cAR1-GFP  n=15  n=37  RacG-  K  5  t  n e m h c i r n e   d o F  l  4  3  2  1  0  Ax2D  J  t  n e m h c i r n e   d o F  l  5  4  3  2  1  0  RGBARG-GFP cAR1-GFP  n=10  n=19  ***  3  2  1  0  cAR1  RGBARG  cAR1  RGBARG  Ax2D  RacG-  Profile across cup  Profile across cup  Figure 5: Binding specificity of the BAR and GEF domains. (A-E) Individual, or combinations of domains from  RGBARG were expressed as GFP fusions in RGBARG- cells. Images shown are single confocal sections. (F) Lipid  binding specificity of BAR-GEF by lipid overlay assay using whole cell lysate from BAR-GFP expressing cells. (G) Rac  binding specificity of the RhoGEF domain determined by co-immunoprecipitation of GEF-GFP by a library of purified  GST-Rac’s bound to beads. (H) and (I) Confocal images of full-length RGBARG-GFP localisation in RacG- cells and  their parental background strain Ax2D. The average profile (± standard deviation) of RGBARG-GFP along the cup  relative in Ax2D and RacG-  cells relative to cAR1-GFP is shown in (J) and (K) respectively. (L) Enrichment of  RGBARG-GFP and cAR1 at the cup tip in each cell line. Bars indicate mean ± SEM, *** P<0.005 Mann-Whitney T-test.  All scale bars indicate 5 μm.  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   RGBARG\tis\ta\thighly\tactive\tdual\tspecificity\tRas/Rap\tGAP\t\t  1\t Racs\tis\tsufficient\tfor\tpartial\tRGBARG\trecruitment\tand\tapparently\tnormal\t 2\t engulfment\tin\tthe\tabsence\tof\tRacG.\t 3\t \t 4\t Combined,\tour\tdata\tindicate\tthat\tRGBARG\tuses\ta\tcoincidence\tdetection\t 5\t recruitment\tmechanism\tto\tdirect\tcup\tformation:\tBAR\tdomain\tbinding\tto\t 6\t whilst\tadditional\tinteractions\twith\tRacG\tand\tactive\tRas\tsynergise\tto\tconstrain\t negatively\tcharged\tphospholipids\tdirects\tthe\tprotein\tto\tthe\tplasma\tmembrane\t 7\t RGBARG\tto\tthe\tcup\trim.\tThis\ttripartite\tregulation\tensures\tthat\tRGBARG\tis\t 8\t 9\t accurately\tpositioned\tto\texert\tits\tRhoGEF\tand\tRasGAP\tactivities\tat\tthe\tinterface\t between\tcup\tinterior\tand\tprotrusion\tand\torganise\tengulfment.\t 10\t 11\t \t 12\t \t 13\t 14\t Forming\ta\tcomplex\t3-dimensional\tshape\tis\tlikely\tto\trequire\tmultiple\tregulators\t and\tRGBARG\tis\tnot\tthe\tonly\tRasGAP\tinvolved\tin\tmacropinocytosis\tin\t 15\t Dictyostelium.\tAxenic\tstrains\t(including\tthe\tAx2\tparental\tstrain\tused\tin\tthis\t 16\t work)\talso\tharbour\tmutations\tin\tNF1\tthat\tenhance\tfluid\tuptake\tvia\tenlarged\t 17\t 18\t active\tRas\tpatches\tand\tsubsequent\tformation\tof\tlarger\tmacropinosomes\t (Bloomfield\tet\tal.,\t2015).\tHowever,\twhilst\tmutations\tin\tNF1\twere\treported\tin\t 19\t each\tof\t10\tindependently\tisolated\taxenic\tstrains,\tRGBARG\tmutations\twere\t 20\t 21\t completely\tabsent\t((Bloomfield\tet\tal.,\t2015)\tand\tG.\tBloomfield,\tpersonal\t communication,\tMay\t2019).\tOur\tattempts\tto\tgenerate\tRGBARG\tmutants\tin\tnon- 22\t 23\t axenic\tstrains\twere\talso\tunsuccessful.\tThis\tindicates\tthat\tdisruption\tof\tNF1\tbut\t not\tRGBARG\tis\tsufficient\tto\tsupport\taxenic\tgrowth,\tand\tthat\tthey\tserve\tdifferent\t 24\t 25\t functions.\t \t 26\t To\tbetter\tunderstand\tthe\tdifferences\tbetween\tNF1\tand\tRGBARG,\twe\tcompared\t 27\t 28\t the\tspecificity\tand\tactivities\tof\ttheir\tRasGAP\tdomains.\tThe\tDictyostelium\tgenome\t encodes\t14\tRas\tsubfamily\tmembers\tof\twhich\tRasB,\tRasG\tand\tRasS\tare\tthe\tmost\t 29\t 30\t important\tfor\tmacropinocytosis\t(Chubb\tet\tal.,\t2000;\tHoeller\tet\tal.,\t2013;\t Junemann\tet\tal.,\t2016;\tKhosla\tet\tal.,\t2000).\tOverexpression\tof\tRasD\tcan\talso\t 31\t partially\tcompensate\tfor\tloss\tof\tRasG\tand\tS\t(Khosla\tet\tal.,\t2000).\tThe\tsmall\t 32\t 33\t GTPase\tRap,\ta\tclose\trelative\tof\tRas,\thas\talso\tbeen\timplicated\tin\tmacropinosome\t \t 12\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   A  20  15  10  )  M u (    P D G  5  0  0  B  0.15  0.10  1 -  RGBARG NF1  Intrinsic Buffer  200  400  Time (mins)  600  800  RasB RasD RasG RasS RapA     i  n m s b o  k  0.05  0.00  RGBARG  NF1  Intrinsic  Figure 6: RasGAP activity of RGBARG and NF1. (A) Stimulation of GDP released from  GTP-loaded RasG upon addition of recombinant RasGAP domains from either RGBARG or NF1,  compared to the intrinsic GAP activity of the GTPase or GTP in buffer. (B) GAP activity of NF1 and  RGBARG against a library of Ras superfamily members. Average of 3 independent experiments  performed as in (A) in parallel. Bars indicate mean ± standard deviation.  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:      All rights reserved. No reuse allowed without permission.   Loss\tof\tRGBARG\timproves\tphagocytosis\tof\tlarge\tobjects\t  found that the RasGAP domain of RGBARG was highly active against all the  GTPases tested (Figure 6). The RasGAP domain of NF1 was also active against each  Ras tested, but with 75% less activity than RGBARG in each case. RGBARG is  therefore a more potent RasGAP in vitro, but the lack of specificity for particular Ras  isoforms for both RGBARG and NF1 indicates their functional differences are likely   formation\t(Inaba\tet\tal.,\t2017).\tWe\ttherefore\tmeasured\tthe\tspecific\tGAP\tactivity\t from\tboth\tNF1\tand\tRGBARG\tagainst\teach\tsmall\tGTPase.\t  Consistent with the inability of RGBARGR1792K to rescue the knockout phenotype, we   1\t 2\t 3\t 4\t 5\t 6\t 7\t 8\t 9\t 10\t imparted by additional factors such as their cellular localisation and dynamics.\t 11\t \t 12\t 13\t \t 14\t The\tdata\tabove\tdemonstrate\tthat\tRGBARG\tis\timportant\tduring\tthe\tspontaneous\t 15\t self-organisation\tof\tmacropinocytic\tcups.\t\tAs\tRGBARG\talso\tlocalises\tto\t 16\t phagocytic\tcups\tand\tengulfment\tof\tsolid\tparticles\tsuch\tas\tmicrobes\tuses\tmuch\tof\t 17\t the\tsame\tmachinery,\twe\talso\tinvestigated\thow\tRGBARG\tcontributes\tto\t 18\t phagocytosis.\t 19\t Disruption\tof\tNF1\thas\tpreviously\tbeen\tshown\tto\tincrease\tthe\tsize\tof\tparticles\t \t 20\t 21\t that\tDictyostelium\tcan\tengulf\t(Bloomfield\tet\tal.,\t2015).\tAs\tRGBARG\talso\taffects\t the\tsize\tof\tthe\tPIP3\tdomains\tthat\tdefine\tthe\tcup\tinterior\twe\tfirst\ttested\tthe\tability\t 22\t 23\t of\tRGBARG-\tcells\tto\tphagocytose\tdifferent\tsized\tbeads.\tWhilst\tdisruption\tof\t RGBARG\thad\tno\teffect\ton\tphagocytosis\tof\t1\tμm\tdiameter\tbeads,\tengulfment\tof\t4.5\t 24\t μm\tbeads\twas\tenhanced\t3-fold,\twith\tan\taverage\tof\t\t2.2±0.4\tbeads\tengulfed\tper\t 25\t 26\t cell\tafter\t1\thour,\tcompared\tto\t1.0±0.4\tin\tcontrol\t(Figure\t7A\tand\tB).\tEnhanced\tRas\t activation\ttherefore\tappears\tto\tbe\tgenerally\tbeneficial\tfor\tthe\tengulfment\tof\t 27\t 28\t large\tspherical\ttargets.\t \t 29\t 30\t Surprisingly,\talthough\texpression\tof\tRGBARG-GFP\tfrom\tan\textrachromosomal\t vector\tfully\trescued\tmacropinosome\tformation\t(Figure\t2A-C),\tthis\treduced\tthe\t 31\t ability\tof\tRGBARG-\tcells\tto\tengulf\t4.5\tμm\tbeads\tto\t63%\tof\tcontrol\tlevels\t(Figure\t 32\t 33\t 7B).\tThis\teffect\twas\teven\tmore\tsevere\tupon\texpression\tof\tdomain\tdeletion\t \t 13\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   B  l l  e c /   s d a e b m μ 4        A  l l  e c /   s d a e b m μ 1        60  40  20  0  0  Ax2 RGBARG-  10 20 Time (min)  30  2.5  2.0  1.5  1.0  0.5  0.0  *  *  *  *  RGBARG- RGBARG- + RGBARGR1792K-GFP Ax2  RGBARG- + RGBARG-GFP  RGBARG- + RGBARGΔGEF-GFP  RGBARG- + RGBARGΔBAR-GFP  RGBARG- + RGBARGΔGAP-GFP  RGBARG- + RGBARGΔRCC1-GFP  0  10 20 Time (min)  30  C  D  E  e r u  l i  a F %     Ax2  0 s  10 s  20 s  30 s  40 s  50 s  PHcrac Yeast  RGBarG-  0 s  20 s  60 s  80 s  100 s  190 s  n=37 n=41  *  30  20  10  0  n=25 n=39  ns  F  ) s (    e m  i t    n o  i t  a s  i l  a n r e  t  n  I  200  150  100  50  0  Ax2  RGBARG-  Ax2  RGBARG-  Figure 7: Phagocytic defects in RGBARG- cells. (A) Ax2 and RGBARG- cell have comparable  rates of phagocytosis of fluorescent 1 μm beads. (B) Phagocytosis of 4 μm beads by Ax2 cells,  RGBARG- cells, or RGBARG- cells expressing full-length or mutant RGBARG-GFP. Phagocytosis  measured by flow cytometry in 3 independent experiments. Phagocytosis of TRITC-labelled yeast  was directly observed by spinning disc confocal microscopy by either (C) Ax2 or (D) RGBARG- cells  expressing PHCRAC-GFP. (C) Shows an example of a failed engulfment. (E) The relative frequency  of phagocytosis failure after cup formation (indicated by PHCRAC-GFP recruitment). The time from  initial contact to complete engulfment in successful phagocytic events is shown in (F). n indictates  total number of phagocytic events over 3 independent experiments. All values plotted are mean ±  standard deviation. * P<0.01 Students T-test.  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   1\t mutants\tincluding\tthe\tΔBAR,\tΔGEF\tand\tΔGAP\tconstructs\twhich\tdo\tnot\tlocalise\t 2\t properly\tand\thave\tno\tdeleterious\teffect\ton\tmacropinosome\tformation.\tThis\t 3\t indicates\ta\tdominant\tnegative\teffect,\tmost\tlikely\tdue\tto\tsequestration\tof\tbinding\t 4\t partners.\tIn\tcontrast,\texpression\tof\tRGBARGR1792K\thad\tno\tinhibitory\teffect\ton\t 5\t RGBARG-\tcells.\tThis\tonly\tdiffers\tfrom\tRGBARG-GFP\tin\tits\tRasGAP\tactivity\t 6\t indicating\tthat\tmislocalisation\tor\toverexpression\tof\tthis\tdomain\tis\tsufficient\tto\t 7\t inhibit\tengulfment\tof\tlarge\ttargets.\t 8\t \t 9\t To\tbetter\tunderstand\thow\tloss\tof\tRGBARG\taffects\tphagocytosis\twe\tperformed\t 10\t time-lapse\tmicroscopy\tof\tcells\texpressing\tPHCRAC-GFP\tengulfing\tTRITC-labeled\t yeast.\tEngulfment\toccurred\trapidly\tin\tboth\tcell\ttypes\tbut\tfailed\tat\ta\tfrequency\tof\t 11\t ~20%\tin\tAx2\tcells\twith\tthe\tPIP3\tpatch\tdissipating\tand\tthe\tyeast\tescaping\tfrom\t 12\t the\tcell\t(Video\t6).\tWhilst\tthe\ttime\tfor\tsuccessful\tengulfment\twas\tno\tsignificantly\t 13\t 14\t altered\tby\tloss\tof\t\tRGBARG\t(129±11\tminutes\tin\tmutants\tvs\t147±18\tminutes\tin\t Ax2),\tcapture\twas\tmuch\tmore\trobust\twith\ta\tfailure\trate\tof\tonly\t4±5\t%\tcompared\t 15\t 16\t to\t22±\t7%\t(Figure\t7C-F\tand\tVideo\t7).\tThe\tmain\tinfluence\ton\tthe\tphagocytic\t efficiency\tof\tlarge\ttargets\tin\tRGBARG-\tcells\tthus\tappears\tto\tbe\tincreased\tcup\t 17\t stability\tand\tenlarged\tRas\tsignaling\tpatch\trather\tthan\trate\tof\tprotrusion\taround\t 18\t the\tobject.\t\t 19\t \t 20\t 21\t 22\t 23\t \t To\tbe\teffective,\tphagocytic\tcells\tmust\tbe\table\tto\tengulf\tmicrobes\twith\tdiffering\t 24\t 25\t physical\tproperties\tsuch\tas\tshape,\tsize,\tstiffness\tand\tsurface\tchemistry.\tAs\t 26\t macropinocytic\tcups,\twe\talso\tinvestigated\tits\trole\tduring\tthe\tengulfment\tof\t RGBARG\tis\timportant\tfor\tthe\torganisation\tand\tstability\tof\tphagocytic\tand\t 27\t 28\t different\tbacteria.\t \t 29\t 30\t Phagocytosis\twas\tmeasured\tby\tthe\tability\tof\tDictyostelium\tcells\tto\treduce\tthe\t turbidity\tof\ta\tbacterial\tsuspension\tover\ttime.\tWhilst\tdisruption\tof\tRGBARG\thad\t 31\t no\teffect\ton\tthe\tability\tto\tclear\ta\tsuspension\tof\tKlebsiella\taerogenes,\tengulfment\t 32\t 33\t of\tEscherishia\tcoli\twas\tsubstantially\treduced\t(Figure\t8A\tand\tB).\tThis\twas\tfully\t \t 14\t  Spatial\tregulation\tof\tRas\tby\tRGBARG\tis\timportant\tfor\tphagocytosis\tof\t elongated\ttargets\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   1\t rescued\tby\tre-expression\tof\tRGBARG-GFP.\tTherefore,\talthough\tloss\tof\tRGBARG\t 2\t has\tno\teffect\ton\tthe\tengulfment\tof\t1\tμm\tbeads\tand\tis\tbeneficial\tfor\tthe\tuptake\tof\t 3\t large\tbeads\tand\tyeast,\tit\tcauses\ta\tspecific\tdefect\tin\tengulfment\tof\tsome\tbacteria.\t 4\t \t 5\t The\tmost\tobvious\tphysical\tdifference\tbetween\tK.\taerogenes\tand\tE.\tcoli\tis\ttheir\t 6\t shape\t(Figure\t8C\tand\tD).\tBoth\thave\tsimilar\tshort\taxes\tbut\tK.\taerogenes\taverage\t 7\t 3.2\tμm\tin\tlength\twhilst\tE.\tcoli\thave\tan\taverage\tlong\taxis\tof\t5.4\tμm.\tPrevious\twork\t 8\t investigating\tphagocytosis\tof\tdifferent\tshaped\tbeads\tby\tmacrophages\tconcluded\t that\tcomplex\telongated\tshapes\tare\tmore\tdifficult\tto\tengulf\t(Champion\tand\t 9\t Mitragotri,\t2006).\tWe\ttherefore\thypothesised\tthat\tthe\tselective\tphagocytosis\t 10\t 11\t defects\tof\tRGBARG-\tcells\twas\tdue\tto\tthe\ttarget\tshape.\tTo\ttest\tthis,\twe\tmeasured\t the\tability\tof\tRGBARG-\tcells\tto\tengulf\tan\tadditional\telongated\trod-shaped\tbacteria\t 12\t (GFP-expressing\tMycobacteria\tsmegmatis,\t3-5\tμm\tlong)\tby\tflow\tcytometry.\tThe\t 13\t 14\t ability\tof\tRGBARG-\tcells\tto\tengulfing\tthese\tbacteria\twas\tagain\treduced\tby\t75%\t (Figure\t8E),\tagain\tcorrelating\twith\tan\tinability\tto\tphagocytose\telongated\ttargets.\t 15\t 16\t \t The\tdata\tabove\tare\tconsistent\twith\ta\trole\tfor\tRGBARG\tin\tenabling\tthe\tengulfment\t 17\t 18\t of\telongated\tbacteria.\tHowever,\tthe\tbacterial\tstrains\tused\twill\talso\tdiffer\tin\tother\t aspects\tsuch\tas\ttheir\tsurface\tcomponents,\tphagocytic\treceptor\tactivation\tand\t 19\t stiffness.\tTo\tdirectly\ttest\tthe\timportance\tof\tRGBARG\tin\tengulfing\ttargets\tof\t 20\t 21\t different\tshape\twe\ttherefore\tstretched\t3\tμm\tlatex\tbeads\tto\tgenerate\toblate\t ellipsoids\tof\tconserved\tvolume\tand\tsurface\tchemistry\t(Ho\tet\tal.,\t1993).\t\t 22\t 23\t \t 24\t with\ta\t1:1\tmix\tof\tspherical\tand\tstretched\tbeads\t(2.6x\taspect\tratio)\tand\tthe\t To\tmeasure\trelative\tphagocytosis\tin\tthe\tsame\texperiment,\tcells\twere\tincubated\t 25\t number\tof\tengulfed\tbeads\tof\teach\tshape\tquantified\tby\tmicroscopy.\tThe\tability\tof\t 26\t Ax2\tcells\tto\tengulf\tellipsoid\tparticles\twas\treduced\tby\t30%\tcompared\tto\tspheres\t 27\t 28\t (P<0.01,\tT-test).\tHowever,\twhilst\tRGBARG-\tcells\tengulfed\tthe\tspheres\twith\tsimilar\t efficiency,\tthe\tnumber\tof\tstretched\tbeads\ttaken\tup\twas\treduced\tby\tover\t70%\t 29\t 30\t (Figure\t8C,\tP<0.01,\tT-test).\tThese\teffects\twere\tagain\trescued\tby\tre-expression\tof\t RGBARG-GFP,\tbut\tnot\tRGBARGR1792K-GFP\tdemonstrating\ta\tkey\trole\tfor\tthe\t 31\t RasGAP\tactivity\tin\tmediating\tphagocytosis\tof\telongated\tparticles.\t 32\t 33\t \t \t  15\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   90  80  70  60  50  0  Ax2 RGBARG- RGBARG- + RGBARG-GFP  100 200 Time (min)  300  A  100  )  %  i  (   y t i d b r u t   s e n e g o r e a   .  K     C  E  E. coli K. aerogenes  B  110  100  90  80  )  %  i  (   y t i d b r u t   i l  o c   .  E     70  0  40  30  20  10  0  D  )  %  (   y c n e u q e r f    e v i t  l  a e R  *  *  *  **  Ax2 RGBARG- RGBARG- + RGBARG-GFP  100 200 Time (min)  300  E.  coli K.  aerogenes  n=1105 n=1760  2  6  4 8 Long axis (μm)  10  )  %  (    e k a  t  p u    s i t  a m g e m s   .  M    -  P F G  120  80  40  0  0  ****  ***  Ax2 RGBARG-  ***  ***  ***  ***  20  40  60 Time (min)  80  Figure 8: RGBARG- cells have selective defects in phagocytosis of bacteria. Phagocytosis of  (A) K. aerogenes  or (B) E. coli was measured by monitoring the decreasing turbidity of a bacterial  suspension after addition of Dictyostelium. (C) show wide-field fluorescence microscopy images of  GFP-expressing K. aerogenes mixed with RFP-expressing E. coli, demonstrating their different  shape and size. The length of each bacteria was quantified automatically and is plotted in (D). (E)  Phagocytosis of GFP-M. smegmatis, an alternative elongated bacteria, measured by flow cytome- try. Values plotted in (A), (B) and (E) are mean ± standard deviation of 3 independent experiments.  *P<0.05, **P<0.01, ***P<0.005, Student’s T-test.  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   1\t 2\t 3\t 4\t 5\t 6\t 7\t 8\t 9\t  All rights reserved. No reuse allowed without permission.   How\tphagocytic\tcups\torganise\tand\tadapt\ttheir\tcytoskeleton\tto\tengulf\ttargets\tof\t differing\tgeometry\tis\tvery\tpoorly\tunderstood.\tOur\tdata\tdemonstrate\tthat\tin\t Dictyostelium,\ta\ttripartite\trecruitment\tmechanism\toperates\tto\tprecisely\tposition\t a\tRasGAP\tand\tRhoGEF\tdomain-containing\tprotein\tat\tthe\tinterface\tbetween\tthe\t protrusive\trim\tand\tstatic\tinterior\tdomains\tof\tphagocytic\tand\tmacropinocytic\t cups.\tWe\tpropose\ta\tmodel\twhereby\tthis\torganises\tthe\tcup\tby\tregulating\tthe\t balance\tbetween\tprotrusion\tand\texpansion\tof\tthe\tinterior\tin\torder\tto\tboth\t efficiently\tform\tmacropinosomes\tand\tfacilitate\tengulfment\tof\tgeometrically\t diverse\ttargets.\t  \t  \t  16\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   Discussion\t\t \t  1\t 2\t 3\t In\tthis\tstudy\twe\thave\tidentified\ta\tnew\tcomponent\tand\tmechanism\tused\tby\tcells\t 4\t to\torganise\ttheir\tprotrusions\tinto\tthe\t3-dimensional\tcup\tshapes\trequired\tto\t 5\t engulf\textracellular\tfluid\tor\tparticles.\tConsistent\twith\tprevious\tstudies,\tour\tdata\t 6\t support\ta\tmodel\twhereby\tcup\tformation\tis\tguided\tby\tthe\tformation\tof\ta\t 7\t protrusive\trim\tencircling\ta\tstatic\tinterior\tdomain\t(Veltman\tet\tal.,\t2016).\tWe\t 8\t show\tthat\tin\tDictyostelium,\tRGBARG\tprovides\ta\tdirect\tlink\tbetween\tthe\tRas\tand\t Rac\tactivities\tthat\tunderlie\tthese\tdifferent\tfunctional\tdomains,\tproviding\ta\tnovel\t 9\t mechanism\tto\tco-ordinate\tcup\torganisation\tin\tspace\tand\ttime.\t\t 10\t 11\t \t RGBARG\tis\tnot\tthe\tonly\tRasGAP\tin\tDictyostelium\tinvolved\tin\tmacropinosome\t 12\t formation.\tHowever,\tit\tis\tthe\tonly\tRasGAP\tin\tthe\tDictyostelium\tgenome\tto\talso\t 13\t 14\t possesses\ta\tRhoGEF\tdomain.\tRGBARG\tis\ttherefore\tunique\tin\tits\tability\tto\t 15\t with\tan\tidentical\tdomain\tstructure\tto\tRGBARG,\tand\twhilst\tmost\tclassical\t integrate\tthe\tactivities\tof\tboth\tGTPase\tfamilies.\tThere\tare\tno\thuman\tproteins\t 16\t RasGAPs\tare\tfound\tin\tmultidomain\tproteins,\tnone\talso\tcontain\ta\tclassical\t 17\t 18\t RhoGEF\tdomain\t(Bos\tet\tal.,\t2007).\tA\tscreen\tfor\tRhoGAPs\tinvolved\tin\tphagosome\t formation\tin\tmacrophages\tidentified\t3\tproteins\t(ARHGAP12,\tARHGAP25\tand\t 19\t SH3BP1),\tbut\talthough\tthey\tall\tcontain\tPIP3\tbinding\t(PH)\tor\tBAR\tdomains,\tnone\t 20\t 21\t contain\tdomains\tthat\tlink\tto\tother\tGTPase\tfamilies\t(Schlam\tet\tal.,\t2015).\tThe\t oncogene\tTIAM1\tcontains\tboth\tRhoGEF\tand\tRas-binding\tdomains\thowever,\tand\t 22\t 23\t BAR\tdomains\tare\tfound\tin\tconjunction\twith\tGAP\tor\tGEFs\tin\tseveral\tother\t proteins.\tTherefore\twhilst\tmammalian\tcells\talso\tneed\tto\tcoordinate\tRas\tand\tRac\t 24\t 25\t activity\tduring\tcup\tformation,\tthis\tis\tlikely\tachieved\tvia\tmultiple\tproteins,\t potentially\tin\ta\tcomplex.\t 26\t \t 27\t Multiple\tGAP’s\tand\tGEF’s\tcollaborate\tto\tshape\tprotrusions\tinto\tcups.\tThis\tis\t 28\t 29\t apparent\tin\tthe\tdifferent\troles\tplayed\tby\tRGBARG\tand\tNF1.\tBoth\tare\timportant\t negative\tregulators\tof\tRas,\tbut\twhilst\tRGBARG\tis\tspecifically\tenriched\tat\tthe\tcup\t 30\t 31\t rim,\tNF1\tappears\tto\tbe\tpresent\tthoughout\tthe\tcup\t(Bloomfield\tet\tal.,\t2015).\t RGBARG\tand\tNF1\ttherefore\tplay\tdifferent\tfunctional\troles;\twhilst\tdisruption\tof\t 32\t \t 17\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   1\t NF1\tleads\tto\tan\tincrease\tin\tthe\tvolume\tof\tfluid\ttaken\tup\tand\tcan\tfacilitate\taxenic\t 2\t growth\t(Bloomfield\tet\tal.,\t2015),\tRGBARG\tappears\tmore\timportant\tfor\tcup\t 3\t structure\tand\tshape.\tWe\ttherefore\tspeculate\ta\tmodel\twhereby\tNF1\tacts\tto\t 4\t generally\tsuppress\tRas\tactivity\tand\tregulate\tthe\tspontaneous\texcitability\tof\t 5\t active\tRas\tpatches,\twhilst\tRGBARG\toperates\tat\ttheir\tperiphery\tto\trestrict\ttheir\t 6\t expansion\tand\tstimulate\tprotrusion\tvia\tRac.\t\t 7\t \t 8\t This\tmodel\tis\tdoubtless\toverly\tsimplistic,\tand\tother\tRasGAP’s\talso\tcontribute\tto\t 9\t shaping\tactive\tRas\tdynamics.\tFor\texample,\tthe\tIQGAP-related\tprotein\tIqgC\twas\t 10\t also\trecently\tshown\tto\thave\tRasGAP\tactivity\tand\tlocalise\tthroughout\tthe\tinterior\t 11\t of\tmacropinocytic\tand\tphagocytic\tcups\tin\tDictyostelium\t(Marinovic\tet\tal.,\t2019).\t 12\t In\tcontrast\tto\tour\tfindings\twith\tNF1\tand\tRGBARG\thowever,\tIqgC\tis\treported\tto\t 13\t have\tspecific\tRasGAP\tactivity\tagainst\tRasG.\tAs\tthe\tdifferent\tRas\tisoforms\tare\t 14\t non-redundant\t(Khosla\tet\tal.,\t2000),\tIqgC\tadds\ta\tfurther\tlayer\tof\tregulatory\t 15\t complexity\tshaping\tthe\tdynamics\tof\tengulfment.\t 16\t Whilst\tthe\tregulation\tof\tRas\tsignaling\tand\tthe\tstatic\tinterior\tdomain\tis\tbecoming\t \t 17\t 18\t clearer,\thow\tprotrusion\tis\tregulated\tduring\tengulfment\tis\tless\twell\tunderstood.\t In\tmammalian\tcells,\tseveral\tstudies\tindicate\tthat\tactin\tdynamics\tand\tprotrusion\t 19\t 20\t at\tthe\tcup\tis\tregulated\tby\tthe\tcombined\tactivities\tof\tRac1\tand\tCDC42\t(Cox\tet\tal.,\t 1997;\tMassol\tet\tal.,\t1998;\tSchlam\tet\tal.,\t2015).\tRac1\tand\tCDC42\tare\tdifferentially\t 21\t 22\t more\trestricted\tto\tthe\trim\t(Hoppe\tand\tSwanson,\t2004).\tWhilst\tDictyostelium\t activated\twith\tactive\tRac1\tthroughout\tthe\tcup\tand\tCDC42\tactivation\tearlier\tand\t 23\t does\tnot\tpossess\ta\tclear\tCDC42\torthologue,\twe\tfind\tthat\tRGBARG\tspecifically\t 24\t 25\t interacts\twith\tthe\tatypical\tRac\tisoforms\tRacG\tand\tRacH.\tCurrently,\tno\tdirect\t effectors\tof\teither\tprotein\tare\tknown,\tand\tRacG\tdoes\tnot\tinteract\twith\tthe\tRac- 26\t 27\t binding\tdomain\tof\tPAK\tcommonly\tused\tas\ta\tprobe\tfor\tactive\tRac1\t(Somesh\tet\tal.,\t 2006b).\tRacG\ttherefore\thas\tat\tleast\tpartly\tdistinct\teffectors\tto\tRac1.\tNonetheless,\t 28\t in\tcell-free\tassays,\tRacG\tis\table\tto\tinduce\tactin\tpolymerisation\tvia\tthe\tARP2/3\t 29\t 30\t complex,\talthough\twhether\tthis\tis\tdependent\ton\tSCAR/WAVE\tor\tother\tWASP\t family\tmembers\tis\tnot\tknown\t(Somesh\tet\tal.,\t2006b).\tRacG\ttherefore\tappears\tto\t 31\t \t 18\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   be\tat\tleast\tpartly\tresponsible\tfor\tdefining\tthe\tprotrusive\trim,\tpossibly\tthrough\t 1\t 2\t some\tcoincidence-detection\tmechanism\twith\tactive\tRac1.\t\t 3\t Whilst\tRacG\thas\tno\tclear\tdirect\torthologue\tin\tmammalian\tcells,\tit\tis\tmost\tsimilar\t \t 4\t to\tCdc42\tin\tprotein\tsequence.\tWhilst\tconstitutively\tactive\tRac1\tinduces\tthe\t 5\t formation\tof\tlamellipodial-type\tprotrusions\t(Dumontier\tet\tal.,\t2000),\t 6\t 7\t constitutively\tactive\tRacG\tand\tCdc42\tboth\tinduce\tfilopodia\t(Nobes\tand\tHall,\t 1995;\tSomesh\tet\tal.,\t2006b).\tWe\ttherefore\tspeculate\tthat\tRacG\tand\tCdc42\tare\t 8\t 9\t functional\torthologues,\tand\tthe\tmechanism\tby\twhich\tthese\tsmall\tGTPases\t integrate\twith\tactive\tRas\tto\trestrict\tthe\tlocalisation\tof\tRhoGEF\tand\tRasGAP\t 10\t 11\t proteins\tsuch\tas\tRGBARG\tis\ta\tgeneral\tdevice\tused\tto\tdefine\tthe\tprotrusive\tcup\t rim.\t 12\t \t 13\t 14\t The\tinvolvement\tof\tboth\tRacG/Cdc42\tand\tRac1\tfamily\tGTPases\tindicates\ta\t complex\trelationship\tbetween\tfilopodial\tand\tlamellipodial\ttype\tprotrusions\t 15\t 16\t during\tcup\tformation.\tWhilst\tmost\tstudies\tin\tDictyostelium,\tRAW\tmacrophages,\t dendritic\tcells\tand\tcancer\tcell\tlines\tdescribe\tmacropinosome\tformation\tfrom\t 17\t smooth,\tsheet-like\tprojections\tor\tcups\t(Swanson,\t2008;\tVeltman\tet\tal.,\t2016;\t 18\t West\tet\tal.,\t2000;\tWilliamson\tand\tDonaldson,\t2019),\tit\twas\trecently\tshown\tthat\t 19\t RAW\tmacrophages\tcan\talso\tform\tmacropinosomes\tby\ta\tmore\tfilopodial\t“tent- 20\t 21\t pole”-type\tmechanism\twhere\tprotrusion\tis\tdriven\tby\tactin-rich\tspikes\t(Condon\t et\tal.,\t2018).\tWhether\tthis\tis\ta\tgeneral\tmechanism,\tor\trepresents\ta\tshift\tin\tthe\t 22\t balance\tof\tfilopodial\tvs\tlamellipodial\tregulatory\tproteins\tin\tthese\tcells\tis\tunclear.\t 23\t However\tit\tis\tprobable\tthat\tfilopodial\tand\tlamellal\tcup\tformation\tare\tnon- 24\t 25\t exclusive\textremes\tof\ta\tcontinuum\t-\tmuch\tas\tthey\tare\tduring\tcell\tmigration.\t\t \t 26\t The\tmulti-layered\tregulation\tof\tsmall\tGTPases\tis\tparticularly\timportant\twhen\t 27\t 28\t cells\tare\tchallenged\tto\tengulf\tparticles\tor\tmicrobes\tof\tdifferent\tshapes.\tThis\tis\t critical\tfor\tamoebae\tto\tfeed\ton\tdiverse\tbacteria\tor\timmune\tcells\tto\tbe\table\tto\t 29\t 30\t capture\tand\tkill\ta\twide\trange\tof\tpathogens,\tbut\thow\tcells\tadapt\tto\tdifferent\ttarget\t geometries\tis\tvery\tpoorly\tunderstood\t(Champion\tand\tMitragotri,\t2006;\t 31\t 32\t mutants\tdescribed\twith\ta\tgeometry-specific\tdefect\tin\tphagocytosis,\tunderlining\t Champion\tand\tMitragotri,\t2009).\tTo\tour\tknowledge,\tRGBARG-\tcells\tare\tthe\tfirst\t 33\t \t 19\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   1\t 2\t 3\t 4\t 5\t 6\t 7\t 8\t 9\t 10\t 11\t 12\t 13\t 14\t  All rights reserved. No reuse allowed without permission.   the\timportance\tof\tco-ordinating\tand\tbalancing\tRas\tand\tRac\tactivities.\tThis\tagain\t differs\tfrom\tthe\trole\tof\tNF1,\tas\tthe\tNF1-deficient\tAx2\tstrain\tused\tin\tthis\tstudy\tis\t able\tto\tefficiently\tengulf\tand\tgrow\ton\ta\twide\trange\tof\tbacteria\tincluding\t elongated\tstrains\tsuch\tas\tE.\tcoli\t(Buckley\tet\tal.,\t2019).\t\tIt\tis\tstill\tnot\tknown\thow\t other\tregulatory\telements\tor\tcytoskeletal\tcomponents\tadapt\tto\tdiffering\tshapes,\t but\tit\tseems\tlikely\tthat\tlarge-scale\trearrangements\tare\tnecessary\tto\t accommodate\tdifferent\ttargets.\t\t \t In\tsummary,\twe\tdescribe\ta\tmechanism\tto\tco-ordinate\tthe\tactivity\tof\tRac\tand\tRas\t family\tGTPases\tduring\tengulfment\tin\tDictyostelium.\tThe\tproteins\tthat\tmediate\t this\tco-ordination\tin\tmammalian\tcells\tremain\tunknown.\tHowever,\twe\tpropose\ta\t general\tmodel\tby\twhich\tspatial\tsignals\tand\teffectors\tfrom\tmultiple\tsmall\t GTPases\tintegrate\tto\tshape\tthe\tprotrusions\tthat\tform\tmacropinocytic\tand\t phagocytic\tcups,\tenabling\tcells\tto\tengulf\tdiverse\ttargets.\t\t  \t  \t  20\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   Methods  Dictyostelium\tculture\tand\tmolecular\tbiology\t  1\t 2\t Unless\totherwise\tstated,\tDictyostelium\tstrains\twere\tderived\tfrom\tthe\tMRC-Ax2\t 3\t 4\t axenic\tstrain\tprovided\tby\tthe\tKay\tlaboratory\tand\twere\troutinely\tcultured\tin\tfilter\t 5\t sterilised\tHL-5\tmedium\t(Formedium)\tat\t22°C.\t\tRacG\tmutants\tand\tcorresponding\t 6\t parental\tstrain\t(from\tthe\tDevreotes\tgroup,\tJohns\tHopkins,\tAx2D)\twere\tkind\tgifts\t 7\t were\tmeasured\tby\tseeding\tcells\tat\t0.5\tx\t105/ml\tin\tHL-5\tand\tcounting\tcell\tnumber\t from\tFrancisco\tRivero\t(University\tof\tHull)(Somesh\tet\tal.,\t2006b).\tGrowth\trates\t 8\t twice\tdaily\tfor\tthree\tdays.\tGrowth\trate\twas\tthen\tcalculated\tby\tfitting\tan\t 9\t exponential\tgrowth\tcurve\tusing\tGraphpad\tPrism\tsoftware.\tCells\twere\t 10\t 11\t transformed\tby\telectroporation:\t6\tx\t106\tcells\twere\tresuspended\tin\t0.4\tmls\tof\tice\t cold\tE-buffer\t(10\tmM\tKH2PO4\tpH\t6.1,\t50\tmM\tsucrose)\tand\ttransferred\tto\t2\tmm\t 12\t 13\t electroporation\tcuvette\tcontaining\tDNA\t(0.5\tµg\tfor\textrachromosomal\tplasmids,\t 15\tµg\tfor\tknockout\tvectors).\tCells\twere\tthen\telectroporated\tat\t1.2\tkV\tand\t3\tµF\t 14\t capacitance\twith\ta\t5\tΩ\tresistor\tin\tseries\tusing\ta\tBio-Rad\tGene\tPulser\tII.\tAfter\t24\t 15\t 16\t hours\ttransformants\twere\tselected\tin\teither\t20\tµg/ml\thygromycin\t(Invitrogen),\t 10\tµg/ml\tG418\t(Sigma)\tor\t10\tµg/ml\tblasticidin\t(Melford).\t 17\t 18\t \t 19\t Dictybase\t(www.dictybase.org)\t(Fey\tet\tal.,\t2013).\tCoding\tsequences\twere\tthen\t BAR\tdomain\tcontain\tproteins\twere\tidentified\tby\tmultiple\tBLAST\tsearches\tusing\t 20\t amplified\tby\tPCR\tfrom\tvegetative\tAx2\tcDNA\tadding\tcompatible\trestriction\tsites\t 21\t for\tsubcloning\tinto\tthe\tBglII/SpeI\tsites\tof\tthe\tN-\tand\tC-\tterminal\tGFP-fusion\t 22\t 23\t Dictyostelium\textrachromosomal\texpression\tvectors\tpDM1043\tand\tpDM1045\t (non-axenically\tselectable\tversions\tof\tthe\tpDM\tmodular\texpression\tsystem\t 24\t 25\t (Veltman\tet\tal.,\t2009)).\tTruncation\tand\tpoint\tmutants\tof\tRGBARG\twere\talso\t generated\tby\tPCR\tand\texpressed\tusing\tpDM1045.\tThe\tRGBARG\t(DDB_G0269934)\t 26\t 27\t knockout\tconstruct\twas\tgenerated\tby\tPCR\tfusion\tof\t~1Kb\t5’\tand\t3’\t recombination\tarms\twith\tthe\tfloxed\tblasticidin\tselection\tcassette\tfrom\t 28\t pDM1079,\tas\tdescribed\tin\tdetail\tin\t(Paschke\tet\tal.,\t2018).\tAfter\ttransformation,\t 29\t 30\t independent\tclones\twere\tobtain\tby\tdilute\tplating\tin\t96\twell\tplates.\tDisruption\tof\t the\tRGBARG\tlocus\twas\tscreened\tby\tPCR\tfrom\tgenomic\tDNA\tisolated\tfrom\t1\tx\t106\t 31\t 32\t NP40,\t0.45%\tTween\t20\tand\t0.4\tmg/ml\tProteinase\tK\t(NEB).\tAfter\t5\tminutes\t cells\tlysed\tin\t100\tμl\t10mM\tTris-\tHCl\tpH8.0,\t50\tmM\tKCl,\t2.5mM\tMgCl1,\t0.45%\t 33\t \t 21\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   Macropinocytosis\tassays\t  1\t minutes\tprior\tto\tPCR\tanalysis.\t\tThe\tRas\tbinding\tdomain\t(RBD)\tof\tPAK1-GFP\t incubation\tat\troom\ttemperature,\tthe\tproteinase\tK\twas\tdenatured\tat\t95°C\tfor\t10\t 2\t construct\tused\tas\tan\tactive\tRas\treporter\twas\ta\tgift\tfrom\tGareth\tBloomfield.\t 3\t 4\t \t 5\t To\tmeasure\tbulk\tfluid\tuptake\t2mg/ml\tFITC-dextran\t(70kDa;\tSigma)\twas\tadded\t 6\t 7\t to\tcells\tat\t5\tx\t106/ml\tin\tshaking\tculture.\tAt\teach\ttimepoint,\t500μl\tof\tcells\twere\t removed\tand\tadded\tto\t1ml\tice-cold\tKK2\t(16.5mM\tKH2PO4,\t3.8mM\tK2HPO4,\t 8\t 9\t pH6.1).\tCells\twere\tthen\tpelleted\tat\t7,000\tx\tg\tfor\t30\tseconds,\twashed\tonce\tin\tKK2\t and\tfrozen.\tPellet\twere\tthen\tlysed\tin\t200μl\t50mM\tNa2PO4\tpH9.3,\t0.2%\tTriton\t 10\t 11\t X100)\tand\tfluorescence\tmeasured\ton\ta\tplate\treader\tat\t485\texcitation/520nm\t emission.\tFluorescence\twas\tthen\tnormalised\tto\ttotal\tprotein\tin\tan\tadditional\t 12\t sample\tand\tcalculated\tas\ta\tpercentage\tof\twild-type\tcells\tat\t90\tminutes.\t 13\t 14\t Macropinosome\tvolume\twas\tmeasured\tby\tincubating\tcells\tfor\t5\tminutes\tin\t \t 15\t 0.1μg/ml\tFITC-dextran\tand\tobtaining\tZ-stacks\ton\ta\tspinning\tdisc\tconfocal\t 16\t microscope.\tFITC\tis\tpH-sensitive\tand\tthe\tsensitivity\twas\tset\tso\tonly\tnew\tnon- 17\t 18\t acidified\tmacropinosomes\twere\tvisible.\tFor\tanalysis,\tindividual\tcells\twere\t cropped\tout,\trandomised,\tand\tvolume\tcalculated\tfrom\tmanually\tmeasuring\tthe\t 19\t maximum\tdiameter\tof\teach\tmacropinosome\tin\teach\tcell,\tassuming\tthey\twere\t 20\t 21\t spherical.\t \t 22\t 23\t Phagocytosis\tof\tfluorescent\tbeads\twas\tmeasured\tby\tflow\tcytometry\tas\t 24\t 25\t previously\tdescribed\tin\tdetail\t(Sattler\tet\tal.,\t2013).\tBriefly,\t1\tor\t4.5\tμm\tdiameter\t YG-carboxylated\tpolystyrene\tbeads\t(Polysciences\tInc)\twere\tshaken\twith\t2\tx\t106\t 26\t 27\t Dictyostelium\t/ml\tat\tratios\tof\t200:1\tand\t10:1\trespectively.\t500\tμl\tsamples\twere\t 28\t removed\tat\teach\ttimepoint\tand\tadded\tto\t3\tml\tice-cold\tSorenson\tsorbitol\tbuffer\t (SSB;\t15\tmM\tKH2PO4,\t2\tmM\tNa2HPO4,\t120\tmM\tSorbitol)\tcontaining\t5mM\tsodium\t 29\t 30\t azide.\tSamples\twere\tthen\tcentrifuged\tat\t100\tx\tg\tfor\t10\tminutes,\tpellets\t resuspended\tin\tSSB\tand\tanalysed\ton\tan\tAttune\tNxT\tflow\tcytometer\t(Life\t 31\t Technologies).\tAnalysis\twas\tperformed\tusing\tFloJo\tsoftware\tas\tdescribed\t 32\t 33\t (Sattler\tet\tal.,\t2013).\t\t \t  Phagocytosis\tassays\t  22\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   1\t \t 2\t To\tmeasure\tuptake\tof\tGFP-expressing\tM.\tsmegmatis\tby\tflow\tcytometry,\tthe\t 3\t bacteria\twere\tgrown\tto\tan\tOD600\tof\t1,\tpelleted\tby\tcentrifugation\tat\t10,625\tx\tg\tfor\t 4\t 4\tminutes\tand\tresuspended\tin\t1ml\tHL5\tmedium.\tBacteria\tclumps\twere\tthen\t 5\t disrupted\tby\tpassing\tthrough\ta\t26-guage\tneedle\tseveral\ttimes,\tbefore\tadding\ta\t 6\t 1/10th\tvolume\tof\tbacteria\tto\ta\tDictyostelium\tculture\tand\tprocessing\tas\tabove.\t\t 7\t \t 8\t To\tmeasure\tphagocytosis\tof\tbacteria\tby\tdecreasing\tturbidity,\tan\tovernight\t 9\t bacterial\tculture\tin\tLB\twas\tdiluted\t1:25\tand\tgrown\tat\t37°C\tuntil\tan\tOD600\tof\t0.7\t 10\t before\tpelleting\tand\tresuspension\tin\tSSB\tat\tan\tOD600\tof\t0.8.\tThis\twas\tthen\tadded\t 11\t to\tan\tequal\tvolume\tof\tDictyostelium\tat\t2\tx\t107\tcells/ml\tin\tSSB\tat\troom\t 12\t temperature\tand\tshaken\tin\tflasks.\tThe\tOD600\twas\tthen\tmeasured\tover\ttime.\t\t 13\t \t 14\t Phagocytosis\tand\tTRITC\tlabeling\tof\theat\tkilled\tS.\tcerevisiae\twas\tperformed\t 15\t essentially\tas\tpreviously\tdescribed\t(Rivero\tand\tManiak,\t2006).\t\tDictyostelium\tat\t 16\t 1\tx\t106\tcells/ml\tin\tHL5\twere\tseeded\tin\tglass-bottomed\tmicroscopy\tdishes\t(Mat- 17\t minutes,\tthe\tfluorescence\tof\textracellular\tyeast\twas\tquenched\tby\taddition\tof\t0.2\t tek)\tand\tleft\tfor\t1\thour\tprior\tto\taddition\tof\ta\t5-fold\texcess\tof\tyeast.\tAfter\t30\t 18\t mg/ml\ttrypan\tblue\tand\timages\tof\tmultiple\tfields\tof\tview\ttaken\ton\ta\twide-field\t 19\t microscope\tscoring\t>100\tcells\tper\tcondition.\t\t 20\t 21\t \t 22\t Live\tcell\timaging\twas\tperformed\tin\tglass-bottomed\tmicroscopy\tdishes\t(Mat-Tek)\t 23\t with\tcells\tseeded\tthe\tpreceding\tday\tin\tfiltered\tHL-5\tmedium,\tunless\totherwise\t 24\t 25\t stated.\tSpinning\tdisc\timages\twere\tcaptured\tusing\ta\tPerkin-Elmer\tUltraview\tVoX\t 26\t spinning\tdisk\tmicroscope\twith\ta\tUplanSApo\t60x\toil\timmersion\tobjective\t(NA\t1.4)\t 27\t were\tobtained\tusing\ta\tZeiss\tLSM880\tAiryscan\tconfocal\tequipped\twith\ta\tFastscan\t and\tHammamatsu\tC9100-50-EM0CCD\tcamera.\tLaser\tscanning\tconfocal\timages\t 28\t detector,\tand\ta\t63x\t1.4\tNA\tobjective.\tImages\twere\tacquired\tin\tfastscan\tmode\tand\t 29\t 30\t deconvolved\tby\tairyprocessessing\tusing\tZen\tblack\tsoftware\t(Zeiss).\t \t 31\t Image\tanalysis\twas\tperformed\tusing\tImageJ\t(https://imageJ.nih.gov)\twith\t 32\t 33\t average\tplots\tof\tprotein\tenrichment\tacross\tcups\tgenerated\tusing\ta\tcustom\tscript\t \t 23\t  Microscopy\tand\timage\tanalysis\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   Western\tblotting\tand\tLipid\toverlay\tassays\t  in\tIgor\tPro\t(Wavemetrics).\tFor\tthis,\tconfocal\timages\twere\tcaptured\tand\tlinescans\t 1\t 2\t of\tGFP-fluorescence\tintensity\tmeasured\tfrom\tthe\tprotrusive\ttip\tto\ttip.\tThe\t 3\t measured\tas\twas\tthe\tlocal\tbackground\toutside\teach\tcell.\tLocal\tbackground\twas\t average\tsignal\tfrom\ta\t1-2\tμm\tnon-protruding\tregion\tof\tthe\tcell\twas\talso\t 4\t subtracted\tand\tsignal\tacross\tthe\tcup\tdivided\tby\tthe\tnon-protruding\tmembrane\t 5\t signal\tto\tgive\tfold-enrichment.\tTo\tcompare\tenrichment\tacross\tmultiple\tcups,\t 6\t 7\t normalised\tlinescans\twere\textrapolated\tover\t1000\tpoints\tand\taveraged.\t Enrichment\tat\tthe\tcup\ttip\twas\tmeasured\tby\tthe\taverage\tof\tthe\tfirst\t100\tpoints\tof\t 8\t 9\t the\tprofile\tfor\teach\tcup.\t \t 10\t 11\t The\tbacterial\tlong\taxis\twas\tmeasured\tautomatically\tfrom\twidefield\timages\tof\t either\tGFP\tor\tRFP\texpressing\tbacteria\tin\tImageJ.\tIndividual\tbacteria\twere\t 12\t identified\tby\tthresholding\tand\tlong\taxis\tmeasured\tusing\tthe\tFeret’s\tdiameter\t 13\t 14\t function.\t\t \t 15\t 16\t Western\tblotting\twas\tperformed\tby\tstandard\ttechniques,\tseparating\tproteins\tby\t 17\t 18\t SDS-PAGE\tand\tprobing\tusing\ta\tcustom\trabbit\tpolyclonal\tantibody\tto\tGFP\t(gift\t from\tAndrew\tPeden).\tEndogenously\tbiotinylated\tproteins\twere\tused\tas\ta\tloading\t 19\t control,\tusing\tAlexa680-labelled\tstreptavidin\t(Life\tTechnologies)(Davidson\tet\tal.,\t 20\t 21\t 2013).\tBlots\twere\timaged\tLiCor\todyssey\tSA\tfluorescence\tgel\timager.\t \t 22\t 23\t For\tlipid\toverlay\tassays,\t1\tx\t107\tAx2\tcells\texpressing\tthe\tBAR\tdomain\tfused\tto\t GFP\t(pCB114)\twere\twashed\tonce\tin\tSSB,\tlysed\tin\t600\tμl\tRIPA\tbuffer\t(50mM\t 24\t 25\t TrisHCl\tpH7.5,\t150mM\tNaCl,\t0.1%\tSDS,\t2mM\tEDTA,\t0.5%\tsodium\tdeoxycholate,\t 1\tx\tHALT\tprotease\tinhibitors\t(Thermo\tFisher),\t0.5%\tTriton\tX100)\tand\tleft\ton\tice\t 26\t for\t45\tminutes.\tInsoluble\tmaterial\twas\tthen\tremoved\tby\tcentrifugation\tat\t15,871\t 27\t 28\t x\tg\tfor\t20\tminutes\tat\t4°C.\tPIP\tstrips\tor\tArrays\t(Echelon\tBiosciences)\twere\t blocked\tin\t3%\tfatty\tacid-free\tbovine\tserum\talbumin\t(BSA)\tin\tTBS-T\t(20mM\tTris\t 29\t 30\t base,\t150mM\tNaCl,\t0.05%\tTween20,\tpH7.2).\tSamples\twere\tthen\tdiluted\tin\tTBS-T\t and\tincubated\twith\tthe\tstrips\tfor\t1\thour\tat\t22°C,\tbefore\twashing\tand\tprocessing\t 31\t as\tfor\tWestern\tblotting.\t\t 32\t 33\t \t \t 24\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   GAP\tand\tGEF\tbiochemistry\t  1\t 2\t Interactions\twith\trecombinant\tGST-Rac\tisoforms\twere\tperformed\tas\tdescribed\t previously\t(Plak\tet\tal.,\t2013).\tDictyostelium\tcells\texpressing\tGST-Rac\tbait\t 3\t 4\t proteins\tand\tGFP-fused\tto\tthe\tGEF\tdomain\tof\tRGBARG\twere\texpressed\tand\tlysed\t in\t2mls\tof\tbuffer\t(10mM\tNa2HPO4\tpH7.2,\t1%\tTriton\tX100,\t10%\tglycerol,\t150mM\t 5\t NaCl,\t10mM\tMgCl2,\t1mM\tEDTA,\t1mM\tNa3VO4,\t5mM\tNaF)\tincluding\tprotease\t 6\t 7\t inhibitor\tcocktail\t(Roche).\tLysates\twere\tmixed\twith\tglutathione\tsepharose\tbeads\t (GE\tHealthcare)\tand\tincubated\tovernight\tat\t4°C.\tUnbound\tproteins\twere\twashed\t 8\t 9\t away\twith\tPBS,\tand\tbound\tproteins\tdetected\tby\tWestern\tblot\tusing\tan\tanti-GFP\t antibody\t(SC9996).\t 10\t 11\t \t For\tGAP\tactivity\tmeasurements,\tHis-NF1\tGAP\tdomain\t(AA\t2530-3158)\tand\tMBP- 12\t His-RGBAR\tGAP\tdomain\t(AA\t1717-2045)\twere\tproduced\tand\tisolated\tfrom\tE.\t 13\t 14\t coli\tRosetta\tcells.\tHis-NF1\tGAP\twas\tpurified\tusing\ta\tHisTrap\texcel\t-\taffinity\t column\t(GE\tHealthcare)\tand\teluted\tin\tbuffer\tcontaining\t50mM\tTris,\t50mM\tNaCl,\t 15\t 16\t 5%\tGlycerol,\t3mM\tβ-mercaptoethanol\tand\t200mM\timidazole,\tpH7.5.\tMBP-His- RGBAR\tGAP,\twas\tpurified\tby\tMaltose\tBinding\tProtein\tTrap\t(MBPTrap)\t-\taffinity\t 17\t 18\t column\t(GE\tHealthcare)\tand\teluted\tin\t20mM\tTris,\t200mM\tNaCl,\t5%\tGlycerol\t 1mM\tβ-Mercaptoethanol\tand\t10mM\tMaltose,\tpH7.5.\tProteins\twere\tfurther\t 19\t purified\tby\tsize\texclusion\tchromatography\t(Sephacryl\t16/60,\tGE\tHealthcare)\tand\t 20\t 21\t stored\tin\t50mM\tTris,\t50mM\tNaCl,\t5mM\tDTT,\tand\t5mM\tMgCl2,\tpH7,5.1.\t \t 22\t 23\t 1µM\tof\tthe\tindicated\tRas\tproteins\twith\tand\twithout\tequal\tamount\tof\tindicated\t 24\t mM\tNaCl\tand\t5mM\tMgCl2.\tAt\teach\ttimepoint\tthe\tGDP\tcontent\tof\tthe\tsamples\twas\t GAP\tdomain\twas\tincubated\twith\t50\tµM\tof\tGTP\tat\t20°C\tin\t50\tmM\tTris\tpH\t7.5,\t50\t 25\t analysed\tby\ta\tHPLC\t(Thermo\tUltimate\t3000):\ta\treversed\tphase\tC18\tcolumn\twas\t 26\t employed\tto\tdetect\tGDP\tand\tGTP\tcontent\t(in\t%)\tas\tpreviously\tdescribed\t(Eberth\t 27\t 28\t and\tAhmadian,\t2009).\tLinear\trates\tof\tGDP\tproduction\t(first\t4-8\ttimepoints)\twere\t calculated\tusing\tGraFit\t5.0\t(Erithacus\tsoftware).\t 29\t 30\t \t 31\t 3µm\tunmodified\tnon-fluorescent\tpolystyrene\tbeads\t(Polysciences\tInc.)\twere\t 32\t 33\t embedded\tin\tpolyvinyl\talcohol\t(PVA)\tfilm\t(Sigma\tAldrich)\tand\tstretched\tas\t \t  Ellipsoid\tbeads\tgeneration\tand\tphagocytosis\t\t  25\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   1\t were\tadded\tto\t20mls\t25%\tw/w\tdissolved\tPVA\tsolution\tand\tpoured\tinto\ta\t10.5\tx\t previously\tdescribed\t(Ho\tet\tal.,\t1993).\tBriefly,\t2.8\tmls\tbeads\tof\tbead\tsolution\t 2\t 3\t with\ta\tgrid\tto\tfollow\tdeformation\tand\tplaced\tin\ta\tcustom\tstretching\tdevice\tas\t 10.5\tcm\tplastic\tmold\tto\tcreate\ta\tfilm.\tThese\twere\tcut\tinto\t3\tx\t2cm\tstrips,\tmarked\t 4\t described\tin\tdetail\tin\t(Ho\tet\tal.,\t1993).\tFilms\twere\tthe\tplaced\tin\ta\t145\t°C\toil\tbath\t 5\t to\tsoften\tbeads\tand\tfilm\tand\tslowly\tpulled\tto\tthe\tdesired\tlength.\tAfter\tcooling\t 6\t 7\t below\tthe\tglass\ttransition\tTg\ttemperature,\tthe\tbeads\twere\textracted\tfrom\tthe\t central\tregion\twhere\tthe\tgrid\twas\tdeformed\tevenly.\tThis\tpart\twas\tcut\tinto\tsmall\t 8\t 9\t pieces\tand\trotated\tin\t10\tmls\tof\ta\t3:7\tmix\tof\tisopropanol:water\tovernight\tat\t20°C\t 10\t washed\tin\tisopropanol:\twater.\tBeads\twere\tthen\twashed\ttwice\tin\t to\tdissolve.\tBeads\twere\taliquoted\tand\ttwice\theated\tat\t75°C\tfor\t10\tminutes\tand\t 11\t isopropanol:water\tat\t22\t°C,\tbefore\ttwo\twashes\tin\twater.\tThe\tamount\tof\tstretch\t 12\t was\tmeasured\tby\timaging\ton\tan\tinverted\tmicroscope\tand\tmanually\tmeasuring\t 13\t 14\t their\tlength\tin\tImageJ.\t\t \t 15\t To\tmeasure\tphagocytosis,\tequal\tnumbers\tof\tstretched\tand\tunstretched\tbeads\t 16\t were\tmixed,\tsonicated\tand\tincubated\tat\ta\t10-fold\texcess\tto\tcells\tat\t1\tx\t106\t/ml,\t 17\t 18\t shaking\tin\tHL5.\tAfter\t30\tminutes,\t500\tµl\tsamples\twere\tadded\tto\t3\tml\tSSB\twith\t 5mM\tsodium\tazide\tto\tdetach\tunengulfed\tbeads.\tCells\twere\twashed\tin\tice-cold\t 19\t SSB,\ttransferred\tto\ta\tmicroscopy\tdish\tand\tallowed\tto\tadhere\tfor\t10\tminutes\t 20\t before\timaging\tand\tthe\tnumber\tof\teach\tshape\tbead\tinternalised\tquantified\t 21\t manually.\t 22\t 23\t 24\t The\tauthors\twould\tlike\tto\tthank\tFrancisco\tRivero\tfor\tproviding\tthe\tRacG\tmutant\t cell\tline\tand\tplasmids,\tAndrew\tPeden\tfor\tthe\tGFP\tantibody,\tGareth\tBloomfied\t 25\t 26\t and\tRob\tKay\tfor\tNF1\tand\tGFP-RBD\tconstructs\tand\tIwan\tEvans\tfor\tthe\tRFP-E.\tcoli\t 27\t UF140624.\t\tMicroscopy\tstudies\twere\tsupported\tby\tUK\tMedical\tResearch\tCouncil\t strain.\tJSK\tis\tsupported\tby\tRoyal\tSociety\tUniversity\tResearch\tFellowship\t 28\t 29\t grant\t(G0700091)\tand\tWellcome\tTrust\tgrant\t(GR077544AIA).\t \t  Acknowledgements   26\t     \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   1\t 2\t 3\t 4\t 5\t 6\t 7\t 8\t 9\t 10\t 11\t 12\t 13\t 14\t 15\t 16\t 17\t 18\t 19\t 20\t 21\t 22\t 23\t 24\t 25\t 26\t 27\t 28\t 29\t 30\t 31\t 32\t 33\t 34\t 35\t 36\t 37\t 38\t 39\t 40\t 41\t 42\t 43\t 44\t 45\t 46\t 47\t  \t  References   \t Amyere,\tM.,\tB.\tPayrastre,\tU.\tKrause,\tP.\tVan\tDer\tSmissen,\tA.\tVeithen,\tand\tP.J.\t Courtoy.\t2000.\tConstitutive\tmacropinocytosis\tin\toncogene-transformed\t fibroblasts\tdepends\ton\tsequential\tpermanent\tactivation\tof\t phosphoinositide\t3-kinase\tand\tphospholipase\tC.\tMol\tBiol\tCell.\t11:3453- 3467.\t Araki,\tN.,\tY.\tEgami,\tY.\tWatanabe,\tand\tT.\tHatae.\t2007.\tPhosphoinositide\t metabolism\tduring\tmembrane\truffling\tand\tmacropinosome\tformation\tin\t EGF-stimulated\tA431\tcells.\tExp\tCell\tRes.\t313:1496-1507.\t Araki,\tN.,\tM.T.\tJohnson,\tand\tJ.A.\tSwanson.\t1996.\tA\trole\tfor\tphosphoinositide\t3- kinase\tin\tthe\tcompletion\tof\tmacropinocytosis\tand\tphagocytosis\tby\t macrophages.\tJ\tCell\tBiol.\t135:1249-1260.\t Aspenstrom,\tP.\t2014.\tBAR\tdomain\tproteins\tregulate\tRho\tGTPase\tsignaling.\tSmall\t GTPases.\t5:7.\t Bar-Sagi,\tD.,\tand\tJ.R.\tFeramisco.\t1986.\tInduction\tof\tmembrane\truffling\tand\tfluid- phase\tpinocytosis\tin\tquiescent\tfibroblasts\tby\tras\tproteins.\tScience.\t 233:1061-1068.\t Bloomfield,\tG.,\tand\tR.R.\tKay.\t2016.\tUses\tand\tabuses\tof\tmacropinocytosis.\tJ\tCell\t Sci.\t129:2697-2705.\t Bloomfield,\tG.,\tD.\tTraynor,\tS.P.\tSander,\tD.M.\tVeltman,\tJ.A.\tPachebat,\tand\tR.R.\tKay.\t 2015.\tNeurofibromin\tcontrols\tmacropinocytosis\tand\tphagocytosis\tin\t Dictyostelium.\teLife.\t4.\t Bos,\tJ.L.,\tH.\tRehmann,\tand\tA.\tWittinghofer.\t2007.\tGEFs\tand\tGAPs:\tcritical\t elements\tin\tthe\tcontrol\tof\tsmall\tG\tproteins.\tCell.\t129:865-877.\t Buckley,\tC.M.,\tand\tJ.S.\tKing.\t2017.\tDrinking\tproblems:\tMechanisms\tof\t macropinosome\tformation\tand\tmaturation.\tThe\tFEBS\tJournal.\t Buckley,\tC.M.,\tVictoria\tL\tHeath,\tA.\tGueho,\tC.\tBosmani,\tP.\tKnobloch,\tP.\tSikakana,\tN.\t Personnic,\tS.K.\tDove,\tR.H.\tMichell,\tR.\tMeier,\tH.\tHilbi,\tT.\tSoldati,\tR.H.\tInsall,\t and\tJ.S.\tKing.\t2019.\tPIKfyve/Fab1\tis\trequired\tfor\tefficient\tV-ATPase\tand\t hydrolase\tdelivery\tto\tphagosomes,\tphagosomal\tkilling,\tand\trestriction\tof\t Legionella\tinfection.\tPloS\tpathogens.\tPLOS\tPathogens.\tIn\tpress.\t Caron,\tE.,\tand\tA.\tHall.\t1998.\tIdentification\tof\ttwo\tdistinct\tmechanisms\tof\t phagocytosis\tcontrolled\tby\tdifferent\tRho\tGTPases.\tScience.\t282:1717- 1721.\t Champion,\tJ.A.,\tand\tS.\tMitragotri.\t2006.\tRole\tof\ttarget\tgeometry\tin\tphagocytosis.\t Proc\tNatl\tAcad\tSci\tU\tS\tA.\t103:4930-4934.\t Champion,\tJ.A.,\tand\tS.\tMitragotri.\t2009.\tShape\tinduced\tinhibition\tof\tphagocytosis\t of\tpolymer\tparticles.\tPharm\tRes.\t26:244-249.\t Chubb,\tJ.R.,\tA.\tWilkins,\tG.M.\tThomas,\tand\tR.H.\tInsall.\t2000.\tThe\tDictyostelium\t RasS\tprotein\tis\trequired\tfor\tmacropinocytosis,\tphagocytosis\tand\tthe\t control\tof\tcell\tmovement.\tJ\tCell\tSci.\t113\t(\tPt\t4):709-719.\t Commisso,\tC.,\tS.M.\tDavidson,\tR.G.\tSoydaner-Azeloglu,\tS.J.\tParker,\tJ.J.\tKamphorst,\t S.\tHackett,\tE.\tGrabocka,\tM.\tNofal,\tJ.A.\tDrebin,\tC.B.\tThompson,\tJ.D.\t Rabinowitz,\tC.M.\tMetallo,\tM.G.\tVander\tHeiden,\tand\tD.\tBar-Sagi.\t2013.\t Macropinocytosis\tof\tprotein\tis\tan\tamino\tacid\tsupply\troute\tin\tRas- transformed\tcells.\tNature.\t497:633-637.\t  27\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   Condon,\tN.D.,\tJ.M.\tHeddleston,\tT.L.\tChew,\tL.\tLuo,\tP.S.\tMcPherson,\tM.S.\tIoannou,\tL.\t 1\t Hodgson,\tJ.L.\tStow,\tand\tA.A.\tWall.\t2018.\tMacropinosome\tformation\tby\ttent\t 2\t pole\truffling\tin\tmacrophages.\tJ\tCell\tBiol.\t 3\t Cox,\tD.,\tP.\tChang,\tQ.\tZhang,\tP.G.\tReddy,\tG.M.\tBokoch,\tand\tS.\tGreenberg.\t1997.\t 4\t 5\t Requirements\tfor\tboth\tRac1\tand\tCdc42\tin\tmembrane\truffling\tand\t phagocytosis\tin\tleukocytes.\tThe\tJournal\tof\texperimental\tmedicine.\t 6\t 186:1487-1494.\t 7\t Cox,\tD.,\tC.C.\tTseng,\tG.\tBjekic,\tand\tS.\tGreenberg.\t1999.\tA\trequirement\tfor\t 8\t 9\t phosphatidylinositol\t3-kinase\tin\tpseudopod\textension.\tJ\tBiol\tChem.\t 10\t Davidson,\tA.J.,\tJ.S.\tKing,\tand\tR.H.\tInsall.\t2013.\tThe\tuse\tof\tstreptavidin\tconjugates\t 274:1240-1247.\t 11\t as\timmunoblot\tloading\tcontrols\tand\tmitochondrial\tmarkers\tfor\tuse\twith\t 12\t Dictyostelium\tdiscoideum.\tBiotechniques.\t55:39-41.\t 13\t Dawson,\tJ.C.,\tJ.A.\tLegg,\tand\tL.M.\tMachesky.\t2006.\tBar\tdomain\tproteins:\ta\trole\tin\t 14\t 15\t tubulation,\tscission\tand\tactin\tassembly\tin\tclathrin-mediated\tendocytosis.\t Trends\tCell\tBiol.\t16:493-498.\t 16\t Dumontier,\tM.,\tP.\tHocht,\tU.\tMintert,\tand\tJ.\tFaix.\t2000.\tRac1\tGTPases\tcontrol\t 17\t 18\t filopodia\tformation,\tcell\tmotility,\tendocytosis,\tcytokinesis\tand\t development\tin\tDictyostelium.\tJ\tCell\tSci.\t113\t(\tPt\t12):2253-2265.\t 19\t 20\t Eberth,\tA.,\tand\tM.R.\tAhmadian.\t2009.\tIn\tvitro\tGEF\tand\tGAP\tassays.\tCurr\tProtoc\t Cell\tBiol.\tChapter\t14:Unit\t14\t19.\t 21\t Eden,\tS.,\tR.\tRohatgi,\tA.V.\tPodtelejnikov,\tM.\tMann,\tand\tM.W.\tKirschner.\t2002.\t 22\t 23\t Mechanism\tof\tregulation\tof\tWAVE1-induced\tactin\tnucleation\tby\tRac1\tand\t 24\t Nck.\tNature.\t418:790-793.\t Fey,\tP.,\tR.J.\tDodson,\tS.\tBasu,\tand\tR.L.\tChisholm.\t2013.\tOne\tstop\tshop\tfor\t 25\t everything\tDictyostelium:\tdictyBase\tand\tthe\tDicty\tStock\tCenter\tin\t2012.\t 26\t 27\t Methods\tMol\tBiol.\t983:59-92.\t Fujii,\tM.,\tK.\tKawai,\tY.\tEgami,\tand\tN.\tAraki.\t2013.\tDissecting\tthe\troles\tof\tRac1\t 28\t activation\tand\tdeactivation\tin\tmacropinocytosis\tusing\tmicroscopic\tphoto- 29\t 30\t manipulation.\tScientific\treports.\t3:2385.\t Griffin,\tF.M.,\tJr.,\tJ.A.\tGriffin,\tJ.E.\tLeider,\tand\tS.C.\tSilverstein.\t1975.\tStudies\ton\tthe\t 31\t mechanism\tof\tphagocytosis.\tI.\tRequirements\tfor\tcircumferential\t 32\t attachment\tof\tparticle-bound\tligands\tto\tspecific\treceptors\ton\tthe\t 33\t 34\t macrophage\tplasma\tmembrane.\tThe\tJournal\tof\texperimental\tmedicine.\t 35\t Ho,\tC.C.,\tA.\tKeller,\tJ.A.\tOdell,\tand\tR.H.\tOttewill.\t1993.\tPreparation\tof\tMonodisperse\t 142:1263-1282.\t 36\t 37\t Hoeller,\tO.,\tP.\tBolourani,\tJ.\tClark,\tL.R.\tStephens,\tP.T.\tHawkins,\tO.D.\tWeiner,\tG.\t Ellipsoidal\tPolystyrene\tParticles.\tColloid\tPolym\tSci.\t271:469-479.\t 38\t Weeks,\tand\tR.R.\tKay.\t2013.\tTwo\tdistinct\tfunctions\tfor\tPI3-kinases\tin\t 39\t 40\t Hoppe,\tA.D.,\tand\tJ.A.\tSwanson.\t2004.\tCdc42,\tRac1,\tand\tRac2\tdisplay\tdistinct\t macropinocytosis.\tJ\tCell\tSci.\t126:4296-4307.\t 41\t patterns\tof\tactivation\tduring\tphagocytosis.\tMol\tBiol\tCell.\t15:3509-3519.\t 42\t Inaba,\tH.,\tK.\tYoda,\tand\tH.\tAdachi.\t2017.\tThe\tF-actin-binding\tRapGEF\tGflB\tis\t 43\t required\tfor\tefficient\tmacropinocytosis\tin\tDictyostelium.\tJ\tCell\tSci.\t 44\t 45\t 130:3158-3172.\t Janetopoulos,\tC.,\tJ.\tBorleis,\tF.\tVazquez,\tM.\tIijima,\tand\tP.\tDevreotes.\t2005.\t 46\t Temporal\tand\tspatial\tregulation\tof\tphosphoinositide\tsignaling\tmediates\t 47\t 48\t cytokinesis.\tDev\tCell.\t8:467-477.\t \t 28\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   Junemann,\tA.,\tV.\tFilic,\tM.\tWinterhoff,\tB.\tNordholz,\tC.\tLitschko,\tH.\tSchwellenbach,\t 1\t T.\tStephan,\tI.\tWeber,\tand\tJ.\tFaix.\t2016.\tA\tDiaphanous-related\tformin\tlinks\t 2\t Ras\tsignaling\tdirectly\tto\tactin\tassembly\tin\tmacropinocytosis\tand\t 3\t phagocytosis.\tProc\tNatl\tAcad\tSci\tU\tS\tA.\t113:E7464-E7473.\t 4\t 5\t Kaplan,\tG.\t1977.\tDifferences\tin\tthe\tmode\tof\tphagocytosis\twith\tFc\tand\tC3\t receptors\tin\tmacrophages.\tScand\tJ\tImmunol.\t6:797-807.\t 6\t Khosla,\tM.,\tG.B.\tSpiegelman,\tR.\tInsall,\tand\tG.\tWeeks.\t2000.\tFunctional\toverlap\tof\t 7\t the\tdictyostelium\tRasG,\tRasD\tand\tRasB\tproteins.\tJ\tCell\tSci.\t113\t(\tPt\t 8\t 9\t 8):1427-1434.\t King,\tJ.S.,\tand\tR.R.\tKay.\t2019.\tThe\torigins\tand\tevolution\tof\tmacropinocytosis.\t 10\t Philos\tTrans\tR\tSoc\tLond\tB\tBiol\tSci.\t374:20180158.\t 11\t Machesky,\tL.M.,\tand\tR.H.\tInsall.\t1998.\tScar1\tand\tthe\trelated\tWiskott-Aldrich\t 12\t 13\t syndrome\tprotein,\tWASP,\tregulate\tthe\tactin\tcytoskeleton\tthrough\tthe\t 14\t Marinovic,\tM.,\tL.\tMijanovic,\tM.\tSostar,\tM.\tVizovisek,\tA.\tJunemann,\tM.\tFonovic,\tB.\t Arp2/3\tcomplex.\tCurr\tBiol.\t8:1347-1356.\t 15\t 16\t Turk,\tI.\tWeber,\tJ.\tFaix,\tand\tV.\tFilic.\t2019.\tIQGAP-related\tprotein\tIqgC\t suppresses\tRas\tsignaling\tduring\tlarge-scale\tendocytosis.\tProc\tNatl\tAcad\t 17\t Sci\tU\tS\tA.\t116:1289-1298.\t 18\t Marshall,\tJ.G.,\tJ.W.\tBooth,\tV.\tStambolic,\tT.\tMak,\tT.\tBalla,\tA.D.\tSchreiber,\tT.\tMeyer,\t 19\t 20\t and\tS.\tGrinstein.\t2001.\tRestricted\taccumulation\tof\tphosphatidylinositol\t3- kinase\tproducts\tin\ta\tplasmalemmal\tsubdomain\tduring\tFc\tgamma\t 21\t 22\t Massol,\tP.,\tP.\tMontcourrier,\tJ.C.\tGuillemot,\tand\tP.\tChavrier.\t1998.\tFc\treceptor- receptor-mediated\tphagocytosis.\tJ\tCell\tBiol.\t153:1369-1380.\t 23\t mediated\tphagocytosis\trequires\tCDC42\tand\tRac1.\tEMBO\tJ.\t17:6219-6229.\t 24\t Nobes,\tC.D.,\tand\tA.\tHall.\t1995.\tRho,\trac,\tand\tcdc42\tGTPases\tregulate\tthe\tassembly\t 25\t 26\t of\tmultimolecular\tfocal\tcomplexes\tassociated\twith\tactin\tstress\tfibers,\t lamellipodia,\tand\tfilopodia.\tCell.\t81:53-62.\t 27\t Norbury,\tC.C.,\tL.J.\tHewlett,\tA.R.\tPrescott,\tN.\tShastri,\tand\tC.\tWatts.\t1995.\tClass\tI\t 28\t MHC\tpresentation\tof\texogenous\tsoluble\tantigen\tvia\tmacropinocytosis\tin\t 29\t 30\t bone\tmarrow\tmacrophages.\tImmunity.\t3:783-791.\t Paschke,\tP.,\tD.A.\tKnecht,\tA.\tSilale,\tD.\tTraynor,\tT.D.\tWilliams,\tP.A.\tThomason,\tR.H.\t 31\t 32\t Insall,\tJ.R.\tChubb,\tR.R.\tKay,\tand\tD.M.\tVeltman.\t2018.\tRapid\tand\tefficient\t genetic\tengineering\tof\tboth\twild\ttype\tand\taxenic\tstrains\tof\tDictyostelium\t 33\t 34\t discoideum.\tPLoS\tOne.\t13:e0196809.\t Peter,\tB.J.,\tH.M.\tKent,\tI.G.\tMills,\tY.\tVallis,\tP.J.\tButler,\tP.R.\tEvans,\tand\tH.T.\tMcMahon.\t 35\t 2004.\tBAR\tdomains\tas\tsensors\tof\tmembrane\tcurvature:\tthe\tamphiphysin\t 36\t 37\t BAR\tstructure.\tScience.\t303:495-499.\t Plak,\tK.,\tD.\tVeltman,\tF.\tFusetti,\tJ.\tBeeksma,\tF.\tRivero,\tP.J.\tVan\tHaastert,\tand\tA.\t 38\t Kortholt.\t2013.\tGxcC\tconnects\tRap\tand\tRac\tsignaling\tduring\tDictyostelium\t 39\t development.\tBMC\tCell\tBiol.\t14:6.\t 40\t 41\t Rivero,\tF.,\tand\tM.\tManiak.\t2006.\tQuantitative\tand\tmicroscopic\tmethods\tfor\t studying\tthe\tendocytic\tpathway.\tMethods\tMol\tBiol.\t346:423-438.\t 42\t Sallusto,\tF.,\tM.\tCella,\tC.\tDanieli,\tand\tA.\tLanzavecchia.\t1995.\tDendritic\tcells\tuse\t 43\t macropinocytosis\tand\tthe\tmannose\treceptor\tto\tconcentrate\t 44\t 45\t macromolecules\tin\tthe\tmajor\thistocompatibility\tcomplex\tclass\tII\t compartment:\tdownregulation\tby\tcytokines\tand\tbacterial\tproducts.\tThe\t 46\t Journal\tof\texperimental\tmedicine.\t182:389-400.\t 47\t 48\t Sattler,\tN.,\tR.\tMonroy,\tand\tT.\tSoldati.\t2013.\tQuantitative\tanalysis\tof\tphagocytosis\t and\tphagosome\tmaturation.\tMethods\tMol\tBiol.\t983:383-402.\t 49\t \t 29\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   1\t Schlam,\tD.,\tR.D.\tBagshaw,\tS.A.\tFreeman,\tR.F.\tCollins,\tT.\tPawson,\tG.D.\tFairn,\tand\tS.\t 2\t Grinstein.\t2015.\tPhosphoinositide\t3-kinase\tenables\tphagocytosis\tof\tlarge\t 3\t particles\tby\tterminating\tactin\tassembly\tthrough\tRac/Cdc42\tGTPase- 4\t activating\tproteins.\tNat\tCommun.\t6:8623.\t 5\t Somesh,\tB.P.,\tC.\tNeffgen,\tM.\tIijima,\tP.\tDevreotes,\tand\tF.\tRivero.\t2006a.\t 6\t Dictyostelium\tRacH\tregulates\tendocytic\tvesicular\ttrafficking\tand\tis\t 7\t required\tfor\tlocalization\tof\tvacuolin.\tTraffic.\t7:1194-1212.\t 8\t Somesh,\tB.P.,\tG.\tVlahou,\tM.\tIijima,\tR.H.\tInsall,\tP.\tDevreotes,\tand\tF.\tRivero.\t2006b.\t 9\t RacG\tregulates\tmorphology,\tphagocytosis,\tand\tchemotaxis.\tEukaryot\tCell.\t 10\t 5:1648-1663.\t 11\t Swanson,\tJ.A.\t2008.\tShaping\tcups\tinto\tphagosomes\tand\tmacropinosomes.\tNat\t 12\t Rev\tMol\tCell\tBiol.\t9:639-649.\t 13\t Swanson,\tJ.A.,\tand\tJ.S.\tKing.\t2019.\tThe\tbreadth\tof\tmacropinocytosis\tresearch.\t 14\t Philos\tTrans\tR\tSoc\tLond\tB\tBiol\tSci.\t374:20180146.\t 15\t Tollis,\tS.,\tA.E.\tDart,\tG.\tTzircotis,\tand\tR.G.\tEndres.\t2010.\tThe\tzipper\tmechanism\tin\t 16\t phagocytosis:\tenergetic\trequirements\tand\tvariability\tin\tphagocytic\tcup\t 17\t shape.\tBMC\tSyst\tBiol.\t4:149.\t 18\t Veltman,\tD.M.,\tG.\tAkar,\tL.\tBosgraaf,\tand\tP.J.\tVan\tHaastert.\t2009.\tA\tnew\tset\tof\t 19\t small,\textrachromosomal\texpression\tvectors\tfor\tDictyostelium\t 20\t discoideum.\tPlasmid.\t61:110-118.\t 21\t Veltman,\tD.M.,\tM.G.\tLemieux,\tD.A.\tKnecht,\tand\tR.H.\tInsall.\t2014.\tPIP(3)- 22\t dependent\tmacropinocytosis\tis\tincompatible\twith\tchemotaxis.\tJ\tCell\tBiol.\t 23\t 204:497-505.\t 24\t Veltman,\tD.M.,\tT.D.\tWilliams,\tG.\tBloomfield,\tB.C.\tChen,\tE.\tBetzig,\tR.H.\tInsall,\tand\t 25\t R.R.\tKay.\t2016.\tA\tplasma\tmembrane\ttemplate\tfor\tmacropinocytic\tcups.\t 26\t eLife.\t5.\t 27\t Vieira,\tO.V.,\tR.J.\tBotelho,\tL.\tRameh,\tS.M.\tBrachmann,\tT.\tMatsuo,\tH.W.\tDavidson,\tA.\t Schreiber,\tJ.M.\tBacker,\tL.C.\tCantley,\tand\tS.\tGrinstein.\t2001.\tDistinct\troles\tof\t 28\t 29\t class\tI\tand\tclass\tIII\tphosphatidylinositol\t3-kinases\tin\tphagosome\t 30\t formation\tand\tmaturation.\tJ\tCell\tBiol.\t155:19-25.\t Vlahou,\tG.,\tand\tF.\tRivero.\t2006.\tRho\tGTPase\tsignaling\tin\tDictyostelium\t 31\t 32\t West,\tM.A.,\tA.R.\tPrescott,\tE.L.\tEskelinen,\tA.J.\tRidley,\tand\tC.\tWatts.\t2000.\tRac\tis\t discoideum:\tinsights\tfrom\tthe\tgenome.\tEur\tJ\tCell\tBiol.\t85:947-959.\t 33\t 34\t required\tfor\tconstitutive\tmacropinocytosis\tby\tdendritic\tcells\tbut\tdoes\tnot\t control\tits\tdownregulation.\tCurr\tBiol.\t10:839-848.\t 35\t Williams,\tT.D.,\tS.Y.\tPeak-Chew,\tP.\tPaschke,\tand\tR.R.\tKay.\t2019.\tAkt\tand\tSGK\t 36\t protein\tkinases\tare\trequired\tfor\tefficient\tfeeding\tby\tmacropinocytosis.\tJ\t 37\t 38\t Williamson,\tC.D.,\tand\tJ.G.\tDonaldson.\t2019.\tArf6,\tJIP3,\tand\tdynein\tshape\tand\t Cell\tSci.\t132.\t 39\t 40\t mediate\tmacropinocytosis.\tMol\tBiol\tCell.\t30:1477-1489.\t 41\t \t  \t  \t  30\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   Supplementary videos   1\t 2\t \t 3\t Video\t1:\tRGBARG-GFP\tis\tenriched\tat\tthe\ttips\tof\tmacropinocytic\tcups.\tMovie\tof\t 4\t RGBARG-\tcells\texpressing\tRGBarG-GFP\tin\taxenic\tculture.\t 5\t \t 6\t Video\t2:\tRGBARG-GFP\tis\tenriched\tat\tthe\ttips\tof\tphagocytic\tcups.\tRGBARG-\tcells\t 7\t expressing\tRGBarG-GFP\tengulfing\ta\tTRITC-labelled\tbudded\tyeast.\t 8\t \t Video\t3:\tRGBARG-GFP\tlocalises\tto\tthe\tperiphery\tof\tPIP3\tpatches\tduring\t 9\t macropinocytosis.\t3D\tmovie\tof\tcells\tmacropinosome\tformation\tin\tRGBARG-\tcells\t 10\t expressing\tRGBARG-GFP\tand\tPHCRAC-RFP.\t 11\t 12\t \t Video\t4:\tPIP3\tdynamics\tand\tcup\tformation\tin\tAx2\tcells.\tMaximum\tintensity\t 13\t projection\tof\tAx2\tcells\texpressing\tPHCRAC-GFP,\tshowing\tremoval\tof\tPIP3\t 14\t signalling\tafter\tmacropinosome\tformation\tis\tcomplete.\t 15\t 16\t \t Video\t5:\tRGBARG-\tcells\thave\tlarger\tand\tmore\tpersistent\tPIP3\tpatches.\tMaximum\t 17\t intensity\tprojection\tof\tcells\texpressing\tPHCRAC-GFP.\t 18\t 19\t \t Video\t6:\tPhagocytosis\tof\tTRITC-labelled\tyeast\tby\tAx2\tcells\texpressing\tPHCRAC- 20\t GFP.\t\t 21\t \t 22\t 23\t Video\t7:\tPhagocytosis\tof\tTRITC-labelled\tyeast\tby\tRGBARG-\tcells\texpressing\t PHCRAC-GFP.\t 24\t  \t  31\t  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   A  N  B  kbp  6 3  5' Recombination arm  3' Recombination arm  RCC1 RhoGEF BAR  RasGAP  2253  3216 3240 3852  3996 4584  5145  6132  C  Independent clones  JSK02  JSK03  5' arm  bsr  3' arm  KO screen primers  Figure 2 supplement: Disruption of DDB_G0269934. (A) Schematic of the genomic  locus indicating the position of the regions encoding each domain, and the 5’ and 3’  recombination arms amplified by PCR. These were attached either side of a blasticidin  selection cassette by fusion PCR and used to transform Ax2 cells and delete ~3 kbp of  the gene. (D) PCR screen of transformants, using one primer within the 5’recombination  arm and another after the 3’ arm. Clones with DDB_G0269934 disrupted will give a  product of 3.1 kbp (green box), the wild-type locus is 6.1 kbp (blue boxes).  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   A  B  GFP-RBD  Ax2  RGBARG-  )  %  (   y c n e u q e r f    e v i t  l  a e R  40  30  20  10  0  n=83  100 200 300 400 500  0 PHCRAC-GFP patch lifetime (s)  Figure 3 supplement: (A) Distribution of the active Ras probe RBD-GFP  in wild-type and RGBARG- cells. Images are single confocal planes, bar  indicates 5 μm. (B) Histogram of PHCRAC-GFP patch lifetime in Ax2 cells  from maximum projection movies. Lifetime was measured from the first  frame an independent patch was visible to when it was completely  removed from both the surface and any internalized vesicle.  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   A  RGBARG(225kD)  RGBARGΔRCC1(189kD)  RGBARGΔGEF(201kD)  RGBARGΔBAR(202kD)  RGBARGΔGAP(188kD)  RGBARGR1792K  (225kD)  RCC1(35kDa)  RhoGEF(22kDa)  BAR(22kDa)  RasGAP(36kDa)  RCC1  RhoGEF  BAR  751  10721080 1284 13321528  RasGAP  1715  2044  RhoGEF 757  961 1009 1205  BAR  RasGAP  1393  1722  RCC1  BAR  RasGAP  751  1072  1121 1317  1505  1834  RCC1  RhoGEF 1072 1080 1284  751  RasGAP  1517  1846  RCC1  RhoGEF 1072 1080 1284  751  BAR  1332 1528  R1792K  RCC1  RhoGEF  BAR  751  10721080 1284 13321529  RasGAP  1715  2044  RCC1  1  321  RhoGEF 204 1  BAR  1  197  RasGAP  1  329  Figure 4 Supplement 1: Schematic of the RGBARG truncation and point mutants  used in this study.  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   A  B  C  RGBARG∆GAP-GFP  PHCRAC-RFP  Merge  RGBARGR1792k-GFP  PHCRAC-RFP  Merge  Ax2  RGBARG-  RGBARGR1792K-GFP  D  )  m μ (    e z s    i  h c t  a p 3     P P  I  PHCRAC-GFP  PHCRAC-GFP  40  30  20  10  0  ****  *  ns  Ax2  RGBARG-  -GFP RGBARG-GFP RGBARG- RGBARG- RGBARG R1792K  Figure 4 Supplement 2: (A) Co-expression of RGBARGdRasGAP-GFP and PHCRAC-RFP  in RGBARG- cells indicating that the RasGAP domain helps excluded RGBARG-GFP from  PIP3 rich regions of the cell. (B) The GAP activity inactivating point mutant R1792K localizes  normally and is excluded from the base of protruding cups (arrowhead). (C) PHCRAC-GFP  localization in Ax2 and RGBARG- cells expressing RGBARGR1792K-GFP, demonstrating  that PIP3 dynamics are not rescued by this construct. PHCRAC-GFP patch size is quantified  in (D).  \f",
      "bioRxiv preprint first posted online Sep. 10, 2019;  (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/763748 .   The copyright holder for this preprint  doi:   All rights reserved. No reuse allowed without permission.   A  B  C  PI  PI(3)P  PI(4)P  PI(5)P  PI(3,4)P2  PI(3,5)P2  PI(4,5)P2  PI(3,4,5)P3  100  50  25  12.5 6.25 3.13  1.56 pmol/spot  )  i  m µ (   e z s   h c t a p   s a R e v i t c A     ) 3 m µ (   e m u o v  l  15  10  5  0  8  6  4  2  0  Ax2D  RacG-  Ax2 D RacG- #2 RacG- #6  Figure 5 Supplement: (A) PIP array analysis of BAR-GFP binding showing a moderate  preference for PI(3,4)P2 in this assay. (B) Active Ras signaling during macropinosome  formation in RacG- cells and their parental cell line Ax2D. Patch size was quantified  from single confocal planes of cells expressing RBD-GFP. (C) Volume of macropino- somes formed by RacG- and control cells. Measured by imaging cells taking up FITC  dextran. Two independent RacG- clones were analysed over 3 independent experi- ments.  \f",
      "\n"
     ]
    }
   ],
   "source": [
    "print(text_out)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://arxiv.org/pdf/1112.1724.pdf\n"
     ]
    }
   ],
   "source": [
    "#example of doing above in one line\n",
    "dfplay = df_arxiv.loc[[4]]\n",
    "dfplay['body'] = dfplay['id'].apply(lambda x: PdfConverter(file_path=download_file(\"https://arxiv.org/pdf/%s.pdf\" %x)).convert_pdf_to_txt().replace('\\n',' '))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "dfplay.body[10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://www.biorxiv.org/content/biorxiv/early/2019/04/04/445486.full.pdf\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/04/04/596551.full.pdf\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/04/04/597369.full.pdf\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/04/04/585455.full.pdf\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/04/04/598409.full.pdf\n"
     ]
    }
   ],
   "source": [
    "#short example of looping through:\n",
    "df_play = df_arxiv.iloc[0:5].copy()\n",
    "df_play['body'] = ''\n",
    "for ix in range(len(df_play.id)):\n",
    "    #df_play['body'].loc[ix] = PdfConverter(file_path=download_file(\"https://arxiv.org/pdf/%s.pdf\" %df_play['id'].loc[ix])).convert_pdf_to_txt().replace('\\n',' ')\n",
    "    df_play['body'].loc[ix] = PdfConverter(file_path=download_file(df_biorxiv.atom_path.loc[ix].replace('/biorxiv/', \n",
    "                                                                                                               'https://www.biorxiv.org/content/biorxiv/').replace( '.atom','')+'.full.pdf')).convert_pdf_to_txt().replace('\\n',' ')\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'bioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  1  2  3  4  5  6  7  8  9  10 11  12 13 14 15  16  Proposal for a Global Adherence Scale for Acute Conditions (GASAC): a prospective   cohort study in two Emergency Departments  Mélanie Sustersic1,2*, Aurélie Gauchet3, Amélie Duvert4☯, Laure Gonnet4☯, Alison Foote4, Céline   Vermorel1, Benoit Allenet1, Jean-Luc Bosson1.   1 TIMC-IMAG,  CNRS-UMR 5525, Univ. Grenoble Alpes, Grenoble, F-38043 France   2 Emergency Department, Grenoble Mutualist Hospital Group (Groupe Hospitalier Mutualiste de   Grenoble), Grenoble, F-38000 France   3 Inter-University Psychology Laboratory, EA 4145, Univ. Grenoble Alpes, Grenoble, F-38040 France  4 Research Division, Grenoble Alpes University Hospital, Grenoble, F-38043 France   * Corresponding author  E-mail: melanie.sustersic@gmail.com (MS)   17 ☯ These authors contributed equally to this work.  18  1  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  1 2 3 Abstract  Conditions (GASAC) that takes into account adherence both to advice and to all types of prescriptions   Background: Adherence in the context of patients with acute conditions is a major public health issue.   It is neglected by the research community and no clinically validated generic scale exists to measure it.  observational study in two hospital emergency departments. Patients were contacted by phone about   one week after their consultation to answer several questionnaires, including GASAC and the Girerd   that the doctor may give. To measure adherence and to study its determinants.  model and a literature search. Then, between 2013 and 2014, we validated it in a prospective   Objective: To construct and validate a Global Adherence Scale usable in the context of Acute   4 5 6 7 8 9 Materials and method: We based the construction of the GASAC questionnaire on a theoretical  10 11 12 13 14 15 16 17 18 19 20 21  self-administered questionnaire about medication adherence as a control.  Results: GASAC consists of four adherence subscales: drug prescriptions; blood test/ radiography   (OR 3.69; CI [1.60-8.52]). The    Doctor-Patient Communication score (OR 1.06 by score point, CI   = 0.78) and was correlated with the Girerd score, as was GASAC as a whole (p<0.01)). The median   score was 0.93 IQR [0.78-1] for a maximum value of 1 (n = 154). In multivariate analysis, infection   prescriptions; lifestyle advice and follow-up instructions. An analysis of the 154 sets of answers from   patients showed that the GASAC drug subscale had satisfactory internal coherence (Cronbach’s alpha   was more conducive of good adherence (cut off at ≥ 0.8; n=115/154; 74.7% [67.0-81.3]) than trauma   [1.02-1.10]) also influenced adherence.  Conclusions: GASAC is a generic score to measure all dimensions of adherence in emergency   departments for clinical research and the evaluation of clinical practice. The level of adherence was   high for acute conditions and could be further improved by good Doctor-Patient Communication.    22 23 24  25  26  2  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  1  Introduction  Adherence is an important component in the assessment of the quality of healthcare. A review of the   2 3 literature on the evaluation of adherence found 90 articles published in the last 10 years [1]. For the  4 World  Health  Organization  (WHO)  \"Optimizing  medication  adherence  would  have  more  impact  in  terms of global health than the development of new drugs [2].\" Non- adherence leads to a considerable  5 6 7  waste of health resources (e.g. unnecessary hospitalizations, unused drugs) and is a cause of avoidable  morbidity and mortality [3,4].   this  field  have  been  performed  for  a  particular  chronic  disease  and  many  authors  mainly  studied   adherence to drug prescriptions [7-13]. In only a few cases did they look at adherence in a broader   8 While  chronic  diseases  (hypertension,  asthma,  HIV  etc.)  have  received  much  attention  [2,5,6],  adherence following an acute condition (AC) has been relatively little studied. Most of the studies in  9 10 11 12 13 14 15 16  discrimination  [21].  The  Girerd  adherence  scale,  although  initially  created  for  chronic  disease,  has   sense [4,14-16]. The most widely used scale, considered a reference in the field [17,18], the Morisky   score, measures drug adherence for chronic conditions [19,20]. It is sometimes criticized for its lack of   several advantages: the items that constitute it are compatible with its use in acute conditions and there   is a validated French version [10,11].  17 18 19 20 21 22 23 24 25 26 27  AC  are  the  most  frequent  reason  for  consultations  in  both  general  practice  [22]  and  Emergency   Departments (ED). Primary care services are overcrowded and the management of AC is becoming a   major public health issue [23]. The period after a consultation for an AC is one of high vulnerability for   patients [24]. They run the risk of further health deterioration, may suffer from a misdiagnosis [23] or   even experience side effects from a newly prescribed drug [24]. Immediate vital (e.g. some infections)   and/or long-term outcomes (eg. sequels of an ankles sprain) might possibly arise and could potentially   affect  the  prognosis  [23].  In  such  a  context,  adherence  to  the  doctor’s  advice  and  to  all  types  of   prescriptions given is key to the success of any treatment [25]. Any aspects of the patient\\'s behaviour   (acting  upon  prescriptions  for  X-rays  and  laboratory  tests,  respecting  appointments  with  specialists   etc.) could be as important as adherence to medications, which is already spontaneously high [3,26,   27].   3  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  The  notion  of  adherence  uniquely  to  prescriptions  for  medication  is  insufficient  [27,28].  Although   some  authors  defined  adherence  “as  the  extent  to  which  a  patient\\'s  behaviour  (in  terms  of  taking   medication, following a diet, modifying habits, or attending clinics) coincides with medical or health   advice”  [4],  there  is  still  no  method  to  measure  adherence  that  includes  all  aspects  of  a  patient’s   behaviour after a consultation. There lacks a standardized tool that is well adapted to clinical research   [29] and assessment of health-care quality [1,17], and none suitable for the context of AC [30,31]. The   question of how best to measure adherence is still open [2].  A  generic  scale  would  be  useful  to  analyse  the  relationship  of  between  adherence  to  and  other   outcomes such as Doctor-Patient Communication (DPC) or satisfaction, and to quantify and compare   the impact of measures introduced to improve adherence, such as Patient information leaflets [30,31].  Our  objective  was  to  create  a  Global  Adherence  Scale  usable  in  the  context  of  an  Acute  Condition   (GASAC)  based  on  a  theoretical  model  describing  the  various  dimensions  of  patient  behaviour   following a consultation [15] and the results of a literature search. Then, to validate it in two hospital   emergency departments and analyse its determinants.   1 2 3 4 5 6 7  8 9 10 11 12 13 14 15  16  17 Materials and methods  18 Literature search  19 We  searched  the  Medline  database  using  the  following  Mesh  terms:  patient  compliance,  adherence  20 21  AND scale, tool, assessment, measures or questionnaires, in various combinations. We also consulted   the Embase and PsycInfo databases, and the Cochrane library in English.   22 23 24 25 26  Our search filter covered the period from 1985 to 2014. Only meta-analyses, randomized controlled   trials,  and  reviews  of  the  literature  were  retained.  In  addition  we  searched  English  (NHS)  and  US   (Agency  for  Healthcare  Research  and  Quality,  AHRQ)  institutional  databases  on  quality  of  care   assessment and books on the field. Two doctors independently screened titles and if necessary abstracts   for all types of articles pertinent to adherence in the context of acute conditions. A manual search was   4  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  also conducted from the bibliographies of promising articles. Since our literature search did not find   specific  articles  for  AC,  we  based  the  elaboration  of  our  scale  on:  1/  a  previously  constructed   theoretical model, itself based on the literature, 2/ commonly used definitions of adherence [2,4] and 3/   on an analysis of the literature so as not to overlook any dimension in our scale [32, 36].  1 2 3 4 5  6  7 Questionnaire development  8 Theoretical model used  Analysis  of  adherence  behaviour  from  a  strictly  medical  point  of  view  is  insufficient  [32].  For  this   reason,  we  based  the  construction  of  our  scale  on  an  existing  model  recently  developed  by  a   multidisciplinary team [15] using a multifactorial approach, as recommended by studies in psychology   and  sociology  [5,32].  It  also  helped  us  to  avoid  the  pitfalls  of  vague  terminology  [31],  poor   construction of the scoring system and redundancy between outcomes [15]. This model describes the   four  aspects  of  a  patient’s  behaviour  following  a  consultation:  taking  medications  as  instructed,   following  prescriptions  for  evaluations  and  tests  (radiography,  blood  tests,  appointments  with   specialists), making appropriate lifestyle changes (i.e. diet, stopping smoking, physical activity, alcohol   consumption) and when to engage the healthcare system for worsening or reoccurring symptoms and   9 10 11 12 13 14 15 16 17 18 19 20 21 22 Requirements of the new scale  follow-up.  Respecting  the  model’s  categories,  we  constructed  a  rigorous  scale  with  pertinent  items.   Moreover  the  model  assisted  us  in  studying  the  determinants  of  adherence  such  as  DPC  and   satisfaction, also defined in the model, with a solid theoretical foundation.   23 24 25 26 27 28  The  scale  needed  to  respect  the  following  criteria:  usable  in  routine  practice,  independent  of  any   particular  clinical  situation,  self-reported  by  the  patient,  easy  to  understand,  easily  evaluable,  brief,   respectful of the patient’s privacy, possible to be completed by or together with carers where necessary,   validated and reliable [20]. Self-assessment by the patient was considered the method of choice as it is   fast,  inexpensive,  non-invasive  and  can  potentially  help  detect  the  underlying  reasons  for  non-  adherence [21,37]. In practice certain authors consider it to be the most suitable method for assessing   5  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  health improvement [20] although it might sometimes suffer from self-deception or dishonest answers.   An  objective  measure  such  as  pill  counts,  sometimes  used  in  assessing  drug  adherence,  was  not   1 2 3 4 5 Validation of the questionnaire    feasible in the broad context of assessing other types of behaviour.  6 The pilot study  7 8 9 10  11  The  first  version  was  tested  in  a  pilot  study  on  30  patients  whatever  the  pathology  diagnosed.    Immediately after the consultation the patient was given the questionnaire to complete, followed by an   additional page about their understanding of the questionnaire and open remarks.   Sample size calculation   Library review [27]. To measure the impact of any intervention on adherence we needed to obtain a   either a non-severe trauma i.e. ankle sprain, or a medical indication i.e. an infection, this meant the   Assuming  good  adherence  (estimated  at  85%)  [3]  a  sample  size  of  150  patients  would  provide  an   minimum  of  75  completed  questionnaires  for  each  group.  As  we  intended  to  include  patients  with   accuracy of measurement, with a 95% confidence interval (CI) of ±6%. The inclusion of 150 patients   12 We  based  our  calculation  on  the  general  adherence  literature  according  to  a  systematic  Cochrane  13 14 15 16 17 18 19 20 21 22 23 24 25 Design  (Cronbach alpha with one-sided 95%; CI = 0.75 for three items of the pharmacological subscale and a   would  also  allow  a  calculation  of  the  Cronbach  coefficient  with  a  satisfactory  level  of  precision   Allowing for 20% patients potentially being lost to follow-up we required 180 patients in total. We   minimum sample size for our study was 150.   coefficient estimated at 0.8).   stopped inclusions when this number was reached.  6  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  1 2 3  4 5 6 7 8 9 10 11 12 13 14 15 16 17 18  19 20 21 22 23  24  25 26 27 28  A two-centre prospective observational study was conducted from November 2013 to May 2014 in the   emergency departments of two hospitals. The study was approved by the regional ethics committee   (IRB n° 5891 on 31-Oct-2013).  Physicians who regularly worked in the ED of the two establishments were contacted and voluntarily   participated. The physician briefly presented the study (orally and in a patient information letter) and   proposed  participation  to  all  consecutive  adults  and  children  (>15  and  accompanied  by  an  adult)   diagnosed with a common traumatic or infectious acute condition (ankle sprain or infectious colitis,   pyelonephritis, diverticulitis, prostatitis or pneumonia). These acute conditions were chosen from those   most  commonly  seen  in  primary  care  [22]  and  which  usually  require  medication,  prescriptions  for   specialist  evaluation  or  tests,  and/or  advice  and  follow-up  instructions.  We  excluded  patients  whose   care led to a hospital stay of more than 48 hours.   The patient information letter explained the broad aims of the study: to help develop tools to measure   the quality of care. Details were not given so as to reduce any self-selection bias. If the patient agreed   to participate, they signed a written informed consent. If they declined to participate, this was recorded.   Physicians included patients in the study by completing a short inclusion-case report form, describing   the patient’s baseline and socio-demographic characteristics.   Patients  were  contacted  by  telephone  between  7  and  10  days  after  the  consultation  by  a  study   investigator  who  did  not  participate  in  patient  recruitment.  They  were  asked  the  series  of  questions   from the Girerd scale (yes/no) and then the GASAC questions (scored on a Likert scale of 1 to 4). Next   they  were  asked  a  series  of  questions  about  DPC,  about  patient  satisfaction  and  some  additional   questions about the intentional or non-intentional nature of their behaviour (not counted in the score).  Statistical analysis   Statistical  analysis  was  performed  using  Stata  Version  13.0  (Stata  Corp,  College  Station,  Texas).   Statistical tests were performed with a significance level of 0.05. Qualitative variables are described by   frequency  and  percentages,  and  quantitative  variables  using  medians  and  IQR  [25th  and  75th   percentiles].  The  internal  consistency  of  the  GASAC  score  items  was  assessed  by  Cronbach\\'s  alpha   7  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  [34].  For  quantitative  variables,  we  used  the  Mann-Whitney  test  to  compare  two  groups,  or  the   Kruskal-Wallis test to compare more than two groups (non-parametric tests). For qualitative variables,   we  used  the  Chi-squared  test.  Finally,  we  conducted  a  multivariate  logistic  regression  to  identify   factors associated with a \"high\" adherence according to the distribution of the histogram. To study the   determinants of adherence we had to dichotomise the variables, so patients with a GASAC score ≥ 0.8   were classed as “highly-adherent”, whereas those with a score < 0.8 were classed as “poorly-adherent”.   All patient characteristics, the level of information, satisfaction and DPC score with p<0.2 in univariate   analysis  were  included  into  the  full  model.  The  final  model  was  obtained  by  a  manual  step-wise   logistic regression. The correlation between GASAC and the DPC, and correlation between subscales   of GASAC scores was explored by calculating Spearman\\'s rho.  1 2 3 4 5 6 7 8 9 10 11  Results  12 Literature search   Our search extracted 845 records, including 80 reviews. Among these, neither of the two doctors found   any  reviews  or  original  articles  that  dealt  with  an  acute  condition,  nor  with  global  adherence.   Concordance  was  100%.  Among  the  reviews,  four  dealt  with  adherence  to  exercises  (e.g.  for   musculoskeletal disease); one with showing up at a mammography appointment and one with keeping   to a diet. The rest concerned drug adherence in chronic diseases (HIV, psychiatric disease, diabetes).   Due to the lack of specific articles we consulted those dealing with adherence from a general point of   view, non-specific to any given disease, and searched their bibliographies manually. We used scales   measuring the quality of care from articles, the institutional websites and books [32-36]. Thus while the   literature  did  not  provide  us  with  any  suitable  scales  it  helped  us  to  be  more  precise  and  to  better   formulate  our  questions.  We  profited  from  the  accumulated  knowledge  of  the  multidisciplinary   research team that included experts in “adherence” (AG, BA).   13 14 15 16 17 18 19 20 21 22 23  24  25 Description of the new scale   8  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  1 2 3 4 5 6 7 8 9 10 11  GASAC  incorporates  four  adherence  subscales:  drug  prescriptions  (3  items);  laboratory  tests  and/or   radiography  prescriptions  (one  item);  lifestyle  advice  (one  item)  and  instructions  on  when,  or  if,  to   consult a medical professional again (one item). These 6 items were each rated from 1 to 4 (1 = no, 2 =   rather not, 3 = rather yes, 4 = yes). As not all questions were relevant to all patients (not all patients   received prescriptions and /or instructions) the final score was expressed as the ratio, between 0 and 1,   of the patient’s answers to the questions actually posed by the study investigator and the maximum   possible score from this number of questions (Table 1). The mean time to complete the questionnaire   was 3 minutes.  Table 1. Global Adherence Scale for Acute Conditions (GASAC).  DOCTOR’S BEHAVIOR  Did the doctor give you any of the following:  A- Prescriptions for medications         B- Prescription orders for evaluations, tests or specialized consultation  (imagery,   laboratory analyses, an appointment with a specialist)       C- Advice on hygiene, diet, lifestyle, physical activity, regular activities,   occupational activity, tobacco or alcohol consumption, hydration, environment  D-  Instructions on when to engage the healthcare system for worsening symptoms   (Not scored)  Yes/No  Yes/No  Yes/No  Yes/No  PATIENT’S BEHAVIOR: THE GASAC QUESTIONNAIRE  SCORE  Possible answers (Likert scale): No; not at all=1; Partly =2; Mainly=3; Yes, that’s right= 4  To each of the following questions:  A- Adherence to prescription(s) for medication/drugs (asked if “yes” to A above):  Sub-total:  I- Have you taken the whole course of prescribed medication?     II- Did you take the prescribed doses?  I + II + III = W  III- Did you comply with the regimen and conditions (time at which you took the medication, if you  were fasted or not, before meals, during meals etc.)?   B- Adherence to prescription orders for evaluations, tests or specialized consultation (asked if  “yes” to B above)   Sub-total:  IV- Did you have the additional examinations (imagery, laboratory analyses) or a specialized  consultation (an appointment with a specialist) prescribed by the doctor?  X   C- Adherence to recommendations and advice (asked if “yes” to C above):  Sub-total:  V- Did you follow the advice and recommendations and/or have you changed some habits as a result  of the consultation:  \\uf0b7 \\uf0b7 \\uf0b7  Diet and weight control Lifestyle Physical activity  9  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  \\uf0b7 \\uf0b7 \\uf0b7 \\uf0b7 \\uf0b7  Reduction or cessation of smoking Alcohol consumption Hydration Avoidance of things causing worsening, recurrence, or transmission of infection Personal hygiene  Y   D- Adherence to use of the health care system  Sub-total:   (Asked if “yes” to D above):  VI- Did you follow the doctor’s instructions and advice on if and when to reconsult a healthcare  professional, did you follow his advice?   Z  TOTAL SCORE = [(W + X + Y+ Z) - number of counted questions] /(3*number of counted  questions)  Total Score  (Between 0 and 1).  1  2 3 4 5 6  Table 2 shows the optional supplementary behavioural questions we asked.  Supporting information S1   file  gives  an  example  of  a  patient’s  replies  and  how  it  was  scored.  Supporting  information  S2  file   contains both the GASAC questionnaire and the supplementary optional questions in French.  Table 2. Additional questions about the intentional or non-intentional nature of the patient’s behavior.   If yes, why?...............................................................................  - Did you take any medications other than those prescribed by your doctor? ............ yes / no        - Because you forgot it................................................................................................ yes/ no - Because the treatment is too complex ...................................................................... yes/ no - Because of its side-effects .........................................................................................yes/ no - Because the medicines do you more harm than good: ..............................................yes / no - Because you thought it was not useful or suitable: ...................................................yes / no - Because you already felt better: ................................................................................yes / no - Because you no longer had any pain:.........................................................................yes / no  - Because you did not go to the pharmacy to collect the medication: ……….….........yes / no        7 8 Why did you not take all your medication or follow the advice? 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30  - Have these medications been prescribed by another doctor?.....................................yes / no - Has your doctor prescribed any laboratory tests? …..................................................yes / no - Has your doctor prescribed any imaging examinations? .......................................... yes / no - Has your doctor prescribed any consultations with a specialist? .............................. yes / no - Has your doctor given you another appointment with himself? ............................... yes / no -Would certain information or advice interest your entourage? ………….………….yes / no   If yes, have you informed them? ....................... ........................................................ yes / no,  - Have you needed to seek medical advice about the same problem? ........................ yes / no - If yes, did you consult your general practitioner? .....................................................yes / no - Or another doctor? .................................................................. ..................................yes / no - Or emergency services? ............................. ...............................................................yes / no  If yes, which? ............................................................................  10  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  - Have you called an emergency number? .................................. ................................yes / no - Was your attitude consistent with the advice? ....................... ...................................yes / no  1 2  3  4 5 Clinical study  Population   The median GASAC score was 0.93 IQR [0.78; 1] (n = 154). Figure 1 shows the patient flow chart.   Figure 1. Flow-diagram for the validation of the GASAC scale. Patients lost to follow-up were those   who could not be contacted by telephone after 3 attempts.  Table 3 shows the baseline and socio-demographic characteristics of the patients.   Table 3. The baseline and socio-demographic characteristics of patients  6  7  8 9  10  11 12 13  All patients   Ankle sprain   (n=154)  (n=82)  Infection (n=72)  93 (60.5%)  40 (43%)  53 (57%)  61 (39.6%)  42 (68.9%)  19 (31.1%)  36 [23-57]  27.5 [20-39]  53.5 [29.5-68.5]  83 (53.9%)  71 (46.1%)  62 (74.7%)  20 (28.2%)  21 (25.3%)  51 (71.8%)  Female  Male  Age  Age bands:  <40 yrs  ≥40 yrs  Highest educational level attended:  Junior High  High school  University studies  62 (40.3%)  38 (24.7%)  54 (35.1%)  25 (40.3%)  25 (65.8%)  32 (59.3%)  37 (59.7%)  13 (34.2%)  22 (40.7%)  11  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  Employed in the health care sector  22 (14.3%)  11 (50.0%)  11 (50.0%)  Socio-professional category:  1. Farmers/artisans/tradesmen/   36 (23.4%)  22 (61.1%)  14 (38.9%)  Intellectuals/managers  2. Employees/workers  3. Retired  4. Unemployed person  Single  Living as a couple  43 (27.9%)  31 (20.1%)  44 (28.6%)  77 (50.0%)  77 (50.0%)  26 (60.5%)  5 (16.1%)  29 (65.9%)  46 (59.7%)  36 (46.8%)  17 (39.5%)  26 (83.9%)  15 (34.1%)  31 (40.3%)  41 (53.2%)  1 2 3  Internal validity of the GASAC questionnaire  concerned by all the questions.   alpha 95% confidence interval 0.72). The other subscales were composed of a single question. It was   Cronbach\\'s alpha coefficient, calculated for the drug subscale was 0.78 (n=149; one-sided Cronbach’s   not  relevant  to  calculate  a  Cronbach  coefficient  for  global  adherence  because  not  all  patients  were   4 5 6 7 8 We calculated the Spearman’s coefficient between the different subscales of the GASAC. There was a  9 10 11 12  0.29 with p = 0.01). In contrast, there was no correlation between the drug subscale and the subscale of   correlation between the drug subscale and the use of the health care system (Spearman coefficient =   recommendations and advice, or between the drug subscale and the test and examination subscale.  13 External validity of the GASAC questionnaire   scores distributed as follows: no patient had a score of less than 3/6; 5 (3.3%) had a score of 3/6; 19   (12.8%) had a score of 4/6; 35 (23.5%) had a score of 5/6 and 90 (60.4%) had a score of 6/6. Using a   14 We compared the GASAC and Girerd scores (Figure 2). The Girerd analysis included 149 patients with  15 16 17 18 score (p<0.01) and between the drug sub-section of the GASAC score and the Girerd score (p<0.01).  19 We also found a statistically significant correlation between the GASAC and satisfaction scores (n=   non-parametric test there was a statistically significant link between the GASAC score and the Girerd   12  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  154; Spearman coefficient=0.22; p<0.01), as well as with the DPC score (n=154; Spearman coefficient   = 0.18, p= 0.03).  Figure 2. Comparison between GASAC and Girerd adherence scores (n =149). Box plot representing   the results by continuous variables using median and IQR [25th and 75 percentile].  1 2 3 4 5  6  7 8 The determinants of adherence   9 10 11 12 13 14 15 16 17 18 19 20  Using an univariate analysis, we determined that the variables associated (p<0.05) with high adherence   were  the  age-band  over  40  years,  an  infectious  pathology  (as  compared  to  trauma),  high  patient   satisfaction and a high DPC score.   In the multivariate logistic regression model the explanatory variables for high adherence were the type   of acute condition, trauma versus infectious (odds ratio, OR 3.69; IC [1.60-8.52] with p<0.01) and the   DPC score (OR 1.06; IC [1.02-1.10] by score point with p<0.01) (Table 4). There was no interaction   between  these  two  variables.  No  individual  characteristic,  the  satisfaction  score  nor  the  level  of   information  given,  were  selected  by  the  manual  step-wise  selection  in  the  final  model.  For  the   multivariate analysis the area under the ROC curve was 0.73. The rate of correctly classified patients   was 75.32%, with a Hosmer-Lemeshow chi2 (8) = 7.7 and Prob > chi2 = 0.46.  Table 4. Determinants of a good adherence (n=154) after emergency department consultations  Characteristics of the population studied  High adherence  (GASAC Score ≥ 0.8; n  =115/154, 74.7%)  OR [95%CI]  p  UNIVARIATE ANALYSIS  Age bands <40 yrs ≥40 yrs  Sex  Male    Female    54/83 (65.1%) 61/71 (85.9%)  41/61 (67.2%) 74/93 (79.6%)  Employed in the health care sector  No    Yes                  99/132 (75.0%) 16/22 (72.7%)  1  0.89 [0.32-2.46]  13  13.28 [1.46-7.34]  < 0.01  1  1.90 [0.91-3.96]  0.09  0.82  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  51/62 (82.3%) 25/38 (65.8%) 39/54 (72.2%)  55/77 (71.4%) 60/77 (77.9%)  26/36 (72.2%) 31/43 (72.1%) 28/31 (90.3%) 30/44 (68.2%)  53/82 (64.6%) 62/72 (86.1%)  24/38 (63.2%) 49/64 (76.6%) 42/52 (80.8%)  1  0.41 [0.16-1.06] 0.56 [0.23-1.36]  1  1.41 [0.68-2.93]  1  0.99 [0.37-2.67] 3.59 [0.89-14.5] 0.82 [0.31-2.17]  0.16  0.35  0.15  1  3.39 [1.51-7.60]  < 0.01  1  1.91 [0.79-4.58] 2.45 [0.94-6.36]  0.15  GASAC ≥ 0,8: 19 [17-20] GASAC < 0.8: 18 [14-20]  1.15 by satisfaction  score point [1.03-1.28]  0.01  GASAC ≥ 0,8: 53,1 [48- 56.8] GASAC < 0,8: 50 [37-53]  1.06 by DPC score   point  [1.02-1.10]  < 0.01  < 0.01  < 0.01  Adjusted OR 1.06  [1.02-1.10] by score  point  Adjusted OR 3.69 [1.60-8.52]  Highest educational level attended Junior High   High school   University studies   Family context   Single   Living as a couple   Socio-professional category: Farmers/artisans/tradesmen Intellectuals/managers  Employees/Workers/Retirees Unemployed person  Type of conditions: Ankle sprain  Infection   Level of information*  0: No information received  1: One information received  2: The two information received    Satisfaction score (continuous)  DPC score (continuous)   MULTIVARIATE ANALYSIS **  DPC score  Infection vs trauma     1 2 3 4 5  * 2 questions were asked about information received: \"Has your doctor given you any advice to follow?\" And   \"Has the doctor given you information on when to reconsult?\"   **Final model with manual stepwise logistic regression   6 Additional Results  7 8 9 10  Ninety-eight  patients  (65.8%)  reported  having  followed  the  entire  treatment;  134  patients  (89.9%)   reported having taken the prescribed daily doses and 110 (73.8%) said they complied with instructions   for the treatment (when and how to take the treatment). Eighty of the 98 patients who had blood tests or   radiography prescriptions (81.6%) said they had fully complied with them. Sixty-five patients out of 89   14  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  who had been given lifestyle and dietary instructions (73%) said they had followed the advice closely.   Sixty-eight  (84%)  of  the  81  patients  said  they  had  respected  the  advice  concerning  potential   reconsultations.   1 2 3 4  5 Discussion   6 Characteristics of the GASAC and strengths  The GASAC is a short, patient self-reported questionnaire evaluating four types of patient behaviour   following  a  consultation  for  an  acute  condition.  The  absence  of  correlations  between  adherence  to   drug-prescriptions  and  adherence  to  prescription  orders  for  evaluations,  tests  or  specialized   consultations,  and  also  with  adherence  to  advice  given  by  the  physician,  shows  that  each  of  the   different subscales provides complementary information and are not redundant.  The validity of the intrinsic properties of the drug sub-section of the GASAC score was confirmed by a   satisfactory Cronbach\\'s alpha coefficient34 and a high rate of correctly classified patients. For external   validity,  the  study  also  showed  convergence  with  the  Girerd  score  as  well  as  with  well-known   determinants of adherence for chronic conditions such as DPC and satisfaction [17,30].  7 8 9 10 11  12 13 14 15  16  17 Clinical results for adherence and comparison with literature   18 19 20 21 22 23 24 25 26 27  Our  median  overall  score  reflects  high  short-term  adherence.  This  adherence  rate  is  consistent  with   those reported in the literature for drug adherence for an AC [3,27]. This high rate can be explained by   the generally strong motivation of patients seeking ED consultations to follow their treatment, even   though  the  conditions  of  patient  management  do  not  always  allow  physicians  to  give  detailed   explanations. In the literature, the average adherence rates are highly dependent on the context of the   study. Nevertheless, adherence rates are higher for acute diseases than for chronic diseases, for which   the median adherence is around 40 to 50% [26,39]. In a study using microelectronic monitors to record   pill taking Cramer et al. showed that 88% of patients adhere to the treatment prescribed during the first   5 days following the consultation, 86% during the next 5 days, dropping to 67% one month after the   consultation [40].  15  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  According to a review of reviews on adherence to medical treatment, the median rate of adherence to   lifestyle and dietary recommendations is generally lower than for drugs [26]. However, this was not the   case  in  our  study:  the  adherence  rate  was  good  for  all  4  dimensions  of  GASAC.  This  might  be   explained by the nature of the acute conditions we chose to study.   In univariate analysis, the factors associated with good adherence were age >40 years, an infectious   pathology, good satisfaction and quality of DPC. Women tended to be more adherent than men, even if   our result wasn\\'t significant. Patients who lived with a partner also tended to be more adherent. In the   literature older and female patients are associated with better adherence [33].  In multivariate analysis, the only determinants of high adherence were infection (versus trauma) and   quality of DPC. Whereas the individual characteristics of the patients (age, sex, level of education and   professional  activity)  were  not  retained  in  the  final  model,  the  DPC  score  was.  This  is  a  major   determinant  for  which  measures  can  be  taken  to  improve,  for  example  through  the  training  of   caregivers.  The  better  adherence  of  patients  with  an  infection  might  be  explained  by  the  risk  of   deterioration without treatment and the well-known efficacy of antibiotics. Older age and female sex   did not appear in the final model because these were also characteristics of patients with infections, as   opposed to younger patients presenting with trauma.   1 2 3 4 5 6 7 8 9  10 11 12 13 14 15 16 17  18  19 Limitations   than others who waited to consult their GP. For minors (n=13, 8%), adherence may well have been   may well have had more severe conditions requiring more urgent attention with better adherence rates   were the least adherent. Our study included patients consulting a hospital emergency department; they   It is possible that patients lost to follow-up (who could not be contacted by telephone after 3 attempts)   20 21 22 23 24 25 26 While the emergency physician who included the patient, and the patient were told that the study was  27  aimed to evaluate tools used to assess the quality of care, they weren’t told in detail what was being   improved by parental supervision.   16  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  1 2  3  4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28  measured  to  avoid  biases.  Moreover,  the  investigators  who  made  the  telephone  interviews  were   completely independent of the emergency departments.   To reduce any social desirability bias (e.g. the patient wanting to be a \"good patient\"), the interviewers   tried  to  create  an  atmosphere  in  which  patients  did  not  feel  judged,  reminding  the  patient  that  their   answers were anonymous and that the interviewer was not part of the medical team. We considered a   telephone questionnaire as being more anonymous and less intimidating for the patient than a face-to-  face interview.  It  is  thought  that  patients  overestimate  their  level  of  adhesion  when  self-reporting  by  10  to  20%   compared with other methods [21]. Despite this drawback, self-evaluation is frequently used in clinical   practice  and  gives  more  reproducible  results,  is  more  reliable,  less  expensive  and  less  invasive  than   direct measurements [28].    To reduce recall bias [20], the telephone interview was made relatively early (7 to 10 days after the   consultation) to obtain more reliable answers, however some patients had not completed their treatment   or  had  not  been  able  have  all  the  additional  tests  required.  Although  telephone  questionnaires  have   limitations  (difficulty  understanding  questions,  call  hours  and  availability  vary  according  to  the   patient), they allow data to be obtained rapidly, the number of patients lost to follow-up is minimized   and  they  are  inexpensive,  To  optimize  accuracy,  when  measuring  adherence  it  is  recommended  to   combine different types of measures [28,41], direct and indirect [3]. However, this is rarely possible in   practice [37,42].  Objective methods would have been further complicated in our study due to the wide   range of behaviours we considered (e.g. following a diet, increasing physical activity etc).   For statistical analysis purposes we needed to dichotomize patients into \"highly-adherent\" or \"poorly-  adherent\" groups, however this led to some simplification and loss of information [21]. For reasons of   feasibility, we limited our study to 6 AC corresponding to 2 subgroups of pathology (trauma versus   infection). Patients who consulted for an infection outnumbered those seen for a sprained ankle, and   17  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  posed  very  different  therapeutic  issues.  However,  we  cannot  generalize  our  findings  to  all  the   pathologies encountered in an ED.  1 2 3 4 Conclusion  The  Global  Adherence  Scale  for  acute  conditions  is  short,  patient  self-reported,  well  adapted  to  the   context of an acute condition, independent of the type of condition and takes into account the different   aspects of a patient’s behaviour. Such a tool can be useful to assess the quality of healthcare both in   clinical research and routine practice. Despite overcrowding in emergency departments, the quality of   doctor-patient  communication  is  a  major  aspect,  which  can  improve  adherence.  Strategies  for   improving doctor patient communication should be tested in an interventional study with adherence as   the major outcome [23,43].   5 6 7 8 9 10 11  12  13 Acknowledgments  Pr Girerd for authorization to use his scale.  14 15 16 Funding  There was no specific funding for this project.  17 18  19 20 References  21 22 23 24 25 26 27 28 29  1. Allenet B, Baudrant M, Lehmann A, Gauchet A, Roustit M, Bedouch P, et al. [How can we   evaluate medication adherence? What are the methods?]. Ann Pharm Fr 2013;71:135–41.   2. Sabate E. Adherence to long-term therapies. Evidence for action. Geneva: World Health   Organization 2003; Available from: whqlibdoc.who.int/publications/2003/9241545992.pdf   3.  Ito H. What Should We Do to Improve Patients’Adherence? J Exp Clin Med 2013; 5:127–30.   18  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58  4. Haynes RB, Mc Donald HP, Garg AX. Helping patients follow prescribed treatment: clinical   applications. J Am Med Assoc 2002 ; 288 :2880-3.   5. Gauchet A, Tarquinio C, Fischer G. Psychosocial predictors of medication adherence among   persons living with HIV. Int J Behav Med 2007;14:141–50.   6. El-Saifi N, Moyle W, Jones C, Tuffaha H. Medication Adherence in older patients with   dementia : a systematic review. J Pharm Pract. 2018 Jun;31(3):322-334.   7. Virjens B, De Geest S, Hughes DA, Przemyslaw K, Demonceau J, Ruppar T, et al. A new   taxonomy for describing and defining adherence to medications: new taxonomy for adherence  to medications. Br J Clin Pharmacol 2012; 3: 691-705.  8. McDonald HP, Garg AX, Haynes RB. Interventions to Enhance Patient Adherence to  Medication Prescriptions: Scientific Review. J Am Med Assoc 2002 ; 288 : 2868-2879.  9. Raynor DK, Blenkinsopp A, Knapp P,  Grime J, Nicolson DJ, Pollock K, et al. A systematic   review of quantitative and qualitative research on the role and effectiveness of written  information available to patients about individual medicines. Health Technol Assess Winch  Engl 2007;11: 1–160.  10. Girerd X, Hanon O, Anagnostopoulos K, Ciupek C. Assessment of antihypertensive   compliance using a self-administered questionnaire: development and use in a hypertension  clinic. Presse Med. 2001 Jun 16-23;30(21):1044-8.  11. Girerd X, Radauceanu A, Achard JM. Evaluation of patient compliance among hypertensive   patients treated by specialists. Arch Mal Cœur Vaiss.2001 Aug;94(8):839-42.  12. Yeowell G, Smith P, Nazir J, Hakimi Z, Siddiqui E, Fatoye F.Real-world persistence   and adherence to oral antimuscarinics and mirabegron in patients with overactive bladder  (OAB): a systematic literature review.BMJ Open. 2018 Nov 21;8(11):e021889.   13. Murage MJ, Tongbram V, Feldman SR, Malatestinic WN, Larmore CJ, Muram TM, et al.  Medication adherence and persistence in patients with rheumatoid arthritis, psoriasis, and  psoriatic arthritis: a systematic literature review. Patient Prefer Adherence. 2018 Aug  21;12:1483-1503.   14. Haynes RB, Taylor WD, Sackett DL. Compliance in health care. Baltimore: Johns Hopkins   University Press 1979; 1-15.   15. Sustersic M, Gauchet A, Foote A, Bosson JL. How best to use and evaluate Patient   Information Leaflets given during a consultation: a systematic review of literature reviews.  Health Expect 2016; 1–12.   16. Sustersic M, Tyrant J, Tissot M, Gauchet A, Foote A, Vermorel C, et al. Impact of patient  information leaflets on doctor–patient communication in the context of acute conditions: a  prospective controlled before–after study in two French emergency departments. BMJ Open  2019;9:e024184.  17. Vermeire E, Hearnshaw H, Van Royen P, Denekens J. Patient adherence to treatment: three   decades of research. A comprehensive review. J Clin Pharm Ther 2001;26:331– 42.   18. Lavsa SM, Holzworth A, Ansani NT. Selection of a validated scale for measuring medication   adherence. J Am Pharm Assoc 2011; 51:90–4.   19. Lehmann A,  Aslani P, Ahmed R, Celio J, Gauchet A, Bedouch P, et al. Assessing medication   adherence: options to consider Int J Clin Pharm 2013; 36: 55-69.   19  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60  20. Garfield S, Clifford S, Eliasson L, Barber N, Willson. Suitability of measures of self- reported  medication adherence for routine clinical use: A systematic review. BMC Med Res Methodol  2011;11:149.   21. Voils CI, Hoyle RH, Thorpe CT. Improving the measurement of self-reported medication   nonadherence. J Clin Epidemiol 2011;64:250–4.  22. French society of general practitioners (SFMG).   http://www.sfmg.org/theorie_pratique/outils_de_la_demarche_medicale/le_dictionnaire_des_r esultats_de_consultation-drc/  23. Ackermann S , Bingisser MB, Heierle A. Discharge communication in the emergency  department: physicians underestimate the time needed. Swiss Med Wkly 2012;142: 1-6.  24. Kripalani S, Jackson AT, Schnipper JL, Coleman EA. Promoting effective transitions of care  at hospital discharge: a review of key issues for hospitalists. J Hosp Med. 2007;2(5):314–23.  Epub 2007/10/16   25. Laufs U, Rettig-Ewen V, Böhm M. Strategies to improve drug adherence. Eur Heart J 2011 ;   32:264–8.   26. Dulmen SV, Sluijs E, Dijk L van, Ridder D de, HeerdinkR, Bensing J. Patient adherence to   medical treatment: a review of reviews. BMC Health Serv Res 2007;7:55.   27. Nieuwlaat R, Wilczynski N, Navarro T, Hobson N, Jeffery R, Keepanasseril A, et al.  Interventions for enhancing medication adherence. Cochrane Database Syst Rev 2014:  CD000011.   28. DiMatteo MR. Variations in patients’ adherence to medical recommendations: a quantitative   review of 50 years of research. Med Care 2004;42:200–9.  29. Nicolson D, Knapp P, Raynor D,  Spoor P, Knapp P. Written information about individual   medicines for consumers. Cochrane Database Syst Rev Online 2009;(2):CsD002104.   30. Simmons S, Sharp B, Fowler J, Fowkes H, Paz-Arabo P, Dilt-Skaggs MK, et al. Mind the   (knowledge) gap: The effect of a communication instrument on emergency department  patients’ comprehension of and satisfaction with care. Patient Educ Couns 2015; 98: 257-262.   31. Bos N, Sizmur S, Graham C, Van Stel HF. The accident and emergency department   questionnaire: a measure for patients’ experiences in the accident and emergency department.  BMJ Qual Saf 2013;22:139–146.  32. Tarquinio C, Fischer G, Grégoire A. Compliance in HIV-positive patients: validation of a   French scale and measurement of psychosocial variables. Rev Int Psychol Soc 2000;13(2):61– 91.   33. Morisky DE, Ang A, Krousel-Wood M, Ward HJ. Predictive validity of a medication  adherence measure in an outpatient setting. J Clin Hypertens. 2008 May;10(5):348-54.  34. Ruiz MA, Pardo A, Rejas J, Soto J, Villasante F, Aranguren JL. Development and validation  of the “Treatment Satisfaction With Medicines Questionnaire” (SATMED-Q). Value Health.  Oct 2008; 11(5) : 913-926.   35. AHRQ. Care Coordination Measures Atlas [Internet]. Agency for Healthcare Research and  Quality; 2010. Available from: http://www.cshcndata.org/docs/drc/ahrq-care-coordination- atlas-dec-2010.pdf  36. Richard C, Lussier MT. [Professional communication in Health]. 2nd edition.Saint-Laurent,   Quebec, Canada: Editions du Renouveau pedagogique; 2016.  20  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21  37. Garratt AM, Bjærtnes A, Krogstad U, Gulbrandsen P. The OutPatient Experiences  Questionnaire (OPEQ): data quality, reliability, and validity in patients attending 52  Norwegian hospitals. Qual Saf Health Care 2005;14:433–437.   38. Bland JM, Altman DG. Statistics notes: Cronbach’s alpha. BMJ 1997; 314:572.   39. Baudrant M. [Reflections  on the role of the pharmacist in patient therapeutic education]. J   Pharm Clin 2008; 27:201–4.   40. Cramer JA, Scheyer RD, Mattson RH. Compliance declines between clinic visits. Arch Intern   Med 1990;150:509e10.   41. Osterberg L, Blaschke T. Adherence to medication. N Engl J Med 2005; 353:487–97.  42. Fossli Jensen B, Dahl FA, Safran DG, Garratt AM, Krupat E, Finset A et al. The ability of a  behaviour-specific patient questionnaire to identify poorly performing doctors. BMJ Qual Saf  2011; 20: 885-893.  43. Hansoti B, Aluisio AR, Barry MA, Davey K, Lentz BA, Modi P, et al. Global Health and   Emergency Care: Defining Clinical Research Priorities. Acad Emerg Med 2017; 24(6) : 742- 753.    21  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  1 Supporting information 2 3 4  S1 file. An example of a patient’s replies and how it was scored.   S2 file. The GASAC questionnaire and the supplementary optional questions in French.   S3 file. Protocol in English.   S4 file. STROBE Checklist.  5  6  7  8  22  \\x0cbioRxiv preprint first posted online Apr. 4, 2019;  The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted bioRxiv a license to display the preprint in perpetuity.  http://dx.doi.org/10.1101/598409 .   doi:   It is made available under a   CC-BY 4.0 International license  .  \\x0c\\x0c'"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_play.body.loc[4]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## loop which goes through all of the files and adds the text to column 'body'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/10/357681.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/10/370114.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/10/762781.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/10/764126.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/10/763748.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/10/763839.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/10/764100.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/10/764266.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/10/764225.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/10/764449.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/10/763953.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/622373.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/692525.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/728089.1.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/737643.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763169.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763094.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/759183.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763185.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763086.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763052.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/761676.1.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763011.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763441.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763003.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/759902.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763144.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763417.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763102.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/726307.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762997.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/652875.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/690677.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763136.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763128.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763177.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763060.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/698845.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763037.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/591610.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763425.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763045.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762989.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/744300.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763433.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763029.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763110.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/722611.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/739326.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/686873.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/759381.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/761189.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/690545.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/749283.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/562322.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/558700.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/645945.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/556621.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/534149.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/756833.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763318.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/750133.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/759498.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762831.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/758987.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762948.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762690.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762765.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763631.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/739060.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763227.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763615.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/761726.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763458.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762500.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762070.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763581.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763532.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/753145.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762112.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762658.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763698.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/761148.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/760322.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/761098.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762138.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763276.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/761874.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763268.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/754093.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763649.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/757278.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762161.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/755975.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763292.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762880.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762062.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/747204.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762922.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/756486.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763235.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762021.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/761999.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/758037.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763334.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/751222.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763672.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/759993.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763367.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763201.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763243.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763573.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/756411.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763599.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763250.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762757.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/761718.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762732.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/755918.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762484.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762278.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/657775.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763987.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/756494.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763326.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762740.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763482.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762476.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/757989.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763623.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762872.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763490.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762971.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763557.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/761700.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762518.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763789.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762054.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763300.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763524.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762229.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/763078.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/760181.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/754903.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/760884.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/761064.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/751297.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/526616.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/468330.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/383331.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762773.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762559.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762666.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/761619.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/678896.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/578526.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762146.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/756734.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/582783.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/629311.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/646208.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/541615.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/465757.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/572057.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/576587.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762708.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/761890.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/762807.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/600148.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/717777.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762443.1.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/537910.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762542.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759779.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760272.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762328.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/755967.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762633.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/748566.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762260.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762534.1.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759449.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/743625.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762179.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762575.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761981.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762294.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762336.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761684.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758078.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761411.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760819.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762450.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/699975.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761734.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761155.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761916.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761817.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/631770.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760223.1.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761791.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761650.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/757393.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761007.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/750729.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760637.full.pdf\n",
      "k\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/749507.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760645.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760835.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/621029.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761247.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760876.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759415.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758359.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761585.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762377.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760918.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760579.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759951.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761049.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761023.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760355.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761122.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760280.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/754754.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761593.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761056.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760298.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761833.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760983.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759613.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761569.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759878.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760215.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760264.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758474.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759738.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760157.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761296.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759159.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759753.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761767.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760173.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759894.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761445.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761502.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760470.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758540.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760934.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761312.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759217.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758425.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/755702.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761387.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761783.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760868.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761692.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/749101.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760082.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759027.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761395.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759696.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760421.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761270.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760330.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760892.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761460.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758615.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761288.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758706.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761551.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759845.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762385.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760926.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758839.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761643.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761528.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760462.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759886.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761379.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/757484.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759860.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761304.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761486.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761338.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760140.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761262.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760603.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761635.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761346.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760025.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760017.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761130.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760975.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759829.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760751.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760843.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760801.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762344.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762351.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761106.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758185.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760744.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760207.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760793.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760652.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/762401.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759464.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758490.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760116.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758102.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758664.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760165.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758128.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760231.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759944.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759191.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760108.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760249.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758755.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/759225.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758094.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761775.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/761320.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/760736.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/758730.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/571612.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/583732.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/08/533653.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/697086.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/757757.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/755710.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/755959.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/751867.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/755280.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/754630.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/755173.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/755983.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/762302.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/740548.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/754671.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/731760.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/727016.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/509364.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/498576.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/761759.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/07/759795.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760785.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760959.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760504.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760678.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760546.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760587.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760611.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760561.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760942.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760538.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760660.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760520.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760496.1.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760777.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760694.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760702.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760769.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760710.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760629.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760595.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760512.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760728.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760553.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/696948.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/716977.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/746339.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/409631.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/440834.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760405.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760256.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760967.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757773.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/758813.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760439.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760371.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/761577.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/752881.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/758680.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760033.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/759787.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/759969.1.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/759043.1.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/418947.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757369.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/290825.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/685503.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/756783.1.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/753863.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/751933.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/753301.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/727743.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/752212.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/751503.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/750323.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/652180.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/753806.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/738526.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/625293.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/622365.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/595751.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/584771.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/573931.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/159129.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/110270.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/733055.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757971.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/756676.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757385.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/759134.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/756445.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/759175.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/756072.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757682.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/758300.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757336.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757054.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757005.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/756270.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/753392.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757260.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/756692.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/756361.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/756668.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/759092.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/759100.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/759068.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/758318.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760389.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760090.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757088.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/760454.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/759928.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/752832.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757153.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/758698.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757906.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/758656.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/758979.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757880.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/756908.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/757146.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/756247.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/418186.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/748087.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/603597.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/741413.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/621250.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/671990.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/583526.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/501502.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/660662.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/666313.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/752972.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/696500.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/748152.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/737783.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/737718.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/532846.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/06/414078.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759340.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759282.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759209.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759266.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759274.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759357.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759324.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759290.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759316.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759233.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759308.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759258.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759332.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758862.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758391.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757542.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757070.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756866.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756809.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756213.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756130.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758458.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759548.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759530.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759985.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/760058.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756593.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759746.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759720.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759852.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759704.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/745091.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/753624.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759506.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759662.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/760041.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759589.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759761.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/746156.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/743690.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758789.1.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755579.1.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757641.1.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/720649.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757229.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/652784.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/501700.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756163.1.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/707620.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/747931.1.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758060.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/739292.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/743419.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/724211.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/712778.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/718841.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/731208.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757062.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/588327.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/614354.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758433.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757252.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/633347.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758029.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/500504.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/575290.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/431643.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758201.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756569.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758219.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756288.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758334.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756536.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/541805.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758805.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756551.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/500512.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759365.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/219931.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758250.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756015.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758607.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756981.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758243.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/754044.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/732917.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755348.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756601.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755553.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759241.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755801.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758524.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755900.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756320.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759407.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755447.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/754648.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/754994.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/752733.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/750547.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/750117.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/752931.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755009.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759001.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/754622.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755017.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757047.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755033.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755454.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758268.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755223.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/754663.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755603.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/748129.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755785.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/752345.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756064.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755595.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755777.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756304.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758797.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755371.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756254.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758854.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755140.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759142.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758581.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759118.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756627.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756767.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758169.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758599.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757534.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757658.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756742.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756726.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755744.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/753954.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757856.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/753707.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757500.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757864.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757922.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757211.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/754143.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758409.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758920.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758375.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758557.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757914.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756395.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755876.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755819.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758276.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756106.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758722.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757096.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756080.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/753459.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756098.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757286.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756924.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757161.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755157.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759076.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758714.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755728.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758516.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757591.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758508.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757948.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755207.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758342.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756056.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756122.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758235.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758367.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757732.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755561.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755629.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757187.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757419.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756544.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756023.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758383.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758284.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756189.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756965.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756197.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757294.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/754929.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756825.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757468.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758326.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758466.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/756155.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758896.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758573.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/757567.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755512.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/753541.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755496.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759050.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758946.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/759084.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/758938.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/754697.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/754473.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/752113.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755611.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/746826.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/751586.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755249.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/753814.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/754986.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/754614.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/745406.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755165.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755421.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/754192.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755694.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/755645.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/746123.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/752964.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/752170.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/05/753293.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/709980.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/533588.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/747279.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757617.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/620245.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757625.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/604249.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757609.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/575399.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757633.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/682245.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/746693.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/746677.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/673350.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/740555.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/572602.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/746743.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753178.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/746263.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/647396.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/637124.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/751982.1.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/750406.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/625707.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/738427.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/704270.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/717405.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/728923.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/570580.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/597161.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/676783.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/659144.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/605550.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/570895.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/490722.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/544304.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/708289.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/480061.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/376756.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/390187.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/503714.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/386334.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/489666.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/427484.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/387282.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/752923.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/752022.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/758144.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756429.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/758110.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/754119.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/755439.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757997.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/752261.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756916.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/755678.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757930.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/754762.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/758193.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/755397.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/758177.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/758151.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757690.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/754747.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757716.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757831.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/755058.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/754408.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/754564.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757328.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/755488.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/755652.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753582.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756528.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753970.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/754085.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757351.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/754077.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753475.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/754135.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757666.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/744136.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757815.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753699.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757435.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757963.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/740761.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/738500.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753772.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/742866.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753566.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753343.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753491.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753384.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753285.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/752394.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753905.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/755025.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756635.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/755066.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757120.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756718.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753525.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753616.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/752410.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753111.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/754499.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/752253.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/752196.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757575.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/755181.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/752121.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/754226.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757179.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756619.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756700.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756858.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753731.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/753210.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/752071.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/754788.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/754168.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756874.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756791.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/752436.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757344.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/749010.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757401.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757013.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756932.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757427.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756841.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/756973.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/757203.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/515809.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/637157.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/747444.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/449645.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/398602.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/282004.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/580365.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/704874.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/214809.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/747303.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/04/245225.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/527739.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/745182.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/756114.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/562272.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/742460.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/748335.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/426759.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/756239.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/756221.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/067736.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/753434.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/720383.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/682880.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755132.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/751065.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/753368.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/757104.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/753244.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755041.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/752063.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/752824.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/756957.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/753335.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755934.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/656637.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/427096.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/745612.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/728022.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/667220.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/654640.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/646042.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/533547.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/530121.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/443648.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/405613.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/752428.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755991.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/756403.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/753681.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/754341.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755827.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/753483.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/752691.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755793.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/756049.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755116.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/756338.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/754812.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/754879.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/741702.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/753558.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/756460.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/752451.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/749135.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/756502.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/754804.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/754887.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/754937.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755769.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755686.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755504.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/754234.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/754721.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/756387.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/689083.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/714345.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/751974.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755322.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755355.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/099697.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/755306.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/476085.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/754705.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/09/753996.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/754739.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/754952.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/750802.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/750778.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/740043.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/700872.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/03/605220.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/755298.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/755314.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/755272.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/743898.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/755256.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/576488.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/676379.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/752485.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/747105.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/681536.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/728451.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/664359.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/740415.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/698647.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/750026.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/356089.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/722462.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/385120.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/701318.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/685081.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/645770.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/480335.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/752493.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/753657.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/752139.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/751628.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/753608.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/752808.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/753590.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/752899.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/754069.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/752683.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/751891.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/753574.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/753350.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/750869.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/750844.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/753160.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/753095.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/753509.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/753822.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/752089.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/753533.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/752246.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/750992.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/752782.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/754838.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/753186.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/753871.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/752519.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/754036.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/755199.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/755389.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/596767.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/754481.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/754960.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/754853.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/755074.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/621938.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/02/594655.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/631341.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/273334.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/729756.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/267047.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/662031.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/739003.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/634576.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/642736.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/606673.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/749911.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/746610.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/743427.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/680504.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/569707.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/623173.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/539445.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/508655.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/753830.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/741264.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/748293.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/751248.full.pdf\n",
      "missed a file\n",
      "https://www.biorxiv.org/content/biorxiv/early/2019/09/01/750299.full.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1709.09236.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1710.02431.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1803.02136.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1803.05404.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1803.08362.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1805.06391.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1805.09101.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1806.08647.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1807.07847.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1807.10872.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1808.09764.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1810.07809.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1811.08885.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1811.09188.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1811.10423.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1811.11885.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1811.12355.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1811.12496.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1812.05780.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1812.08401.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1812.08491.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1901.07945.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1902.00018.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1902.00817.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1902.01155.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1902.02303.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1902.02444.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1902.05527.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1902.07942.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1902.10240.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1903.02594.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1903.03048.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1903.03232.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1903.10042.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1903.10120.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1903.10327.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1904.03410.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1904.04328.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1904.05183.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1904.06326.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1904.10664.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1904.11161.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1905.06236.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1905.08129.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1905.11594.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1905.11848.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1905.12355.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1905.13432.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1906.00905.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1906.02076.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1906.04266.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1906.07570.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1907.00097.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1907.03341.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1907.05036.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1907.05529.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1907.07895.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1907.08371.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1907.09551.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1907.10117.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1907.10419.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1907.10748.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1907.13223.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1907.13371.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1907.13549.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.01867.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.04290.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.07075.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.07209.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.11374.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.11399.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.11436.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.11463.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.11541.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.11623.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.11655.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1908.11739.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.11770.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.11805.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1908.11865.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00012.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00023.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00038.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00042.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00072.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00237.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00337.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1909.00404.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00454.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00455.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00489.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00585.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00651.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00653.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00731.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00892.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.00911.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1909.00954.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01012.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01080.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01196.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01266.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01288.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01319.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01401.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01437.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01439.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1909.01468.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01498.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01561.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01689.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01711.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01735.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01803.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01856.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01872.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01885.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01908.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.01915.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1909.01963.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02093.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02136.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02148.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1909.02199.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02295.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02297.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02336.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02456.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02462.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02492.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02603.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02636.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02664.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02684.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02695.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02833.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02931.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02944.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02947.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02949.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.02953.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.03083.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.03091.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.03109.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.03139.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.03153.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1909.03202.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.03253.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.03283.pdf\n",
      "missed a file\n",
      "https://arxiv.org/pdf/1909.03311.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.03335.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.03801.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.03818.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.03986.pdf\n",
      "k\n",
      "https://arxiv.org/pdf/1909.03992.pdf\n",
      "k\n"
     ]
    }
   ],
   "source": [
    "\n",
    "#every once in a while the pdf reader *stumbles* - instead of crashing, \n",
    "#we want to pass over the article.\n",
    "\n",
    "\n",
    "    \n",
    "df_biorxiv['body'] = ''\n",
    "for ix in range(len(df_biorxiv.atom_path)):\n",
    "    try:\n",
    "        df_biorxiv['body'].loc[ix] = PdfConverter(file_path=download_file(df_biorxiv.atom_path.loc[ix].replace('/biorxiv/', 'https://www.biorxiv.org/content/biorxiv/').replace( '.atom','')+'.full.pdf')).convert_pdf_to_txt().replace('\\n',' ')\n",
    "        df_biorxiv.to_pickle('biorxiv_withbody'+today+'.pkl')\n",
    "        print('k')\n",
    "    except:\n",
    "        print('missed a file')\n",
    "        pass\n",
    "    \n",
    "\n",
    "df_arxiv['body'] = ''\n",
    "for ix in range(len(df_arxiv.id)):\n",
    "    try:\n",
    "        df_arxiv['body'].loc[ix] = PdfConverter(file_path=download_file(\"https://arxiv.org/pdf/%s.pdf\" %df_arxiv['id'].loc[ix])).convert_pdf_to_txt().replace('\\n',' ')\n",
    "        df_arxiv.to_pickle('arxiv_withbody'+today+'.pkl')\n",
    "        print('k')\n",
    "    except:\n",
    "        print('missed a file')\n",
    "        pass\n",
    "\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
